<!DOCTYPE html> <html><head>
		<title>2011 MER 从核苷酸序列估计人口统计历史的天际线图方法</title>
		<base href="..\../">
		<meta id="root-path" root-path="..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 2011 MER 从核苷酸序列估计人口统计历史的天际线图方法">
		<meta property="og:title" content="2011 MER 从核苷酸序列估计人口统计历史的天际线图方法">
		<meta property="og:description" content="思维领域 - 2011 MER 从核苷酸序列估计人口统计历史的天际线图方法">
		<meta property="og:type" content="website">
		<meta property="og:url" content="文献及报道/文献/2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html">
		<meta property="og:image" content="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132250000.png">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon trim-cols h1-l h2-l h3-l h4-l h5-l h6-l image-blend-light active-line-on checkbox-square styled-scrollbars fast-animations"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-container[jax="CHTML"] { line-height: 0; }
mjx-container [space="1"] { margin-left: 0.111em; }
mjx-container [space="2"] { margin-left: 0.167em; }
mjx-container [space="3"] { margin-left: 0.222em; }
mjx-container [space="4"] { margin-left: 0.278em; }
mjx-container [space="5"] { margin-left: 0.333em; }
mjx-container [rspace="1"] { margin-right: 0.111em; }
mjx-container [rspace="2"] { margin-right: 0.167em; }
mjx-container [rspace="3"] { margin-right: 0.222em; }
mjx-container [rspace="4"] { margin-right: 0.278em; }
mjx-container [rspace="5"] { margin-right: 0.333em; }
mjx-container [size="s"] { font-size: 70.7%; }
mjx-container [size="ss"] { font-size: 50%; }
mjx-container [size="Tn"] { font-size: 60%; }
mjx-container [size="sm"] { font-size: 85%; }
mjx-container [size="lg"] { font-size: 120%; }
mjx-container [size="Lg"] { font-size: 144%; }
mjx-container [size="LG"] { font-size: 173%; }
mjx-container [size="hg"] { font-size: 207%; }
mjx-container [size="HG"] { font-size: 249%; }
mjx-container [width="full"] { width: 100%; }
mjx-box { display: inline-block; }
mjx-block { display: block; }
mjx-itable { display: inline-table; }
mjx-row { display: table-row; }
mjx-row > * { display: table-cell; }
mjx-mtext { display: inline-block; }
mjx-mstyle { display: inline-block; }
mjx-merror { display: inline-block; color: red; background-color: yellow; }
mjx-mphantom { visibility: hidden; }
mjx-assistive-mml { top: 0px; left: 0px; clip: rect(1px, 1px, 1px, 1px); user-select: none; position: absolute !important; padding: 1px 0px 0px !important; border: 0px !important; display: block !important; width: auto !important; overflow: hidden !important; }
mjx-assistive-mml[display="block"] { width: 100% !important; }
mjx-math { display: inline-block; text-align: left; line-height: 0; text-indent: 0px; font-style: normal; font-weight: normal; font-size: 100%; letter-spacing: normal; border-collapse: collapse; overflow-wrap: normal; word-spacing: normal; white-space: nowrap; direction: ltr; padding: 1px 0px; }
mjx-container[jax="CHTML"][display="true"] { display: block; text-align: center; margin: 1em 0px; }
mjx-container[jax="CHTML"][display="true"][width="full"] { display: flex; }
mjx-container[jax="CHTML"][display="true"] mjx-math { padding: 0px; }
mjx-container[jax="CHTML"][justify="left"] { text-align: left; }
mjx-container[jax="CHTML"][justify="right"] { text-align: right; }
mjx-msub { display: inline-block; text-align: left; }
mjx-mi { display: inline-block; text-align: left; }
mjx-c { display: inline-block; }
mjx-utext { display: inline-block; padding: 0.75em 0px 0.2em; }
mjx-texatom { display: inline-block; text-align: left; }
mjx-mn { display: inline-block; text-align: left; }
mjx-msqrt { display: inline-block; text-align: left; }
mjx-root { display: inline-block; white-space: nowrap; }
mjx-surd { display: inline-block; vertical-align: top; }
mjx-sqrt { display: inline-block; padding-top: 0.07em; }
mjx-sqrt > mjx-box { border-top: 0.07em solid; }
mjx-sqrt.mjx-tall > mjx-box { padding-left: 0.3em; margin-left: -0.3em; }
mjx-mo { display: inline-block; text-align: left; }
mjx-stretchy-h { display: inline-table; width: 100%; }
mjx-stretchy-h > * { display: table-cell; width: 0px; }
mjx-stretchy-h > * > mjx-c { display: inline-block; transform: scaleX(1); }
mjx-stretchy-h > * > mjx-c::before { display: inline-block; width: initial; }
mjx-stretchy-h > mjx-ext { overflow: clip visible; width: 100%; }
mjx-stretchy-h > mjx-ext > mjx-c::before { transform: scaleX(500); }
mjx-stretchy-h > mjx-ext > mjx-c { width: 0px; }
mjx-stretchy-h > mjx-beg > mjx-c { margin-right: -0.1em; }
mjx-stretchy-h > mjx-end > mjx-c { margin-left: -0.1em; }
mjx-stretchy-v { display: inline-block; }
mjx-stretchy-v > * { display: block; }
mjx-stretchy-v > mjx-beg { height: 0px; }
mjx-stretchy-v > mjx-end > mjx-c { display: block; }
mjx-stretchy-v > * > mjx-c { transform: scaleY(1); transform-origin: left center; overflow: hidden; }
mjx-stretchy-v > mjx-ext { display: block; height: 100%; box-sizing: border-box; border: 0px solid transparent; overflow: visible clip; }
mjx-stretchy-v > mjx-ext > mjx-c::before { width: initial; box-sizing: border-box; }
mjx-stretchy-v > mjx-ext > mjx-c { transform: scaleY(500) translateY(0.075em); overflow: visible; }
mjx-mark { display: inline-block; height: 0px; }
mjx-c::before { display: block; width: 0px; }
.MJX-TEX { font-family: MJXZERO, MJXTEX; }
.TEX-B { font-family: MJXZERO, MJXTEX-B; }
.TEX-I { font-family: MJXZERO, MJXTEX-I; }
.TEX-MI { font-family: MJXZERO, MJXTEX-MI; }
.TEX-BI { font-family: MJXZERO, MJXTEX-BI; }
.TEX-S1 { font-family: MJXZERO, MJXTEX-S1; }
.TEX-S2 { font-family: MJXZERO, MJXTEX-S2; }
.TEX-S3 { font-family: MJXZERO, MJXTEX-S3; }
.TEX-S4 { font-family: MJXZERO, MJXTEX-S4; }
.TEX-A { font-family: MJXZERO, MJXTEX-A; }
.TEX-C { font-family: MJXZERO, MJXTEX-C; }
.TEX-CB { font-family: MJXZERO, MJXTEX-CB; }
.TEX-FR { font-family: MJXZERO, MJXTEX-FR; }
.TEX-FRB { font-family: MJXZERO, MJXTEX-FRB; }
.TEX-SS { font-family: MJXZERO, MJXTEX-SS; }
.TEX-SSB { font-family: MJXZERO, MJXTEX-SSB; }
.TEX-SSI { font-family: MJXZERO, MJXTEX-SSI; }
.TEX-SC { font-family: MJXZERO, MJXTEX-SC; }
.TEX-T { font-family: MJXZERO, MJXTEX-T; }
.TEX-V { font-family: MJXZERO, MJXTEX-V; }
.TEX-VB { font-family: MJXZERO, MJXTEX-VB; }
mjx-stretchy-v mjx-c, mjx-stretchy-h mjx-c { font-family: MJXZERO, MJXTEX-S1, MJXTEX-S4, MJXTEX, MJXTEX-A !important; }
@font-face { font-family: MJXZERO; src: url("lib/fonts/mathjax_zero.woff") format("woff"); }
@font-face { font-family: MJXTEX; src: url("lib/fonts/mathjax_main-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-B; src: url("lib/fonts/mathjax_main-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-I; src: url("lib/fonts/mathjax_math-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-MI; src: url("lib/fonts/mathjax_main-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-BI; src: url("lib/fonts/mathjax_math-bolditalic.woff") format("woff"); }
@font-face { font-family: MJXTEX-S1; src: url("lib/fonts/mathjax_size1-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S2; src: url("lib/fonts/mathjax_size2-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S3; src: url("lib/fonts/mathjax_size3-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S4; src: url("lib/fonts/mathjax_size4-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-A; src: url("lib/fonts/mathjax_ams-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-C; src: url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-CB; src: url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-FR; src: url("lib/fonts/mathjax_fraktur-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-FRB; src: url("lib/fonts/mathjax_fraktur-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SS; src: url("lib/fonts/mathjax_sansserif-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSB; src: url("lib/fonts/mathjax_sansserif-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSI; src: url("lib/fonts/mathjax_sansserif-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-SC; src: url("lib/fonts/mathjax_script-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-T; src: url("lib/fonts/mathjax_typewriter-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-V; src: url("lib/fonts/mathjax_vector-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-VB; src: url("lib/fonts/mathjax_vector-bold.woff") format("woff"); }
mjx-c.mjx-c1D436.TEX-I::before { padding: 0.705em 0.76em 0.022em 0px; content: "C"; }
mjx-c.mjx-c31::before { padding: 0.666em 0.5em 0px 0px; content: "1"; }
mjx-c.mjx-c34::before { padding: 0.677em 0.5em 0px 0px; content: "4"; }
mjx-c.mjx-c221A::before { padding: 0.8em 0.853em 0.2em 0px; content: "√"; }
mjx-c.mjx-c32::before { padding: 0.666em 0.5em 0px 0px; content: "2"; }
</style><pre class="frontmatter language-yaml" style="display: none;" tabindex="0"><code class="is-loaded language-yaml"><span class="token key atrule">icon</span><span class="token punctuation">:</span> LiThumbsUp</code><button class="copy-code-button">复制</button></pre><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="摘要"><div id="webpage-icon"><p dir="auto"><span class="cm-iconize-icon" aria-label="LiThumbsUp" data-icon="LiThumbsUp" aria-hidden="true" style="display: inline-flex; transform: translateY(13%);"><svg xmlns="http://www.w3.org/2000/svg" width="18px" height="18px" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide-thumbs-up"><path d="M7 10v12"></path><path d="M15 5.88 14 10h5.83a2 2 0 0 1 1.92 2.56l-2.33 8A2 2 0 0 1 17.5 22H4a2 2 0 0 1-2-2v-8a2 2 0 0 1 2-2h2.76a2 2 0 0 0 1.79-1.11L12 2h0a3.13 3.13 0 0 1 3 3.88Z"></path></svg></span></p></div><p dir="auto">摘要</p></h1><div class="heading-wrapper"><div class="heading-children"><div><p dir="auto">从核苷酸序列估计人口历史是分子生态学许多研究的重要组成部分。过去，<strong>人口统计分析通常仅限于相对简单的人口模型</strong>，例如指数增长或逻辑增长。现在可以使用更灵活的方法，包括能够重建人口规模随时间变化的天际线图方法。本文主要评论这些技术。</p></div></div></div><div class="heading-wrapper"><h1 data-heading="介绍" dir="auto" class="heading" id="介绍">介绍</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="研究背景" dir="auto" class="heading" id="研究背景"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究背景</h2><div class="heading-children"><div><p dir="auto">重建人群历史对于探索古代气候、追踪病毒传播、厘清人群动态十分重要<sup data-footnote-id="fnref-1-090953f52bb79d17" class="footnote-ref" id="fnref-1-090953f52bb79d17"><a data-footref="1" href="#fn-1-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[1]</a></sup> <sup data-footnote-id="fnref-2-090953f52bb79d17" class="footnote-ref" id="fnref-2-090953f52bb79d17"><a data-footref="2" href="#fn-2-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[2]</a></sup> <sup data-footnote-id="fnref-3-090953f52bb79d17" class="footnote-ref" id="fnref-3-090953f52bb79d17"><a data-footref="3" href="#fn-3-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[3]</a></sup>。一个很好的方法是使用<strong>核苷酸数据</strong><sup data-footnote-id="fnref-4-090953f52bb79d17" class="footnote-ref" id="fnref-4-090953f52bb79d17"><a data-footref="4" href="#fn-4-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[4]</a></sup>。利用数据进行建模可以得到众多参数，这些参数有助于阐明人群的增长或减少。<br>
然而，现实情况可能更复杂。这种疑虑催生了两大类别：<strong>非参数方法</strong>，<strong>半参数方法</strong><sup data-footnote-id="fnref-5-090953f52bb79d17" class="footnote-ref" id="fnref-5-090953f52bb79d17"><a data-footref="5" href="#fn-5-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[5]</a></sup> <sup data-footnote-id="fnref-6-090953f52bb79d17" class="footnote-ref" id="fnref-6-090953f52bb79d17"><a data-footref="6" href="#fn-6-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[6]</a></sup> <sup data-footnote-id="fnref-7-090953f52bb79d17" class="footnote-ref" id="fnref-7-090953f52bb79d17"><a data-footref="7" href="#fn-7-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[7]</a></sup>。<br>
天际线方法允许我们从谱系信息探究<a data-href="有效群体规模 (effective population size，Ne)" href="术语/有效群体规模-(effective-population-size，ne).html" class="internal-link" target="_self" rel="noopener">有效群体规模 (effective population size，Ne)</a>，而不必需要<strong>严格的先验与复杂的模型</strong>。目前出现了多种天际线方法。所有的天际线方法都基于<a data-href="溯祖理论（coalescent theory）" href="疾病与基因/溯祖理论（coalescent-theory）.html" class="internal-link" target="_self" rel="noopener">溯祖理论（coalescent theory）</a><sup data-footnote-id="fnref-8-090953f52bb79d17" class="footnote-ref" id="fnref-8-090953f52bb79d17"><a data-footref="8" href="#fn-8-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[8]</a></sup> <sup data-footnote-id="fnref-9-090953f52bb79d17" class="footnote-ref" id="fnref-9-090953f52bb79d17"><a data-footref="9" href="#fn-9-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[9]</a></sup>。利用支系的数量、长度等指标可以重建不同时间段的<a data-href="有效群体规模 (effective population size，Ne)" href="术语/有效群体规模-(effective-population-size，ne).html" class="internal-link" target="_self" rel="noopener">有效群体规模 (effective population size，Ne)</a>，这是天际线的核心，也导致了天际线无法避免的<strong>局限性</strong>。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="科学问题" dir="auto" class="heading" id="科学问题"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>科学问题</h2><div class="heading-children"><div><p dir="auto">不同的天际线方法有不同的特点，但是这些特点和局限性目前还没有被研究者广泛且深刻地领悟或研究。因此，本研究着重阐明不同天际线方法的优缺点并作出阶段性的总结。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="研究方法" dir="auto" class="heading" id="研究方法"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究方法</h2><div class="heading-children"><div><p dir="auto">利用公共参考数据集对不同的天际线方法的不同参数进行试验，并对结果进行评估、解读和比较。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="数据选择" dir="auto" class="heading" id="数据选择">数据选择</h1><div class="heading-children"><div><p dir="auto">一个理想的研究样本应该符合几个条件：</p></div><div><ol>
<li data-line="0" dir="auto">足够多的个体<sup data-footnote-id="fnref-10-090953f52bb79d17" class="footnote-ref" id="fnref-10-090953f52bb79d17"><a data-footref="10" href="#fn-10-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[10]</a></sup>。保证我们获得了无偏的多样性估计。</li>
<li data-line="1" dir="auto">高度分化的群体中，建议单独分析不同遗传亚结构的人群<sup data-footnote-id="fnref-11-090953f52bb79d17" class="footnote-ref" id="fnref-11-090953f52bb79d17"><a data-footref="11" href="#fn-11-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[11]</a></sup>。复杂的群体结构可能导致天际线对参数的估计出现偏差<sup data-footnote-id="fnref-12-090953f52bb79d17" class="footnote-ref" id="fnref-12-090953f52bb79d17"><a data-footref="12" href="#fn-12-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[12]</a></sup> <sup data-footnote-id="fnref-13-090953f52bb79d17" class="footnote-ref" id="fnref-13-090953f52bb79d17"><a data-footref="13" href="#fn-13-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[13]</a></sup>。这种偏差会在图形中产生反映结构变化的模式而不是<a data-href="有效群体规模 (effective population size，Ne)" href="术语/有效群体规模-(effective-population-size，ne).html" class="internal-link" target="_self" rel="noopener">有效群体规模 (effective population size，Ne)</a> 的变化，<strong>导致对结果的误读</strong>。</li>
<li data-line="2" dir="auto">不合理的采样可能导致天际线方法对突变率的错误估计。</li>
<li data-line="3" dir="auto">利用古代 DNA（<span class="math math-inline is-loaded"><mjx-container class="MathJax" jax="CHTML"><mjx-math class="MJX-TEX"><mjx-msub><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D436 TEX-I"></mjx-c></mjx-mi><mjx-script style="vertical-align: -0.15em; margin-left: -0.045em;"><mjx-texatom size="s" texclass="ORD"><mjx-mn class="mjx-n"><mjx-c class="mjx-c31"></mjx-c><mjx-c class="mjx-c34"></mjx-c></mjx-mn></mjx-texatom></mjx-script></mjx-msub></mjx-math></mjx-container></span> 校准）是一个理想的情况。但在联合使用古代 DNA 和现代 DNA 时，有必要假设<strong>研究群体存在时间连续性</strong>。</li>
</ol></div></div></div><div class="heading-wrapper"><h1 data-heading="基因座的选择" dir="auto" class="heading" id="基因座的选择">基因座的选择</h1><div class="heading-children"><div><ol>
<li data-line="0" dir="auto">这不是一言概之的。</li>
<li data-line="1" dir="auto">天际线方法适用于群体水平，因此最好使用<strong>高进化率</strong>的基因组，以便采集明显的变异<sup data-footnote-id="fnref-14-090953f52bb79d17" class="footnote-ref" id="fnref-14-090953f52bb79d17"><a data-footref="14" href="#fn-14-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[14]</a></sup>。在这一方面，mtDNA 是一个理想的材料，因为它不重组，单系遗传，且突变速率快。然而，线粒体基因组的非重组性质意味着即使有完整的序列，它也必须被视为单个基因座。并且由于仅母系遗传，因此可能<strong>误导我们对性别差异性的人口统计学历史</strong>。</li>
<li data-line="2" dir="auto">增加<strong>基因组的数量</strong>而非增加基因座的<strong>长度</strong>，时提高准确性的有效方法。将独立基因座的数量加倍可以降低 <span class="math math-inline is-loaded"><mjx-container class="MathJax" jax="CHTML"><mjx-math class="MJX-TEX"><mjx-msqrt><mjx-sqrt><mjx-surd><mjx-mo class="mjx-n"><mjx-c class="mjx-c221A"></mjx-c></mjx-mo></mjx-surd><mjx-box style="padding-top: 0.174em;"><mjx-mn class="mjx-n"><mjx-c class="mjx-c32"></mjx-c></mjx-mn></mjx-box></mjx-sqrt></mjx-msqrt></mjx-math></mjx-container></span> 的误差<sup data-footnote-id="fnref-15-090953f52bb79d17" class="footnote-ref" id="fnref-15-090953f52bb79d17"><a data-footref="15" href="#fn-15-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[15]</a></sup>。同时还有助于提高检测<a data-href="瓶颈效应（bottleneck）" href="术语/瓶颈效应（bottleneck）.html" class="internal-link" target="_self" rel="noopener">瓶颈效应（bottleneck）</a>的能力<sup data-footnote-id="fnref-16-090953f52bb79d17" class="footnote-ref" id="fnref-16-090953f52bb79d17"><a data-footref="16" href="#fn-16-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[16]</a></sup>。</li>
<li data-line="3" dir="auto">天际线方法默认承认<a data-href="中性理论" href="中性理论" class="internal-link" target="_self" rel="noopener">中性理论</a>。这是不符合现实的<sup data-footnote-id="fnref-17-090953f52bb79d17" class="footnote-ref" id="fnref-17-090953f52bb79d17"><a data-footref="17" href="#fn-17-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[17]</a></sup> <sup data-footnote-id="fnref-18-090953f52bb79d17" class="footnote-ref" id="fnref-18-090953f52bb79d17"><a data-footref="18" href="#fn-18-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[18]</a></sup>。<a data-href="净化选择（purifying selection）" href="术语/净化选择（purifying-selection）.html" class="internal-link" target="_self" rel="noopener">净化选择（purifying selection）</a>会导致末端节点出出现更多的突变<sup data-footnote-id="fnref-19-090953f52bb79d17" class="footnote-ref" id="fnref-19-090953f52bb79d17"><a data-footref="19" href="#fn-19-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[19]</a></sup> <sup data-footnote-id="fnref-20-090953f52bb79d17" class="footnote-ref" id="fnref-20-090953f52bb79d17"><a data-footref="20" href="#fn-20-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[20]</a></sup> <sup data-footnote-id="fnref-21-090953f52bb79d17" class="footnote-ref" id="fnref-21-090953f52bb79d17"><a data-footref="21" href="#fn-21-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[21]</a></sup>。这可能导致人口历史重建出现偏差。</li>
<li data-line="4" dir="auto">除了数量，天际线方法对质量也敏感。研究显示即使是低水平的序列错误（0.01%）也可能导致估算的双倍错误<sup data-footnote-id="fnref-22-090953f52bb79d17" class="footnote-ref" id="fnref-22-090953f52bb79d17"><a data-footref="22" href="#fn-22-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[22]</a></sup>。<a data-href="下一代测序（next-generation sequencing，NGS）" href="术语/下一代测序（next-generation-sequencing，ngs）.html" class="internal-link" target="_self" rel="noopener">下一代测序（next-generation sequencing，NGS）</a>有助于减轻该问题。</li>
</ol></div></div></div><div class="heading-wrapper"><h1 data-heading="时间校正" dir="auto" class="heading" id="时间校正">时间校正</h1><div class="heading-children"><div><p dir="auto">已知突变速率，可以被用作固定值作为先验。未知突变率则必须引入时间信息，化石记录的分化时间是一个常常被使用的先验。但是存在很大的不确定性。在古代DNA研究或连续采样病毒的研究中，不同年代采集的样本序列本身可以提供足够的校准信息，特别是在采样间隔相对于种群总进化历史较大的情况下<sup data-footnote-id="fnref-23-090953f52bb79d17" class="footnote-ref" id="fnref-23-090953f52bb79d17"><a data-footref="23" href="#fn-23-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[23]</a></sup>。<a data-href="异时性序列（heterochronous sequence）" href="术语/异时性序列（heterochronous-sequence）.html" class="internal-link" target="_self" rel="noopener">异时性序列（heterochronous sequence）</a>则需要慎重的考虑<sup data-footnote-id="fnref-24-090953f52bb79d17" class="footnote-ref" id="fnref-24-090953f52bb79d17"><a data-footref="24" href="#fn-24-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[24]</a></sup>。</p></div></div></div><div class="heading-wrapper"><h1 data-heading="基本方法" dir="auto" class="heading" id="基本方法">基本方法</h1><div class="heading-children"><div><p dir="auto">从序列数据估计系谱，基于系谱估计人口历史。</p></div><div class="heading-wrapper"><h2 data-heading="从系谱估计人口历史" dir="auto" class="heading" id="从系谱估计人口历史"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>从系谱估计人口历史</h2><div class="heading-children"><div><p dir="auto">核心：依赖于<a data-href="溯祖理论（coalescent theory）" href="疾病与基因/溯祖理论（coalescent-theory）.html" class="internal-link" target="_self" rel="noopener">溯祖理论（coalescent theory）</a>。快速连续发生的共祖事件通常表明群体规模较小。天际线图方法（Skyline-plot methods）利用人口规模与共祖间隔长度之间的简单关系来重建人口历史。通过估计每个间隔的平均人口规模，可以逐段重构人口历史。这种方法的输出结果类似于城市的天际线。<strong>过程具有随机性，任何一个谱系都只是这一过程的随机表现之一</strong>。因此，估计每个共祖间隔内的群体大小可能存在较大的误差，<strong>尤其在支系根部时，误差会增加，因为那时仅有少数信息用于重建较远古的群体历史</strong>。最古老的共祖间隔占据整个支系的半数，并且贡献了支系整体深度中大部分的方差<sup data-footnote-id="fnref-25-090953f52bb79d17" class="footnote-ref" id="fnref-25-090953f52bb79d17"><a data-footref="25" href="#fn-25-090953f52bb79d17" class="footnote-link" target="_self" rel="noopener">[25]</a></sup>。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="==天际线方法比较==" dir="auto" class="heading" id="==天际线方法比较=="><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><mark>天际线方法比较</mark></h2><div class="heading-children"><div dir="ltr" style="overflow-x: auto;"><table>
<thead>
<tr>
<th dir="ltr"><strong>方法名称</strong></th>
<th dir="ltr"><strong>主要特点</strong></th>
<th dir="ltr"><strong>优点</strong></th>
<th dir="ltr"><strong>缺点</strong></th>
<th dir="ltr"><strong>软件实现</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td dir="ltr">经典 Skyline 绘图</td>
<td dir="ltr">- 假设已知无误差的系统发育树  <br>- 对每个合并区间估计独立的人口大小  <br>- 噪声较大，特别是在短或零长度合并区间</td>
<td dir="ltr">- 能处理异时序序列  <br>- 输入需求简单，只需系统发育树或合并区间列表</td>
<td dir="ltr">- 重建的人口历史噪声大  <br>- 忽略系统发育不确定性</td>
<td dir="ltr">GENIE 3.0  <br>APE 包</td>
</tr>
<tr>
<td dir="ltr">广义 Skyline 绘图</td>
<td dir="ltr">- 通过合并短合并区间减少噪声  <br>- 使用阈值 ε，低于 ε 的区间与邻近区间合并  <br>- 最优 ε 通过校正的 AIC 选择  <br>- 假设已知系统发育树  <br>- 不能处理异时序序列</td>
<td dir="ltr">- 减少了因短合并区间导致的噪声</td>
<td dir="ltr">- 仍然假设已知系统发育树  <br>- 不能处理异时序序列</td>
<td dir="ltr">GENIE 3.0  <br>APE 包</td>
</tr>
<tr>
<td dir="ltr">贝叶斯多变点方法</td>
<td dir="ltr">- 平滑广义 Skyline 的阶梯式人口函数  <br>- 提供合并误差估计  <br>- 假设人口大小随时间自相关  <br>- 使用样条函数（变点不需与合并事件重合）  <br>- 实现于贝叶斯框架中</td>
<td dir="ltr">- 提供平滑的人口历史估计  <br>- 能估计合并误差</td>
<td dir="ltr">- 假设已知系统发育树  <br>- 难以捕捉突然的人口变化</td>
<td dir="ltr">APE 包</td>
</tr>
<tr>
<td dir="ltr">贝叶斯 Skyline 绘图</td>
<td dir="ltr">- 共估计系统发育树、人口历史和替换模型参数  <br>- 包含系统发育和合并不确定性的置信区间  <br>- 需要预先选择分组数  <br>- 假设人口大小在连续区间相关  <br>- 可采用松弛分子钟</td>
<td dir="ltr">- 考虑了系统发育和合并不确定性  <br>- 能处理异时序序列</td>
<td dir="ltr">- 分组数选择具有主观性  <br>- 过多分组可能增加估计误差</td>
<td dir="ltr">BEAST 软件</td>
</tr>
<tr>
<td dir="ltr">贝叶斯 Skyride</td>
<td dir="ltr">- 提供人口变化的替代模型  <br>- 通过惩罚连续区间人口大小差异平滑人口历史  <br>- 惩罚可时间感知，假设人口大小随时间逐渐变化  <br>- 平滑程度由数据决定的单一参数控制</td>
<td dir="ltr">- 能平滑人口历史，减少噪声  <br>- 不需预先指定分组数</td>
<td dir="ltr">- 可能无法捕捉突然的人口变化</td>
<td dir="ltr">BEAST 软件</td>
</tr>
<tr>
<td dir="ltr">扩展贝叶斯 Skyline 绘图</td>
<td dir="ltr">- 可分析多个不连锁的基因座  <br>- 使用分段线性模型，人口大小在区间内连续变化  <br>- 分组数由随机搜索变量选择确定  <br>- 分配人口大小因子，考虑不同遗传模式  <br>- 通过整合多个基因座，减少估计误差</td>
<td dir="ltr">- 考虑多个基因座，提高估计精度  <br>- 能检测过去的人口瓶颈  <br>- 分组数自动确定</td>
<td dir="ltr">- 模型复杂，计算量大</td>
<td dir="ltr">BEAST 软件</td>
</tr>
</tbody>
</table></div><div><p dir="auto"><strong>相同点：</strong></p></div><div><ul>
<li data-line="0" dir="auto">所有方法都旨在通过基因序列数据重建人口历史。</li>
<li data-line="1" dir="auto">大部分方法都需要系统发育树作为输入，有些假设系统发育树已知且无误差。</li>
<li data-line="2" dir="auto">贝叶斯方法（贝叶斯多变点方法、贝叶斯 Skyline 绘图、贝叶斯 Skyride、扩展贝叶斯 Skyline 绘图）都在贝叶斯框架下工作，可以共同估计系统发育树和人口历史。</li>
</ul></div><div><p dir="auto"><strong>不同点：</strong></p></div><div><ul>
<li data-line="0" dir="auto"><strong>处理系统发育不确定性：</strong> 经典 Skyline 和广义 Skyline 假设已知系统发育树，而贝叶斯方法可以共同估计系统发育树，考虑系统发育不确定性。</li>
<li data-line="1" dir="auto"><strong>处理异时序序列：</strong> 经典 Skyline 和贝叶斯 Skyline 绘图能处理异时序序列，广义 Skyline 不能。</li>
<li data-line="2" dir="auto"><strong>人口变化模型：</strong> 经典和广义 Skyline 采用分段常数模型，贝叶斯多变点方法使用样条函数，贝叶斯 Skyride 假设人口大小随时间平滑变化，扩展贝叶斯 Skyline 可使用分段线性模型。</li>
<li data-line="3" dir="auto"><strong>处理多基因座：</strong> 扩展贝叶斯 Skyline 绘图可以分析多个基因座，其他方法一般只适用于单一基因座。</li>
</ul></div><div><p dir="auto"><strong>优缺点总结：</strong></p></div><div><ul>
<li data-line="0" dir="auto"><strong>经典 Skyline 绘图：</strong> 简单但噪声大，忽略系统发育不确定性。</li>
<li data-line="1" dir="auto"><strong>广义 Skyline 绘图：</strong> 通过合并短区间减少噪声，但不能处理异时序序列，仍假设已知系统发育树。</li>
<li data-line="2" dir="auto"><strong>贝叶斯多变点方法：</strong> 平滑人口历史，能估计合并误差，但难以捕捉突然的人口变化，假设已知系统发育树。</li>
<li data-line="3" dir="auto"><strong>贝叶斯 Skyline 绘图：</strong> 考虑系统发育不确定性，能处理异时序序列，但分组数选择主观，过多分组增加误差。</li>
<li data-line="4" dir="auto"><strong>贝叶斯 Skyride：</strong> 自动平滑，无需预设分组数，但可能遗漏突然的人口变化。</li>
<li data-line="5" dir="auto"><strong>扩展贝叶斯 Skyline 绘图：</strong> 能分析多个基因座，提高精度，自动确定分组数，但计算复杂度高。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="结果解读" dir="auto" class="heading" id="结果解读"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>结果解读</h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>天际线图的解读：</strong>
<ul>
<li data-line="1" dir="auto">天际线图显示了人口规模（y轴）与变化量（x轴）的关系，轴上的单位取决于校准信息的形式。</li>
<li data-line="2" dir="auto">三种校准情况：无校准信息、使用已知速率校准、使用已知节点年龄校准。</li>
</ul>
</li>
<li data-line="3" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>选择合适的方法：</strong>
<ul>
<li data-line="4" dir="auto">可用数据的特性通常决定了最适合的天际线图方法。</li>
<li data-line="5" dir="auto">若数据为长时间范围内快速进化的序列，系统发育误差较小时，可以使用<strong>广义天际线图</strong>。</li>
<li data-line="6" dir="auto">为了<strong>考虑共祖误差，建议使用贝叶斯方法，以便同时估计系谱和人口历史</strong>。若有多个独立基因位点的序列数据，使用<strong>扩展贝叶斯天际线方法</strong>。</li>
</ul>
</li>
<li data-line="7" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>人口模型选择：</strong>
<ul>
<li data-line="8" dir="auto">通过可视化天际线图，可以判断是否需要简单的参数模型来描述人口历史。例如，<strong>完全平坦的天际线图可能表明群体规模保持不变</strong>。</li>
<li data-line="9" dir="auto">95%的可信区间应被考虑，以判断数据是否支持明显的人口变化。</li>
</ul>
</li>
<li data-line="10" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>贝叶斯因子进行模型比较：</strong>
<ul>
<li data-line="11" dir="auto">在贝叶斯分析中，模型的比较可以通过贝叶斯因子进行。贝叶斯因子值大于10时，表示对某个模型的强支持。</li>
</ul>
</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="总结" dir="auto" class="heading" id="总结">总结</h1><div class="heading-children"><div><p dir="auto">随着测序技术的发展有助于进一步提高天际线方法的检测能力，并更精准地探索群体统计学历史。</p></div></div></div><div class="heading-wrapper"><h1 data-heading="图表" dir="auto" class="heading" id="图表">图表</h1><div class="heading-children"><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132250000.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
表 1 根据 DNA 序列数据估计人口历史的天际线图方法比较。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132246697.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 1 根据谱系估计人口历史。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132246328.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 2 模拟数据集不同天际线图方法的性能。用于模拟的实际人口统计历史显示在面板 A 中，50 kyr 处的垂直线表示人口统计函数的变化点。其余面板显示了通过五种天际线图方法重建的人口历史。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132247614.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 3 模拟数据集的不同天际线图方法的性能。用于模拟的实际人口统计历史显示在面板 A 中，50 kyr 处的微弱垂直线表示人口统计函数的变化点。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132249240.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 4 使用线粒体 D 环序列的贝叶斯天际线图估计棕熊 (Ursus arctos) 和洞熊 (Ursus spelaeus) 的人口统计历史。地平线图的终止时间距今大约 24 000 年，表明洞熊已灭绝。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409132250358.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
表 2 天际线图中的轴单位。</p></div></div></div><div class="heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div>
</div><div><section class="footnotes"><hr><ol>
<li data-line="-48" data-footnote-id="fn-1-090953f52bb79d17" id="fn-1-090953f52bb79d17" dir="auto"><p>Drummond AJ, Rambaut A, Shapiro B et al. (2005) Bayesian coalescent inference of past population dynamics from molecular sequences. Molecular Biology &amp; Evolution, 22, 1185–1192.<a href="#fnref-1-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-46" data-footnote-id="fn-2-090953f52bb79d17" id="fn-2-090953f52bb79d17" dir="auto"><p>Finlay EK, Gaillard C, Vahidi SM et al. (2007) Bayesian inference of population expansions in domestic bovines. Biology Letters, 3, 449–452.<a href="#fnref-2-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-44" data-footnote-id="fn-3-090953f52bb79d17" id="fn-3-090953f52bb79d17" dir="auto"><p>Kitchen A, Miyamoto MM, Mulligan CJ (2008) Utility of DNA viruses for studying human host history: case study of JC virus. Molecular Phylogenetics and Evolution, 46, 673–682.<a href="#fnref-3-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-42" data-footnote-id="fn-4-090953f52bb79d17" id="fn-4-090953f52bb79d17" dir="auto"><p>Weiss G, von Haeseler A (1998) Inference of population history using a likelihood approach. Genetics, 149, 1539–1546.<a href="#fnref-4-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-40" data-footnote-id="fn-5-090953f52bb79d17" id="fn-5-090953f52bb79d17" dir="auto"><p>Fu Y-X (1994) A phylogenetic estimator of effective population size or mutation rate. Genetics, 136, 685–692.<a href="#fnref-5-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-38" data-footnote-id="fn-6-090953f52bb79d17" id="fn-6-090953f52bb79d17" dir="auto"><p>Polanski A, Kimmel M, Chakraborty R (1998) Application of a timedependent coalescence process for inferring the history of population size changes from DNA sequence data. Proceedings of the National Academy of Sciences of the United States of America, 95, 5456–5461.<a href="#fnref-6-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-36" data-footnote-id="fn-7-090953f52bb79d17" id="fn-7-090953f52bb79d17" dir="auto"><p>Pybus OG, Rambaut A, Harvey PH (2000) An integrated framework for the inference of viral population history from reconstructed genealogies. Genetics, 155, 1429–1437.<a href="#fnref-7-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-34" data-footnote-id="fn-8-090953f52bb79d17" id="fn-8-090953f52bb79d17" dir="auto"><p>Kingman JFC (1982a) The coalescent. Stochastic Processes and their Applications, 13, 235–248.<a href="#fnref-8-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-32" data-footnote-id="fn-9-090953f52bb79d17" id="fn-9-090953f52bb79d17" dir="auto"><p>Kingman JFC (1982b) On the genealogy of large populations. Journal of Applied Probability, 19A, 27–43.<a href="#fnref-9-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-30" data-footnote-id="fn-10-090953f52bb79d17" id="fn-10-090953f52bb79d17" dir="auto"><p>Drummond AJ, Rambaut A, Shapiro B et al. (2005) Bayesian coalescent inference of past population dynamics from molecular sequences. Molecular Biology &amp; Evolution, 22, 1185–1192.<a href="#fnref-10-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-28" data-footnote-id="fn-11-090953f52bb79d17" id="fn-11-090953f52bb79d17" dir="auto"><p>Miller HC, Moore JA, Allendorf FW et al. (2009) The evolutionary rate of tuatara revisited. Trends in Genetics, 25, 13–15.<a href="#fnref-11-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-26" data-footnote-id="fn-12-090953f52bb79d17" id="fn-12-090953f52bb79d17" dir="auto"><p>Strimmer K, Pybus OG (2001) Exploring the demographic history of DNA sequences using the generalized skyline plot. Molecular Biology and Evolution, 18, 2298–2305.<a href="#fnref-12-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-24" data-footnote-id="fn-13-090953f52bb79d17" id="fn-13-090953f52bb79d17" dir="auto"><p>Navascue´s M, Emerson BC (2009) Elevated substitution rate estimates from ancient DNA: model violation and bias of Bayesian methods. Molecular Ecology, 18, 4390–4397.<a href="#fnref-13-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-22" data-footnote-id="fn-14-090953f52bb79d17" id="fn-14-090953f52bb79d17" dir="auto"><p>Heled J, Drummond AJ (2008) Bayesian inference of population size history from multiple loci. BMC Evolutionary Biology, 8, 289.<a href="#fnref-14-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-20" data-footnote-id="fn-15-090953f52bb79d17" id="fn-15-090953f52bb79d17" dir="auto"><p>Heled J, Drummond AJ (2008) Bayesian inference of population size history from multiple loci. BMC Evolutionary Biology, 8, 289.<a href="#fnref-15-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-18" data-footnote-id="fn-16-090953f52bb79d17" id="fn-16-090953f52bb79d17" dir="auto"><p>Heled J, Drummond AJ (2008) Bayesian inference of population size history from multiple loci. BMC Evolutionary Biology, 8, 289.<a href="#fnref-16-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-16" data-footnote-id="fn-17-090953f52bb79d17" id="fn-17-090953f52bb79d17" dir="auto"><p>Stewart JB, Freyer C, Elson JL et al. (2008) Strong purifying selection in transmission of mammalian mitochondrial DNA. PLoS Biology, 6, e10.<a href="#fnref-17-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-14" data-footnote-id="fn-18-090953f52bb79d17" id="fn-18-090953f52bb79d17" dir="auto"><p>Subramanian S (2009) Temporal trails of natural selection in human mitogenomes. Molecular Biology and Evolution, 26, 715–717.<a href="#fnref-18-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-12" data-footnote-id="fn-19-090953f52bb79d17" id="fn-19-090953f52bb79d17" dir="auto"><p>Fu YX, Li WH (1993) Statistical test of neutrality of mutations. Genetics, 133, 693–709.<a href="#fnref-19-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-10" data-footnote-id="fn-20-090953f52bb79d17" id="fn-20-090953f52bb79d17" dir="auto"><p>Williamson S, Orive ME (2002) The genealogy of a sequence subject to purifying selection at multiple sites. Molecular Biology &amp; Evolution, 19, 1376–1384.<a href="#fnref-20-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-8" data-footnote-id="fn-21-090953f52bb79d17" id="fn-21-090953f52bb79d17" dir="auto"><p>[[2024 MBE MtDNA基因组揭示了末次盛冰期后人口扩张过程中宽松的净化选择<br>
]]<a href="#fnref-21-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-6" data-footnote-id="fn-22-090953f52bb79d17" id="fn-22-090953f52bb79d17" dir="auto"><p>Heled J, Drummond AJ (2008) Bayesian inference of population size history from multiple loci. BMC Evolutionary Biology, 8, 289.<a href="#fnref-22-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-4" data-footnote-id="fn-23-090953f52bb79d17" id="fn-23-090953f52bb79d17" dir="auto"><p>Rambaut A, Ho SYW, Drummond AJ et al. (2009) Accommodating the effect of ancient DNA damage on inferences of demographic histories. Molecular Biology and Evolution, 26, 245–248.<a href="#fnref-23-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-2" data-footnote-id="fn-24-090953f52bb79d17" id="fn-24-090953f52bb79d17" dir="auto"><p>Ho SYW, Saarma U, Barnett R et al. (2008) The effect of inappropriate calibration: three case studies in molecular ecology. PLoS ONE, 3, e1615.<a href="#fnref-24-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="0" data-footnote-id="fn-25-090953f52bb79d17" id="fn-25-090953f52bb79d17" dir="auto"><p>Hein J, Schierup MH, Wiuf C (2005) Gene Genealogies, Variation and Evolution. Oxford University Press, Oxford.<a href="#fnref-25-090953f52bb79d17" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
</ol></section></div><div class="mod-footer"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#摘要"><div class="tree-item-contents heading-link" heading-name="摘要"><span class="tree-item-title">摘要</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#介绍"><div class="tree-item-contents heading-link" heading-name="介绍"><span class="tree-item-title">介绍</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#研究背景"><div class="tree-item-contents heading-link" heading-name="研究背景"><span class="tree-item-title">研究背景</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#科学问题"><div class="tree-item-contents heading-link" heading-name="科学问题"><span class="tree-item-title">科学问题</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#研究方法"><div class="tree-item-contents heading-link" heading-name="研究方法"><span class="tree-item-title">研究方法</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#数据选择"><div class="tree-item-contents heading-link" heading-name="数据选择"><span class="tree-item-title">数据选择</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#基因座的选择"><div class="tree-item-contents heading-link" heading-name="基因座的选择"><span class="tree-item-title">基因座的选择</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#时间校正"><div class="tree-item-contents heading-link" heading-name="时间校正"><span class="tree-item-title">时间校正</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#基本方法"><div class="tree-item-contents heading-link" heading-name="基本方法"><span class="tree-item-title">基本方法</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#从系谱估计人口历史"><div class="tree-item-contents heading-link" heading-name="从系谱估计人口历史"><span class="tree-item-title">从系谱估计人口历史</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#==天际线方法比较=="><div class="tree-item-contents heading-link" heading-name="==天际线方法比较=="><span class="tree-item-title"><mark>天际线方法比较</mark></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#结果解读"><div class="tree-item-contents heading-link" heading-name="结果解读"><span class="tree-item-title">结果解读</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#总结"><div class="tree-item-contents heading-link" heading-name="总结"><span class="tree-item-title">总结</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#图表"><div class="tree-item-contents heading-link" heading-name="图表"><span class="tree-item-title">图表</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2011-mer-从核苷酸序列估计人口统计历史的天际线图方法.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>