<html><head>
		<title>2015 NC 人群重测序揭示欧洲父系支系的近期大规模扩张</title>
		<base href="..\../">
		<meta id="root-path" root-path="..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 2015 NC 人群重测序揭示欧洲父系支系的近期大规模扩张">
		<meta property="og:title" content="2015 NC 人群重测序揭示欧洲父系支系的近期大规模扩张">
		<meta property="og:description" content="思维领域 - 2015 NC 人群重测序揭示欧洲父系支系的近期大规模扩张">
		<meta property="og:type" content="website">
		<meta property="og:url" content="文献及报道/文献/2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html">
		<meta property="og:image" content="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon trim-cols h1-l h2-l h3-l h4-l h5-l h6-l image-blend-light active-line-on checkbox-square styled-scrollbars fast-animations"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles"></style><pre class="frontmatter language-yaml" tabindex="0" style="display: none;"><code class="language-yaml is-loaded"><span class="token key atrule">icon</span><span class="token punctuation">:</span> ❔</code><button class="copy-code-button">复制</button></pre><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="摘要"><div id="webpage-icon"><p dir="auto"><span><img class="emoji" draggable="false" alt="❔" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/2754.svg" width="16px" height="16px"></span></p></div><p dir="auto">摘要</p></h1><div><blockquote dir="auto">
<p>Large-scale recent expansion of European  patrilineages shown by population resequencing ——Chiara Batini &amp; Mark A. Jobling</p>
</blockquote></div><div class="heading-wrapper"><div class="heading-children"><div><p dir="auto">欧洲人的血统是来自大约1万年前的新石器时代农民，还是来自旧石器时代的狩猎采集者，这一比例<strong>一直存在争议</strong>。<a data-href="Y染色体男性特异性区域 (The male-specific region of  the Y chromosome, MSY)" href="术语/y染色体男性特异性区域-(the-male-specific-region-of-the-y-chromosome,-msy).html" class="internal-link" target="_self" rel="noopener">Y染色体男性特异性区域 (The male-specific region of  the Y chromosome, MSY)</a>已广泛用于这个问题的研究，但<strong>一直缺乏无偏的多样性估计和时间深度分析</strong>。本研究展示了欧洲父系家系最近经历了一次跨大陆的大规模扩张。<br>
对<strong>334</strong>名男性的3.7兆碱基对的 <a data-href="Y染色体男性特异性区域 (The male-specific region of  the Y chromosome, MSY)" href="术语/y染色体男性特异性区域-(the-male-specific-region-of-the-y-chromosome,-msy).html" class="internal-link" target="_self" rel="noopener">Y染色体男性特异性区域 (The male-specific region of  the Y chromosome, MSY)</a>  进行重新测序，这些男性来自<strong>17个欧洲和中东地区的群体</strong>，构建了一个包含5,996个单核苷酸多态性（SNP）的系统发育树。<br>
时间测定表明，<strong>三个主要谱系（I1, R1a 和 R1b）占</strong>样本的64%，其<a data-href="最近共同祖先时间(Time to most recent common ancestor，MRCA)" href="术语/最近共同祖先时间(time-to-most-recent-common-ancestor，mrca).html" class="internal-link" target="_self" rel="noopener">最近共同祖先时间(Time to most recent common ancestor，MRCA)</a> 非常近，介于<strong>3500-7300年前</strong>。17个群体中有13个群体展示了类似的历史特征，这些特征包括在约<strong>2100-4200年前开始的一个人口扩张</strong>。本研究的结果与古代DNA数据相符，并与线粒体DNA的数据形成对比，表明这是一个广泛的男性特异性现象。</p></div></div></div><div class="heading-wrapper"><h1 data-heading="介绍" dir="auto" class="heading" id="介绍">介绍</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="研究背景" dir="auto" class="heading" id="研究背景"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究背景</h2><div class="heading-children"><div><p dir="auto">欧洲人群中的旧石器时代群体和新石器时代的群体的比例一直饱受争议<sup data-footnote-id="fnref-1-8196e1382e012111" class="footnote-ref" id="fnref-1-8196e1382e012111"><a data-footref="1" href="#fn-1-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[1]</a></sup>。早期研究在经典基因频率数据中观察到欧洲的东南-西北渐变，这被归因于农民的种群扩散，或者根据另一种观点，归因于第一次旧石器时代的殖民。最近的全基因组SNP数据集对早期的人口历史的了解很有限<sup data-footnote-id="fnref-2-8196e1382e012111" class="footnote-ref" id="fnref-2-8196e1382e012111"><a data-footref="2" href="#fn-2-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[2]</a></sup> <sup data-footnote-id="fnref-3-8196e1382e012111" class="footnote-ref" id="fnref-3-8196e1382e012111"><a data-footref="3" href="#fn-3-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[3]</a></sup> <sup data-footnote-id="fnref-4-8196e1382e012111" class="footnote-ref" id="fnref-4-8196e1382e012111"><a data-footref="4" href="#fn-4-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[4]</a></sup>。<br>
线粒体DNA（mtDNA）中显示的空间模式是非渐变的，单倍群的年龄估计表明有<strong>重要的旧石器时代贡献</strong><sup data-footnote-id="fnref-5-8196e1382e012111" class="footnote-ref" id="fnref-5-8196e1382e012111"><a data-footref="5" href="#fn-5-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[5]</a></sup>。而 Y 染色体的研究结果并非如此<sup data-footnote-id="fnref-6-8196e1382e012111" class="footnote-ref" id="fnref-6-8196e1382e012111"><a data-footref="6" href="#fn-6-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[6]</a></sup>，导致了不同的解释和争论<sup data-footnote-id="fnref-7-8196e1382e012111" class="footnote-ref" id="fnref-7-8196e1382e012111"><a data-footref="7" href="#fn-7-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[7]</a></sup> <sup data-footnote-id="fnref-8-8196e1382e012111" class="footnote-ref" id="fnref-8-8196e1382e012111"><a data-footref="8" href="#fn-8-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[8]</a></sup>。最常见的西欧谱系R1b-M269单倍群，最初被认为起源于旧石器时代<sup data-footnote-id="fnref-9-8196e1382e012111" class="footnote-ref" id="fnref-9-8196e1382e012111"><a data-footref="9" href="#fn-9-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[9]</a></sup>，但在最近的分析中被认为起源于新石器时代<sup data-footnote-id="fnref-10-8196e1382e012111" class="footnote-ref" id="fnref-10-8196e1382e012111"><a data-footref="10" href="#fn-10-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[10]</a></sup>，这一主张又因STR的选择和样本采集方法而受到质疑<sup data-footnote-id="fnref-11-8196e1382e012111" class="footnote-ref" id="fnref-11-8196e1382e012111"><a data-footref="11" href="#fn-11-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[11]</a></sup>。一般来说，基于STR的日期存在问题，因为适当的突变率尚不确定，且由于其逐步突变过程可能导致长期突变饱和。旧石器时代的主要谱系日期也受到稀少的古代MSY DNA数据的挑战，这些数据表明在5到7千年前与现在之间存在明显的不连续性。<br>
<a data-href="下一代测序（next-generation sequencing，NGS）" href="术语/下一代测序（next-generation-sequencing，ngs）.html" class="internal-link" target="_self" rel="noopener">下一代测序（next-generation sequencing，NGS）</a>有助于解释上述问题。目前尽管有一些研究<sup data-footnote-id="fnref-12-8196e1382e012111" class="footnote-ref" id="fnref-12-8196e1382e012111"><a data-footref="12" href="#fn-12-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[12]</a></sup> <sup data-footnote-id="fnref-13-8196e1382e012111" class="footnote-ref" id="fnref-13-8196e1382e012111"><a data-footref="13" href="#fn-13-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[13]</a></sup>，但是还没有系统性的欧洲人群<a data-href="下一代测序（next-generation sequencing，NGS）" href="术语/下一代测序（next-generation-sequencing，ngs）.html" class="internal-link" target="_self" rel="noopener">下一代测序（next-generation sequencing，NGS）</a> 研究。在人口重建中，来自巴尔干半岛到不列颠群岛和爱尔兰群岛的13/17个群体展示了类似的历史特征，这些特征包括在约<strong>2100年到4200千年前的最低的<a data-href="有效群体规模 (effective population size，Ne)" href="术语/有效群体规模-(effective-population-size，ne).html" class="internal-link" target="_self" rel="noopener">有效群体规模 (effective population size，Ne)</a></strong>，随后扩张。结合其他关于母系遗传的线粒体DNA<sup data-footnote-id="fnref-14-8196e1382e012111" class="footnote-ref" id="fnref-14-8196e1382e012111"><a data-footref="14" href="#fn-14-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[14]</a></sup> <sup data-footnote-id="fnref-15-8196e1382e012111" class="footnote-ref" id="fnref-15-8196e1382e012111"><a data-footref="15" href="#fn-15-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[15]</a></sup>和常染色体DNA<sup data-footnote-id="fnref-16-8196e1382e012111" class="footnote-ref" id="fnref-16-8196e1382e012111"><a data-footref="16" href="#fn-16-8196e1382e012111" class="footnote-link" target="_self" rel="noopener">[16]</a></sup>的数据，结果表明这是一个<strong>最近发生的广泛的男性特异性现象</strong>，这可能与社会选择有关。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="科学问题" dir="auto" class="heading" id="科学问题"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>科学问题</h2><div class="heading-children"><div><p dir="auto">欧洲父系支系的起源和扩张时间，以及它们与旧石器时代和新石器时代群体的关系。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="研究方法" dir="auto" class="heading" id="研究方法"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究方法</h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto">样本采集：最终获得了338个男性 Y 染色体样本。</li>
<li data-line="1" dir="auto">测序、变异调用、过滤等质量控制。</li>
<li data-line="2" dir="auto">使用 PHYLIP v3.69 创建最大简约系统发育树。经历了10次构建并合并。</li>
<li data-line="3" dir="auto">FigTree v1.4.0用于可视化树。</li>
<li data-line="4" dir="auto">使用 AMY-tree v1.2 检查、预测 MSY 单倍群。</li>
<li data-line="5" dir="auto">命名规则 Y 染色体联盟系统发育树。</li>
<li data-line="6" dir="auto">利用<a data-href="贝叶斯进化树分析（Bayesian Evolutionary Analysis by Sampling Trees，BEAST）" href="术语/贝叶斯进化树分析（bayesian-evolutionary-analysis-by-sampling-trees，beast）.html" class="internal-link" target="_self" rel="noopener">贝叶斯进化树分析（Bayesian Evolutionary Analysis by Sampling Trees，BEAST）</a> 和 <a data-href="ρ（rho）方法" href="术语/ρ（rho）方法.html" class="internal-link" target="_self" rel="noopener">ρ（rho）方法</a>计算<a data-href="最近共同祖先时间(Time to most recent common ancestor，MRCA)" href="术语/最近共同祖先时间(time-to-most-recent-common-ancestor，mrca).html" class="internal-link" target="_self" rel="noopener">最近共同祖先时间(Time to most recent common ancestor，MRCA)</a> 和节点的年龄。</li>
<li data-line="7" dir="auto"><a data-href="贝叶斯天际线图（Bayesian Skyline Plot, BSP）" href="术语/贝叶斯天际线图（bayesian-skyline-plot,-bsp）.html" class="internal-link" target="_self" rel="noopener">贝叶斯天际线图（Bayesian Skyline Plot, BSP）</a>计算<a data-href="有效群体规模 (effective population size，Ne)" href="术语/有效群体规模-(effective-population-size，ne).html" class="internal-link" target="_self" rel="noopener">有效群体规模 (effective population size，Ne)</a></li>
<li data-line="8" dir="auto">群体内多样性和地理相关性：<strong>Arlequin 3.5</strong>软件计算<a data-href="Tajima’s D" href="术语/tajima’s-d.html" class="internal-link" target="_self" rel="noopener">Tajima’s D</a> 指标和 <a data-href="Fu’s FS" href="术语/fu’s-fs.html" class="internal-link" target="_self" rel="noopener">Fu’s FS</a></li>
<li data-line="9" dir="auto">使用 R包 stats 的函数 cor.test 进行遗传多样性测量与纬度和经度之间的相关性测试。</li>
<li data-line="10" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><a data-href="近似贝叶斯计算 (Approximate Bayesian Computation, ABC)" href="术语/近似贝叶斯计算-(approximate-bayesian-computation,-abc).html" class="internal-link" target="_self" rel="noopener">近似贝叶斯计算 (Approximate Bayesian Computation, ABC)</a>：使用了<strong>FastSimcoal2</strong>程序生成了每个测试模型的100万个模拟数据集：
<ul>
<li data-line="11" dir="auto">定义祖先状态</li>
<li data-line="12" dir="auto">逻辑回归程序</li>
<li data-line="13" dir="auto">局部加权多元回归</li>
<li data-line="14" dir="auto">计算模型和参数的<a data-href="后验概率" href="术语/后验概率.html" class="internal-link" target="_self" rel="noopener">后验概率</a></li>
</ul>
</li>
<li data-line="15" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>人口统计学模型和先验：考虑了五种不同的人口历史模型（M1-M5），每种模型分别用于每个研究群体：
<ul>
<li data-line="16" dir="auto"><strong>M1模型</strong>：这是最简单的模型，假设群体的有效人口规模在时间上保持恒定。有效人口规模的先验分布为均匀分布，范围在20到20,000之间。</li>
<li data-line="17" dir="auto"><strong>M2模型</strong>：假设群体在古代保持恒定规模（先验分布：1,001–20,000），但在T1代前开始指数减少。减少过程持续LEX代（先验分布：5–634），然后群体在T2代前恢复到恒定规模（先验分布：20–1,000）。</li>
<li data-line="18" dir="auto"><strong>M3模型</strong>：在M2模型的基础上，减少过程之后群体经历了扩张，从T2代前开始，扩张到当前（当前有效人口规模的先验分布：1,001–20,000）。</li>
<li data-line="19" dir="auto"><strong>M4模型</strong>：与M2类似，但不同之处在于M4从T1代前开始的不是减少而是扩张。扩张过程中的有效人口规模的先验分布为20到1,000，而当前有效人口规模为1,001到20,000。</li>
<li data-line="20" dir="auto"><strong>M5模型</strong>：在M4模型的基础上，扩张过程在T2代前结束，随后群体经历了减少，直到现在。</li>
</ul>
</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div>
</div><div><section class="footnotes"><hr><ol>
<li data-line="-15" data-footnote-id="fn-1-8196e1382e012111" id="fn-1-8196e1382e012111" dir="auto"><p>Ammerman, A. J. &amp; Cavalli-Sforza, L. L. Neolithic transition and the genetics of populations in Europe (Princeton University Press, 1984).<a href="#fnref-1-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-14" data-footnote-id="fn-2-8196e1382e012111" id="fn-2-8196e1382e012111" dir="auto"><p>Novembre, J. et al. Genes mirror geography within Europe. Nature 456, 98–101 (2008).<a href="#fnref-2-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-13" data-footnote-id="fn-3-8196e1382e012111" id="fn-3-8196e1382e012111" dir="auto"><p>Lao, O. et al. Correlation between genetic and geographic structure in Europe. Curr. Biol. 18, 1241–1248 (2008).<a href="#fnref-3-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-12" data-footnote-id="fn-4-8196e1382e012111" id="fn-4-8196e1382e012111" dir="auto"><p>Hellenthal, G. et al. A genetic atlas of human admixture history. Science 343, 747–751 (2014).<a href="#fnref-4-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-11" data-footnote-id="fn-5-8196e1382e012111" id="fn-5-8196e1382e012111" dir="auto"><p>Richards, M. et al. Tracing European founder lineages in the near eastern mtDNA pool. Am. J. Hum. Genet. 67, 1251–1276 (2000).<a href="#fnref-5-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-10" data-footnote-id="fn-6-8196e1382e012111" id="fn-6-8196e1382e012111" dir="auto"><p>Rosser, Z. H. et al. Y-chromosomal diversity within Europe is clinal and inﬂuenced primarily by geography, rather than by language. Am. J. Hum. Genet. 67, 1526–1543 (2000).<a href="#fnref-6-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-9" data-footnote-id="fn-7-8196e1382e012111" id="fn-7-8196e1382e012111" dir="auto"><p>Rootsi, S. et al. Phylogeography of Y-chromosome haplogroup I reveals distinct domains of prehistoric gene ﬂow in Europe. Am. J. Hum. Genet. 75, 128–137 (2004).<a href="#fnref-7-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-8" data-footnote-id="fn-8-8196e1382e012111" id="fn-8-8196e1382e012111" dir="auto"><p>Semino, O. et al. Origin, diffusion, and differentiation of Y-chromosome haplogroups E and J: inferences on the neolithization of Europe and later migratory events in the Mediterranean area. Am. J. Hum. Genet. 74, 1023–1034 (2004).<a href="#fnref-8-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-7" data-footnote-id="fn-9-8196e1382e012111" id="fn-9-8196e1382e012111" dir="auto"><p>Semino, O. et al. The genetic legacy of Paleolithic Homo sapiens sapiens in extant Europeans: a Y chromosome perspective. Science 290, 1155–1159 (2000).<a href="#fnref-9-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-6" data-footnote-id="fn-10-8196e1382e012111" id="fn-10-8196e1382e012111" dir="auto"><p>Balaresque, P. et al. A predominantly Neolithic origin for European paternal lineages. PLoS Biol. 8, e1000285 (2010).<a href="#fnref-10-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-5" data-footnote-id="fn-11-8196e1382e012111" id="fn-11-8196e1382e012111" dir="auto"><p>Busby, G. B. et al. The peopling of Europe and the cautionary tale of Y chromosome lineage R-M269. Proc. Biol. Sci. 279, 884–892 (2012).<a href="#fnref-11-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-4" data-footnote-id="fn-12-8196e1382e012111" id="fn-12-8196e1382e012111" dir="auto"><p>Francalacci, P. et al. Low-pass DNA sequencing of 1200 Sardinians reconstructs European Y-chromosome phylogeny. Science 341, 565–569 (2013).<a href="#fnref-12-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-3" data-footnote-id="fn-13-8196e1382e012111" id="fn-13-8196e1382e012111" dir="auto"><p>Lippold, S. et al. Human paternal and maternal demographic histories: insights from high-resolution Y chromosome and mtDNA sequences. Investig. Genet. 5, 13 (2014).<a href="#fnref-13-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-2" data-footnote-id="fn-14-8196e1382e012111" id="fn-14-8196e1382e012111" dir="auto"><p>Lippold, S. et al. Human paternal and maternal demographic histories: insights from high-resolution Y chromosome and mtDNA sequences. Investig. Genet. 5, 13 (2014).<a href="#fnref-14-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-1" data-footnote-id="fn-15-8196e1382e012111" id="fn-15-8196e1382e012111" dir="auto"><p>Brandt, G. et al. Ancient DNA reveals key stages in the formation of central European mitochondrial genetic diversity. Science 342, 257–261 (2013).<a href="#fnref-15-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="0" data-footnote-id="fn-16-8196e1382e012111" id="fn-16-8196e1382e012111" dir="auto"><p>Olalde, I. et al. Derived immune and ancestral pigmentation alleles in a 7,000-year-old Mesolithic European. Nature 507, 225–228 (2014).<a href="#fnref-16-8196e1382e012111" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
</ol></section></div><div class="mod-footer"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#摘要"><div class="tree-item-contents heading-link" heading-name="摘要"><span class="tree-item-title">摘要</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#介绍"><div class="tree-item-contents heading-link" heading-name="介绍"><span class="tree-item-title">介绍</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#研究背景"><div class="tree-item-contents heading-link" heading-name="研究背景"><span class="tree-item-title">研究背景</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#科学问题"><div class="tree-item-contents heading-link" heading-name="科学问题"><span class="tree-item-title">科学问题</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#研究方法"><div class="tree-item-contents heading-link" heading-name="研究方法"><span class="tree-item-title">研究方法</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2015-nc-人群重测序揭示欧洲父系支系的近期大规模扩张.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>