<!DOCTYPE html> <html><head>
		<title>2024 MBE soibean：利用线粒体泛基因组图对古代环境 DNA 进行高分辨率分类鉴定</title>
		<base href="..\../">
		<meta id="root-path" root-path="..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 2024 MBE soibean：利用线粒体泛基因组图对古代环境 DNA 进行高分辨率分类鉴定">
		<meta property="og:title" content="2024 MBE soibean：利用线粒体泛基因组图对古代环境 DNA 进行高分辨率分类鉴定">
		<meta property="og:description" content="思维领域 - 2024 MBE soibean：利用线粒体泛基因组图对古代环境 DNA 进行高分辨率分类鉴定">
		<meta property="og:type" content="website">
		<meta property="og:url" content="文献及报道/文献/2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html">
		<meta property="og:image" content="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201249354.png">
		<meta property="og:site_name" content="思维领域">
		<meta name="author" content="Victory_Hugo"><link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon trim-cols h1-l h2-l h3-l h4-l h5-l h6-l image-blend-light active-line-on checkbox-square hide-help styled-scrollbars fast-animations"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-c.mjx-c2192::before { padding: 0.511em 1em 0.011em 0px; content: "→"; }
mjx-c.mjx-c2C::before { padding: 0.121em 0.278em 0.194em 0px; content: ","; }
mjx-c.mjx-c1D43A.TEX-I::before { padding: 0.705em 0.786em 0.022em 0px; content: "G"; }
mjx-c.mjx-c1D434.TEX-I::before { padding: 0.716em 0.75em 0px 0px; content: "A"; }
mjx-container[jax="CHTML"] { line-height: 0; }
mjx-container [space="1"] { margin-left: 0.111em; }
mjx-container [space="2"] { margin-left: 0.167em; }
mjx-container [space="3"] { margin-left: 0.222em; }
mjx-container [space="4"] { margin-left: 0.278em; }
mjx-container [space="5"] { margin-left: 0.333em; }
mjx-container [rspace="1"] { margin-right: 0.111em; }
mjx-container [rspace="2"] { margin-right: 0.167em; }
mjx-container [rspace="3"] { margin-right: 0.222em; }
mjx-container [rspace="4"] { margin-right: 0.278em; }
mjx-container [rspace="5"] { margin-right: 0.333em; }
mjx-container [size="s"] { font-size: 70.7%; }
mjx-container [size="ss"] { font-size: 50%; }
mjx-container [size="Tn"] { font-size: 60%; }
mjx-container [size="sm"] { font-size: 85%; }
mjx-container [size="lg"] { font-size: 120%; }
mjx-container [size="Lg"] { font-size: 144%; }
mjx-container [size="LG"] { font-size: 173%; }
mjx-container [size="hg"] { font-size: 207%; }
mjx-container [size="HG"] { font-size: 249%; }
mjx-container [width="full"] { width: 100%; }
mjx-box { display: inline-block; }
mjx-block { display: block; }
mjx-itable { display: inline-table; }
mjx-row { display: table-row; }
mjx-row > * { display: table-cell; }
mjx-mtext { display: inline-block; }
mjx-mstyle { display: inline-block; }
mjx-merror { display: inline-block; color: red; background-color: yellow; }
mjx-mphantom { visibility: hidden; }
mjx-assistive-mml { top: 0px; left: 0px; clip: rect(1px, 1px, 1px, 1px); user-select: none; position: absolute !important; padding: 1px 0px 0px !important; border: 0px !important; display: block !important; width: auto !important; overflow: hidden !important; }
mjx-assistive-mml[display="block"] { width: 100% !important; }
mjx-math { display: inline-block; text-align: left; line-height: 0; text-indent: 0px; font-style: normal; font-weight: normal; font-size: 100%; letter-spacing: normal; border-collapse: collapse; overflow-wrap: normal; word-spacing: normal; white-space: nowrap; direction: ltr; padding: 1px 0px; }
mjx-container[jax="CHTML"][display="true"] { display: block; text-align: center; margin: 1em 0px; }
mjx-container[jax="CHTML"][display="true"][width="full"] { display: flex; }
mjx-container[jax="CHTML"][display="true"] mjx-math { padding: 0px; }
mjx-container[jax="CHTML"][justify="left"] { text-align: left; }
mjx-container[jax="CHTML"][justify="right"] { text-align: right; }
mjx-mo { display: inline-block; text-align: left; }
mjx-stretchy-h { display: inline-table; width: 100%; }
mjx-stretchy-h > * { display: table-cell; width: 0px; }
mjx-stretchy-h > * > mjx-c { display: inline-block; transform: scaleX(1); }
mjx-stretchy-h > * > mjx-c::before { display: inline-block; width: initial; }
mjx-stretchy-h > mjx-ext { overflow: clip visible; width: 100%; }
mjx-stretchy-h > mjx-ext > mjx-c::before { transform: scaleX(500); }
mjx-stretchy-h > mjx-ext > mjx-c { width: 0px; }
mjx-stretchy-h > mjx-beg > mjx-c { margin-right: -0.1em; }
mjx-stretchy-h > mjx-end > mjx-c { margin-left: -0.1em; }
mjx-stretchy-v { display: inline-block; }
mjx-stretchy-v > * { display: block; }
mjx-stretchy-v > mjx-beg { height: 0px; }
mjx-stretchy-v > mjx-end > mjx-c { display: block; }
mjx-stretchy-v > * > mjx-c { transform: scaleY(1); transform-origin: left center; overflow: hidden; }
mjx-stretchy-v > mjx-ext { display: block; height: 100%; box-sizing: border-box; border: 0px solid transparent; overflow: visible clip; }
mjx-stretchy-v > mjx-ext > mjx-c::before { width: initial; box-sizing: border-box; }
mjx-stretchy-v > mjx-ext > mjx-c { transform: scaleY(500) translateY(0.075em); overflow: visible; }
mjx-mark { display: inline-block; height: 0px; }
mjx-c { display: inline-block; }
mjx-utext { display: inline-block; padding: 0.75em 0px 0.2em; }
mjx-mn { display: inline-block; text-align: left; }
mjx-mi { display: inline-block; text-align: left; }
mjx-c::before { display: block; width: 0px; }
.MJX-TEX { font-family: MJXZERO, MJXTEX; }
.TEX-B { font-family: MJXZERO, MJXTEX-B; }
.TEX-I { font-family: MJXZERO, MJXTEX-I; }
.TEX-MI { font-family: MJXZERO, MJXTEX-MI; }
.TEX-BI { font-family: MJXZERO, MJXTEX-BI; }
.TEX-S1 { font-family: MJXZERO, MJXTEX-S1; }
.TEX-S2 { font-family: MJXZERO, MJXTEX-S2; }
.TEX-S3 { font-family: MJXZERO, MJXTEX-S3; }
.TEX-S4 { font-family: MJXZERO, MJXTEX-S4; }
.TEX-A { font-family: MJXZERO, MJXTEX-A; }
.TEX-C { font-family: MJXZERO, MJXTEX-C; }
.TEX-CB { font-family: MJXZERO, MJXTEX-CB; }
.TEX-FR { font-family: MJXZERO, MJXTEX-FR; }
.TEX-FRB { font-family: MJXZERO, MJXTEX-FRB; }
.TEX-SS { font-family: MJXZERO, MJXTEX-SS; }
.TEX-SSB { font-family: MJXZERO, MJXTEX-SSB; }
.TEX-SSI { font-family: MJXZERO, MJXTEX-SSI; }
.TEX-SC { font-family: MJXZERO, MJXTEX-SC; }
.TEX-T { font-family: MJXZERO, MJXTEX-T; }
.TEX-V { font-family: MJXZERO, MJXTEX-V; }
.TEX-VB { font-family: MJXZERO, MJXTEX-VB; }
mjx-stretchy-v mjx-c, mjx-stretchy-h mjx-c { font-family: MJXZERO, MJXTEX-S1, MJXTEX-S4, MJXTEX, MJXTEX-A !important; }
@font-face { font-family: MJXZERO; src: url("lib/fonts/mathjax_zero.woff") format("woff"); }
@font-face { font-family: MJXTEX; src: url("lib/fonts/mathjax_main-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-B; src: url("lib/fonts/mathjax_main-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-I; src: url("lib/fonts/mathjax_math-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-MI; src: url("lib/fonts/mathjax_main-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-BI; src: url("lib/fonts/mathjax_math-bolditalic.woff") format("woff"); }
@font-face { font-family: MJXTEX-S1; src: url("lib/fonts/mathjax_size1-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S2; src: url("lib/fonts/mathjax_size2-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S3; src: url("lib/fonts/mathjax_size3-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S4; src: url("lib/fonts/mathjax_size4-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-A; src: url("lib/fonts/mathjax_ams-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-C; src: url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-CB; src: url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-FR; src: url("lib/fonts/mathjax_fraktur-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-FRB; src: url("lib/fonts/mathjax_fraktur-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SS; src: url("lib/fonts/mathjax_sansserif-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSB; src: url("lib/fonts/mathjax_sansserif-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSI; src: url("lib/fonts/mathjax_sansserif-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-SC; src: url("lib/fonts/mathjax_script-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-T; src: url("lib/fonts/mathjax_typewriter-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-V; src: url("lib/fonts/mathjax_vector-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-VB; src: url("lib/fonts/mathjax_vector-bold.woff") format("woff"); }
mjx-c.mjx-c3C::before { padding: 0.54em 0.778em 0.04em 0px; content: "<"; }
mjx-c.mjx-c34::before { padding: 0.677em 0.5em 0px 0px; content: "4"; }
mjx-c.mjx-c30::before { padding: 0.666em 0.5em 0.022em 0px; content: "0"; }
mjx-c.mjx-c1D44F.TEX-I::before { padding: 0.694em 0.429em 0.011em 0px; content: "b"; }
mjx-c.mjx-c1D45D.TEX-I::before { padding: 0.442em 0.503em 0.194em 0px; content: "p"; }
mjx-c.mjx-c1D436.TEX-I::before { padding: 0.705em 0.76em 0.022em 0px; content: "C"; }
mjx-c.mjx-c1D447.TEX-I::before { padding: 0.677em 0.704em 0px 0px; content: "T"; }
</style><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="摘要"><p dir="auto">摘要</p></h1><div class="el-h1 heading-wrapper"><div class="heading-children"><div class="el-p"><p dir="auto"><a data-href="古环境DNA (Ancient environmental DNA, aeDNA)" href="术语/古环境dna-(ancient-environmental-dna,-aedna).html" class="internal-link" target="_self" rel="noopener nofollow">古环境DNA (Ancient environmental DNA, aeDNA)</a>正成为探索过去生态系统的有力工具，克服了传统化石记录的局限性。然而，<strong>把DNA分类到物种层面和进行系统发育分析方面，仍然存在一些方法学上的挑战</strong>。目前的方法<strong>主要为现代数据集设计</strong>，<strong>未能充分捕捉aeDNA的独特性</strong>。例如，<a data-href="相近物种混合样本（species mixtures from closely related species）" href="术语/相近物种混合样本（species-mixtures-from-closely-related-species）.html" class="internal-link" target="_self" rel="noopener nofollow">相近物种混合样本（species mixtures from closely related species）</a>与<a data-href="祖先分化（ancestral divergence）" href="术语/祖先分化（ancestral-divergence）.html" class="internal-link" target="_self" rel="noopener nofollow">祖先分化（ancestral divergence）</a>。<br>
本研究介绍了soibean，这是一种利用<strong>线粒体泛基因组图谱</strong>从aeDNA读取中识别物种的<strong>新工具</strong>。它在从样本中多个密切相关来源中准确识别物种方面<strong>优于现有方法</strong>，<strong>增强了对aeDNA的系统发育分析</strong>。<br>
soibean采用一种损伤感知的似然模型，以在<strong>低覆盖度</strong>和<strong>高损伤率</strong>的情况下实现精确识别。soibean通过模拟数据测试和实证验证证明了其有效性。值得注意的是，我们的方法在已发表的数据集中揭示了新的实证结果，例如发现了<strong>瑞典中石器时代社区使用鼠海豚作为食物的证据</strong>，这表明该工具在aeDNA研究中<strong>有潜力揭示以前未被发现的结果</strong>。</p></div></div></div><div class="el-h1 heading-wrapper"><h1 data-heading="介绍" dir="auto" class="heading" id="介绍">介绍</h1><div class="heading-children"><div class="el-h2 heading-wrapper"><h2 data-heading="研究背景" dir="auto" class="heading" id="研究背景"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究背景</h2><div class="heading-children"><div class="el-p"><p dir="auto">古代 DNA 帮助理解考古学和古生物学的发现。古环境 DNA 改变了我们对过去环境和物种在时间、空间尺度的组成的理解。在生物体的整个生命周期中，它会在环境中、沉积物或永久冻土等沉积物中留下遗传痕迹<sup data-footnote-id="fnref-1-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-1-60e71cc97b2ca8d3"><a data-footref="1" href="#fn-1-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[1]</a></sup> <sup data-footnote-id="fnref-2-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-2-60e71cc97b2ca8d3"><a data-footref="2" href="#fn-2-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[2]</a></sup> <sup data-footnote-id="fnref-3-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-3-60e71cc97b2ca8d3"><a data-footref="3" href="#fn-3-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[3]</a></sup> <sup data-footnote-id="fnref-4-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-4-60e71cc97b2ca8d3"><a data-footref="4" href="#fn-4-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[4]</a></sup>。 <a data-href="古环境DNA (Ancient environmental DNA, aeDNA)" href="术语/古环境dna-(ancient-environmental-dna,-aedna).html" class="internal-link" target="_self" rel="noopener nofollow">古环境DNA (Ancient environmental DNA, aeDNA)</a> 允许我们从古代的环境中探索种群和其生存环境<sup data-footnote-id="fnref-5-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-5-60e71cc97b2ca8d3"><a data-footref="5" href="#fn-5-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[5]</a></sup> <sup data-footnote-id="fnref-6-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-6-60e71cc97b2ca8d3"><a data-footref="6" href="#fn-6-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[6]</a></sup>。<br>
分析古 DNA 存在如下重要的问题：</p></div><div class="el-ul"><ul>
<li data-line="0" dir="auto">古 DNA 高度碎片化，且伴随许多化学损伤（例如 <span class="math math-inline is-loaded"><mjx-container class="MathJax" jax="CHTML"><mjx-math class="MJX-TEX"><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D436 TEX-I"></mjx-c></mjx-mi><mjx-mo class="mjx-n" space="4"><mjx-c class="mjx-c2192"></mjx-c></mjx-mo><mjx-mi class="mjx-i" space="4"><mjx-c class="mjx-c1D447 TEX-I"></mjx-c></mjx-mi><mjx-mo class="mjx-n"><mjx-c class="mjx-c2C"></mjx-c></mjx-mo><mjx-mi class="mjx-i" space="2"><mjx-c class="mjx-c1D43A TEX-I"></mjx-c></mjx-mi><mjx-mo class="mjx-n" space="4"><mjx-c class="mjx-c2192"></mjx-c></mjx-mo><mjx-mi class="mjx-i" space="4"><mjx-c class="mjx-c1D434 TEX-I"></mjx-c></mjx-mi></mjx-math></mjx-container></span> 碱基替换）。</li>
<li data-line="1" dir="auto">古 DNA 分析必须考虑物种演化过程，古代物种与现代物种存在序列的差异，可能导致参考基因组的失效，从而导致错误。<br>
<a data-href="古环境DNA (Ancient environmental DNA, aeDNA)" href="术语/古环境dna-(ancient-environmental-dna,-aedna).html" class="internal-link" target="_self" rel="noopener nofollow">古环境DNA (Ancient environmental DNA, aeDNA)</a> 存在类似的问题。同时还有独特的问题：古环境 DNA 往往是多种物种的混合 DNA，这更加剧了分析的不确定性<sup data-footnote-id="fnref-7-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-7-60e71cc97b2ca8d3"><a data-footref="7" href="#fn-7-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[7]</a></sup>。aeDNA 的分类精确度受到 DNA 来源<strong>相对丰度的影响</strong>，<strong>较低丰度的 DNA 难以达到高物种层面的分类精确度</strong>。<br>
目前存在许多方法：</li>
</ul></div><div class="el-ol"><ol>
<li data-line="0" dir="auto">标准的分类方法（如最低共同祖先算法，LCA），aeDNA 的线粒体 DNA 分析通常只能达到较低的分类精度<sup data-footnote-id="fnref-8-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-8-60e71cc97b2ca8d3"><a data-footref="8" href="#fn-8-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[8]</a></sup>。</li>
<li data-line="1" dir="auto"><strong>euka</strong>是一种新的分类工具，适用于较低的分类层次，并通过估计DNA片段的损伤模式或覆盖范围来验证分类结果<sup data-footnote-id="fnref-9-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-9-60e71cc97b2ca8d3"><a data-footref="9" href="#fn-9-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[9]</a></sup>。</li>
<li data-line="2" dir="auto"><strong>HAYSTAC</strong>是一种高分辨率分类工具，使用 Bowtie2 进行全对全映射，并提供验证过滤器（如可能性过滤器、覆盖均匀性过滤器），有助于准确的物种检测。然而，它不考虑私有突变或在系统发育参考中定位样本，限制了对祖先物种的识别<sup data-footnote-id="fnref-10-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-10-60e71cc97b2ca8d3"><a data-footref="10" href="#fn-10-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[10]</a></sup>。</li>
<li data-line="3" dir="auto">系统发育定位是一种基于序列相似性将提取的片段放置在系统发育树上以进行分类的方法，但 aeDNA 通常覆盖率太低，难以可靠调用共识序列。<strong>pathPhynder</strong>工具通过考虑独特的 SNP，在低覆盖率 aDNA 数据中进行物种检测，推断祖先物种状态，但它只适用于单一来源数据分析<sup data-footnote-id="fnref-11-60e71cc97b2ca8d3" class="footnote-ref" id="fnref-11-60e71cc97b2ca8d3"><a data-footref="11" href="#fn-11-60e71cc97b2ca8d3" class="footnote-link" target="_self" rel="noopener nofollow">[11]</a></sup>。</li>
</ol></div></div></div><div class="el-h2 heading-wrapper"><h2 data-heading="科学问题" dir="auto" class="heading" id="科学问题"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>科学问题</h2><div class="heading-children"><div class="el-p"><p dir="auto">如何在古环境DNA（aeDNA）中，准确分类来自多个密切相关物种的DNA片段，并确定它们的系统发育位置？</p></div></div></div><div class="el-h2 heading-wrapper"><h2 data-heading="研究结果" dir="auto" class="heading" id="研究结果"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究结果</h2><div class="heading-children"><div class="el-p"><p dir="auto">本研究提出一个新的分析工具：<strong>soibean</strong> 。该工具结合<strong>线粒体泛基因组图谱</strong>和<strong>贝叶斯推理方法</strong>进行高分辨率分类。泛基因组图谱能够同时表示多个基因组序列，降低参考基因组偏差。soibean 的<strong>输入是已分类到较低层次（如科级）的 aDNA 片段，利用 MCMC 采样估计分类树枝上最可能的物种位置，并估计每个来源的相对丰度</strong>。soibean 的结果提供可信区间和诊断指标，甚至在数据稀少时也能显示不确定性。它还能够在低覆盖率数据中准确识别祖先状态。soibean 通过模拟数据和实证数据证明其特异性和敏感性，展现了发现新实证结果的能力，如发现中石器时代的瑞典社区使用鼠海豚作为食物。</p></div></div></div></div></div><div class="el-h1 heading-wrapper"><h1 data-heading="图表" dir="auto" class="heading" id="图表">图表</h1><div class="heading-children"><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201249354.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 1.soibean 的主要工作流程。</p></div><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201250690.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 2. 使用来自熊科(Ursidae)祖先状态N4的单一DNA来源模拟的古代片段的soibean结果。</p></div><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201250094.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 3. 使用两个相似度为93.1%的熊科(Ursidae)物种混合的模拟古代片段的soibean结果，覆盖度约为2X。</p></div><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201253603.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 4. a) 使用三种来源的模拟古代样本的44%--30%--23%混合物，来自有翅昆虫家族(Saturniidae)。b) 使用来自无耳海豹家族(Phocidae)的四种来源模拟古代样本的25%--25%--25%--25%混合物。</p></div><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201253964.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 5. 两项独立研究的确认结果。</p></div><div class="el-p"><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202410201253742.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 6. 新的发现：a) soibean对从瑞典西北海岸Huseby-Klev发现的9500年前实证样本的结果。b) 包含所有鼠海豚线粒体基因组的最大似然系统发育树图，包括Huseby-Klev样本的共识序列。</p></div></div></div><div class="el-h1 heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div>
</div><div class="el-section"><section class="footnotes"><hr><ol>
<li data-line="-20" data-footnote-id="fn-1-60e71cc97b2ca8d3" id="fn-1-60e71cc97b2ca8d3" dir="auto"><p>Hofreiter M, Mead JI, Martin P, Poinar HN. Molecular caving. Curr Biol. 2003:13(18):R693–R695. <a data-tooltip-position="top" aria-label="https://doi.org/10.1016/j.cub.2003.08.039" rel="noopener nofollow" class="external-link" href="https://doi.org/10.1016/j.cub.2003.08.039" target="_blank">https://doi.org/10.1016/j.cub. 2003.08.039</a>.<a href="#fnref-1-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-18" data-footnote-id="fn-2-60e71cc97b2ca8d3" id="fn-2-60e71cc97b2ca8d3" dir="auto"><p>Willerslev E, Hansen AJ, Binladen J, Brand TB, Gilbert MTP, Shapiro B, Bunce M, Wiuf C, Gilichinsky DA, Cooper A. Diverse plant and animal genetic records from holocene and pleistocene sediments. Science. 2003:300(5620):791–795. <a data-tooltip-position="top" aria-label="https://doi.org/10.1126/science.1084114" rel="noopener nofollow" class="external-link" href="https://doi.org/10.1126/science.1084114" target="_blank">https://doi.org/10. 1126/science.1084114</a>.<a href="#fnref-2-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-16" data-footnote-id="fn-3-60e71cc97b2ca8d3" id="fn-3-60e71cc97b2ca8d3" dir="auto"><p>Lydolph MC, Jacobsen J, Arctander P, Gilbert MTP, Gilichinsky DA, Hansen AJ, Willerslev E, Lange L. Beringian paleoecology inferred from permafrost-preserved fungal DNA. Appl Environ Microbiol. 2005:71(2):1012–1017. <a data-tooltip-position="top" aria-label="https://doi.org/10.1128/AEM.71.2.1012-1017.2005" rel="noopener nofollow" class="external-link" href="https://doi.org/10.1128/AEM.71.2.1012-1017.2005" target="_blank">https://doi.org/10.1128/AEM.71.2.10121017.2005</a>.<a href="#fnref-3-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-14" data-footnote-id="fn-4-60e71cc97b2ca8d3" id="fn-4-60e71cc97b2ca8d3" dir="auto"><p>Willerslev E, Cooper A. Ancient DNA. Proc R Soc B Biol Sci. 2005:272(1558):3–16. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1098/rspb.2004.2813" target="_blank">https://doi.org/10.1098/rspb.2004.2813</a>.<a href="#fnref-4-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-12" data-footnote-id="fn-5-60e71cc97b2ca8d3" id="fn-5-60e71cc97b2ca8d3" dir="auto"><p>Jørgensen T, Haile J, Möller P, Andreev A, Boessenkool S, Rasmussen M, Kienast F, Coissac E, Taberlet P, Brochmann C, et al. A comparative study of ancient sedimentary DNA, pollen and macrofossils from permafrost sediments of northern Siberia reveals long-term vegetational stability. Mol Ecol. 2012:21(8): 1989–2003. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1111/j.1365-294X.2011.05287.x" target="_blank">https://doi.org/10.1111/j.1365-294X.2011.05287.x</a>.<a href="#fnref-5-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-10" data-footnote-id="fn-6-60e71cc97b2ca8d3" id="fn-6-60e71cc97b2ca8d3" dir="auto"><p>Graham RW, Belmecheri S, Choy K, Culleton BJ, Davies LJ, Froese D, Heintzman PD, Hritz C, Kapp JD, Newsom LA, et al. Timing and causes of mid-Holocene mammoth extinction on St. Paul Island, Alaska. Proc Natl Acad Sci USA. 2016:113(33):9310–9314. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1073/pnas.1604903113" target="_blank">https://doi.org/10.1073/pnas.1604903113</a>.<a href="#fnref-6-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-8" data-footnote-id="fn-7-60e71cc97b2ca8d3" id="fn-7-60e71cc97b2ca8d3" dir="auto"><p>Slon V, Clark JL, Friesem DE, Orbach M, Porat N, Meyer M, Kandel AW, Shimelmitz R. Extended longevity of DNA preservation in levantine paleolithic sediments, Sefunim cave, Israel. Sci Rep. 2022:12(1):14528. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1038/s41598-022-17399-2" target="_blank">https://doi.org/10.1038/s41598-022-17399-2</a>.<a href="#fnref-7-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-6" data-footnote-id="fn-8-60e71cc97b2ca8d3" id="fn-8-60e71cc97b2ca8d3" dir="auto"><p>Bender MA, Farach-Colton M, Pemmasani G, Skiena S, Sumazin P. Lowest common ancestors in trees and directed acyclic graphs. J Algo. 2005:57(2):75–94. <a data-tooltip-position="top" aria-label="https://doi.org/10.1016/j.jalgor.2005.08.001" rel="noopener nofollow" class="external-link" href="https://doi.org/10.1016/j.jalgor.2005.08.001" target="_blank">https://doi.org/10.1016/j.jalgor.2005. 08.001</a>.<a href="#fnref-8-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-4" data-footnote-id="fn-9-60e71cc97b2ca8d3" id="fn-9-60e71cc97b2ca8d3" dir="auto"><p>Vogel NA, Rubin JD, Swartz M, Vlieghe J, Sackett PW, Pedersen AG, Pedersen MW, Renaud G. euka: robust tetrapodic and arthropodic taxa detection from modern and ancient environmental DNA using pangenomic reference graphs. Methods Ecol Evol. 2023:14(11):2717–2727. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1111/2041-210X.14214" target="_blank">https://doi.org/10.1111/2041-210X.14214</a>.<a href="#fnref-9-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-2" data-footnote-id="fn-10-60e71cc97b2ca8d3" id="fn-10-60e71cc97b2ca8d3" dir="auto"><p>Dimopoulos EA, Carmagnini A, Velsko IM, Warinner C, Larson G, Frantz LA, Irving-Pease EK. Haystac: a Bayesian framework for robust and rapid species identification in high-throughput sequencing data. PLoS Comput Biol. 2022:18(9):e1010493. <a data-tooltip-position="top" aria-label="https://doi.org/10.1371/journal.pcbi.1010493" rel="noopener nofollow" class="external-link" href="https://doi.org/10.1371/journal.pcbi.1010493" target="_blank">https://doi. org/10.1371/journal.pcbi.1010493</a>.<a href="#fnref-10-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="0" data-footnote-id="fn-11-60e71cc97b2ca8d3" id="fn-11-60e71cc97b2ca8d3" dir="auto"><p>Martiniano R, De Sanctis B, Hallast P, Durbin R. Placing ancient DNA sequences into reference phylogenies. Mol Biol Evol. 2022:39(2): msac017. <a rel="noopener nofollow" class="external-link" href="https://doi.org/10.1093/molbev/msac017" target="_blank">https://doi.org/10.1093/molbev/msac017</a>.<a href="#fnref-11-60e71cc97b2ca8d3" class="footnote-backref footnote-link" target="_self" rel="noopener nofollow"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="18px" style="max-width: 100%;"></span>︎</a></p></li>
</ol></section></div><div class="mod-footer mod-ui"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#摘要"><div class="tree-item-contents heading-link" heading-name="摘要"><span class="tree-item-title">摘要</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#介绍"><div class="tree-item-contents heading-link" heading-name="介绍"><span class="tree-item-title">介绍</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#研究背景"><div class="tree-item-contents heading-link" heading-name="研究背景"><span class="tree-item-title">研究背景</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#科学问题"><div class="tree-item-contents heading-link" heading-name="科学问题"><span class="tree-item-title">科学问题</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#研究结果"><div class="tree-item-contents heading-link" heading-name="研究结果"><span class="tree-item-title">研究结果</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#图表"><div class="tree-item-contents heading-link" heading-name="图表"><span class="tree-item-title">图表</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\2024-mbe-soibean：利用线粒体泛基因组图对古代环境-dna-进行高分辨率分类鉴定.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>