<!DOCTYPE html> <html><head>
		<title>BMC Bioinformatics 用于估算线粒体 DNA 变异的全球多样化参考比对和面板</title>
		<base href="..\../">
		<meta id="root-path" root-path="..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="科研思路 - BMC Bioinformatics 用于估算线粒体 DNA 变异的全球多样化参考比对和面板">
		<meta property="og:title" content="BMC Bioinformatics 用于估算线粒体 DNA 变异的全球多样化参考比对和面板">
		<meta property="og:description" content="科研思路 - BMC Bioinformatics 用于估算线粒体 DNA 变异的全球多样化参考比对和面板">
		<meta property="og:type" content="website">
		<meta property="og:url" content="文献及报道/文献/bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html">
		<meta property="og:image" content="undefined">
		<meta property="og:site_name" content="科研思路">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon h2-underline"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-c.mjx-c1D449.TEX-I::before{padding:.683em .769em .022em 0;content:"V"}mjx-c.mjx-c1D43C.TEX-I::before{padding:.683em .504em 0 0;content:"I"}mjx-c.mjx-c2212::before{padding:.583em .778em .082em 0;content:"−"}mjx-c.mjx-c36::before{padding:.666em .5em .022em 0;content:"6"}mjx-c.mjx-c33::before{padding:.665em .5em .022em 0;content:"3"}mjx-c.mjx-c37::before{padding:.676em .5em .022em 0;content:"7"}mjx-c.mjx-c3E::before{padding:.54em .778em .04em 0;content:">"}mjx-c.mjx-c1D44D.TEX-I::before{padding:.683em .723em 0 0;content:"Z"}mjx-c.mjx-c3C::before{padding:.54em .778em .04em 0;content:"<"}mjx-c.mjx-c35::before{padding:.666em .5em .022em 0;content:"5"}mjx-texatom{display:inline-block;text-align:left}mjx-c.mjx-c5B::before{padding:.75em .278em .25em 0;content:"["}mjx-c.mjx-c30::before{padding:.666em .5em .022em 0;content:"0"}mjx-c.mjx-c2C::before{padding:.121em .278em .194em 0;content:","}mjx-c.mjx-c5D::before{padding:.75em .278em .25em 0;content:"]"}mjx-c.mjx-c1D453.TEX-I::before{padding:.705em .55em .205em 0;content:"f"}mjx-c.mjx-c2035.TEX-A::before{padding:.56em .275em 0 0;content:"‵"}mjx-c.mjx-c1D45A.TEX-I::before{padding:.442em .878em .011em 0;content:"m"}mjx-c.mjx-c1D45D.TEX-I::before{padding:.442em .503em .194em 0;content:"p"}mjx-c.mjx-c2E::before{padding:.12em .278em 0 0;content:"."}mjx-c.mjx-c1D465.TEX-I::before{padding:.442em .572em .011em 0;content:"x"}mjx-msup{display:inline-block;text-align:left}mjx-c.mjx-c1D445.TEX-I::before{padding:.683em .759em .021em 0;content:"R"}mjx-c.mjx-c32::before{padding:.666em .5em 0 0;content:"2"}mjx-c.mjx-c1D440.TEX-I::before{padding:.683em 1.051em 0 0;content:"M"}mjx-c.mjx-c39::before{padding:.666em .5em .022em 0;content:"9"}mjx-c.mjx-c2B::before{padding:.583em .778em .082em 0;content:"+"}mjx-c.mjx-c2192::before{padding:.511em 1em .011em 0;content:"→"}mjx-c.mjx-c1D443.TEX-I::before{padding:.683em .751em 0 0;content:"P"}mjx-c.mjx-c1D437.TEX-I::before{padding:.683em .828em 0 0;content:"D"}mjx-c.mjx-c1D442.TEX-I::before{padding:.704em .763em .022em 0;content:"O"}mjx-c.mjx-c1D435.TEX-I::before{padding:.683em .759em 0 0;content:"B"}mjx-c.mjx-c1D44E.TEX-I::before{padding:.441em .529em .01em 0;content:"a"}mjx-c.mjx-c1D456.TEX-I::before{padding:.661em .345em .011em 0;content:"i"}mjx-c.mjx-c5F::before{padding:0 .5em .062em 0;content:"_"}mjx-c.mjx-c1D43A.TEX-I::before{padding:.705em .786em .022em 0;content:"G"}mjx-c.mjx-c1D467.TEX-I::before{padding:.442em .465em .011em 0;content:"z"}mjx-c.mjx-c210E.TEX-I::before{padding:.694em .576em .011em 0;content:"h"}mjx-c.mjx-c1D45C.TEX-I::before{padding:.441em .485em .011em 0;content:"o"}mjx-container[jax=CHTML]{line-height:0}mjx-container [space="1"]{margin-left:.111em}mjx-container [space="2"]{margin-left:.167em}mjx-container [space="3"]{margin-left:.222em}mjx-container [space="4"]{margin-left:.278em}mjx-container [space="5"]{margin-left:.333em}mjx-container [rspace="1"]{margin-right:.111em}mjx-container [rspace="2"]{margin-right:.167em}mjx-container [rspace="3"]{margin-right:.222em}mjx-container [rspace="4"]{margin-right:.278em}mjx-container [rspace="5"]{margin-right:.333em}mjx-container [size="s"]{font-size:70.7%}mjx-container [size=ss]{font-size:50%}mjx-container [size=Tn]{font-size:60%}mjx-container [size=sm]{font-size:85%}mjx-container [size=lg]{font-size:120%}mjx-container [size=Lg]{font-size:144%}mjx-container [size=LG]{font-size:173%}mjx-container [size=hg]{font-size:207%}mjx-container [size=HG]{font-size:249%}mjx-container [width=full]{width:100%}mjx-box{display:inline-block}mjx-block{display:block}mjx-itable{display:inline-table}mjx-row{display:table-row}mjx-row>*{display:table-cell}mjx-mtext{display:inline-block}mjx-mstyle{display:inline-block}mjx-merror{display:inline-block;color:red;background-color:#ff0}mjx-mphantom{visibility:hidden}mjx-assistive-mml{top:0;left:0;clip:rect(1px,1px,1px,1px);user-select:none;position:absolute!important;padding:1px 0 0!important;border:0!important;display:block!important;width:auto!important;overflow:hidden!important}mjx-assistive-mml[display=block]{width:100%!important}mjx-math{display:inline-block;text-align:left;line-height:0;text-indent:0;font-style:normal;font-weight:400;font-size:100%;letter-spacing:normal;border-collapse:collapse;overflow-wrap:normal;word-spacing:normal;white-space:nowrap;direction:ltr;padding:1px 0}mjx-container[jax=CHTML][display=true]{display:block;text-align:center;margin:1em 0}mjx-container[jax=CHTML][display=true][width=full]{display:flex}mjx-container[jax=CHTML][display=true] mjx-math{padding:0}mjx-container[jax=CHTML][justify=left]{text-align:left}mjx-container[jax=CHTML][justify=right]{text-align:right}mjx-mi{display:inline-block;text-align:left}mjx-c{display:inline-block}mjx-utext{display:inline-block;padding:.75em 0 .2em}mjx-mo{display:inline-block;text-align:left}mjx-stretchy-h{display:inline-table;width:100%}mjx-stretchy-h>*{display:table-cell;width:0}mjx-stretchy-h>*>mjx-c{display:inline-block;transform:scaleX(1)}mjx-stretchy-h>*>mjx-c::before{display:inline-block;width:initial}mjx-stretchy-h>mjx-ext{overflow:clip visible;width:100%}mjx-stretchy-h>mjx-ext>mjx-c::before{transform:scaleX(500)}mjx-stretchy-h>mjx-ext>mjx-c{width:0}mjx-stretchy-h>mjx-beg>mjx-c{margin-right:-.1em}mjx-stretchy-h>mjx-end>mjx-c{margin-left:-.1em}mjx-stretchy-v{display:inline-block}mjx-stretchy-v>*{display:block}mjx-stretchy-v>mjx-beg{height:0}mjx-stretchy-v>mjx-end>mjx-c{display:block}mjx-stretchy-v>*>mjx-c{transform:scaleY(1);transform-origin:left center;overflow:hidden}mjx-stretchy-v>mjx-ext{display:block;height:100%;box-sizing:border-box;border:0 solid transparent;overflow:visible clip}mjx-stretchy-v>mjx-ext>mjx-c::before{width:initial;box-sizing:border-box}mjx-stretchy-v>mjx-ext>mjx-c{transform:scaleY(500) translateY(.075em);overflow:visible}mjx-mark{display:inline-block;height:0}mjx-mn{display:inline-block;text-align:left}mjx-mfrac{display:inline-block;text-align:left}mjx-frac{display:inline-block;vertical-align:.17em;padding:0 .22em}mjx-frac[type="d"]{vertical-align:.04em}mjx-frac[delims]{padding:0 .1em}mjx-frac[atop]{padding:0 .12em}mjx-frac[atop][delims]{padding:0}mjx-dtable{display:inline-table;width:100%}mjx-dtable>*{font-size:2000%}mjx-dbox{display:block;font-size:5%}mjx-num{display:block;text-align:center}mjx-den{display:block;text-align:center}mjx-mfrac[bevelled]>mjx-num{display:inline-block}mjx-mfrac[bevelled]>mjx-den{display:inline-block}mjx-den[align=right],mjx-num[align=right]{text-align:right}mjx-den[align=left],mjx-num[align=left]{text-align:left}mjx-nstrut{display:inline-block;height:.054em;width:0;vertical-align:-.054em}mjx-nstrut[type="d"]{height:.217em;vertical-align:-.217em}mjx-dstrut{display:inline-block;height:.505em;width:0}mjx-dstrut[type="d"]{height:.726em}mjx-line{display:block;box-sizing:border-box;min-height:1px;height:.06em;border-top:.06em solid;margin:.06em -.1em;overflow:hidden}mjx-line[type="d"]{margin:.18em -.1em}mjx-mrow{display:inline-block;text-align:left}mjx-c::before{display:block;width:0}.MJX-TEX{font-family:MJXZERO,MJXTEX}.TEX-B{font-family:MJXZERO,MJXTEX-B}.TEX-I{font-family:MJXZERO,MJXTEX-I}.TEX-MI{font-family:MJXZERO,MJXTEX-MI}.TEX-BI{font-family:MJXZERO,MJXTEX-BI}.TEX-S1{font-family:MJXZERO,MJXTEX-S1}.TEX-S2{font-family:MJXZERO,MJXTEX-S2}.TEX-S3{font-family:MJXZERO,MJXTEX-S3}.TEX-S4{font-family:MJXZERO,MJXTEX-S4}.TEX-A{font-family:MJXZERO,MJXTEX-A}.TEX-C{font-family:MJXZERO,MJXTEX-C}.TEX-CB{font-family:MJXZERO,MJXTEX-CB}.TEX-FR{font-family:MJXZERO,MJXTEX-FR}.TEX-FRB{font-family:MJXZERO,MJXTEX-FRB}.TEX-SS{font-family:MJXZERO,MJXTEX-SS}.TEX-SSB{font-family:MJXZERO,MJXTEX-SSB}.TEX-SSI{font-family:MJXZERO,MJXTEX-SSI}.TEX-SC{font-family:MJXZERO,MJXTEX-SC}.TEX-T{font-family:MJXZERO,MJXTEX-T}.TEX-V{font-family:MJXZERO,MJXTEX-V}.TEX-VB{font-family:MJXZERO,MJXTEX-VB}mjx-stretchy-h mjx-c,mjx-stretchy-v mjx-c{font-family:MJXZERO,MJXTEX-S1,MJXTEX-S4,MJXTEX,MJXTEX-A!important}@font-face{font-family:MJXZERO;src:url("lib/fonts/mathjax_zero.woff") format("woff")}@font-face{font-family:MJXTEX;src:url("lib/fonts/mathjax_main-regular.woff") format("woff")}@font-face{font-family:MJXTEX-B;src:url("lib/fonts/mathjax_main-bold.woff") format("woff")}@font-face{font-family:MJXTEX-I;src:url("lib/fonts/mathjax_math-italic.woff") format("woff")}@font-face{font-family:MJXTEX-MI;src:url("lib/fonts/mathjax_main-italic.woff") format("woff")}@font-face{font-family:MJXTEX-BI;src:url("lib/fonts/mathjax_math-bolditalic.woff") format("woff")}@font-face{font-family:MJXTEX-S1;src:url("lib/fonts/mathjax_size1-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S2;src:url("lib/fonts/mathjax_size2-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S3;src:url("lib/fonts/mathjax_size3-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S4;src:url("lib/fonts/mathjax_size4-regular.woff") format("woff")}@font-face{font-family:MJXTEX-A;src:url("lib/fonts/mathjax_ams-regular.woff") format("woff")}@font-face{font-family:MJXTEX-C;src:url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff")}@font-face{font-family:MJXTEX-CB;src:url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff")}@font-face{font-family:MJXTEX-FR;src:url("lib/fonts/mathjax_fraktur-regular.woff") format("woff")}@font-face{font-family:MJXTEX-FRB;src:url("lib/fonts/mathjax_fraktur-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SS;src:url("lib/fonts/mathjax_sansserif-regular.woff") format("woff")}@font-face{font-family:MJXTEX-SSB;src:url("lib/fonts/mathjax_sansserif-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SSI;src:url("lib/fonts/mathjax_sansserif-italic.woff") format("woff")}@font-face{font-family:MJXTEX-SC;src:url("lib/fonts/mathjax_script-regular.woff") format("woff")}@font-face{font-family:MJXTEX-T;src:url("lib/fonts/mathjax_typewriter-regular.woff") format("woff")}@font-face{font-family:MJXTEX-V;src:url("lib/fonts/mathjax_vector-regular.woff") format("woff")}@font-face{font-family:MJXTEX-VB;src:url("lib/fonts/mathjax_vector-bold.woff") format("woff")}mjx-c.mjx-c1D439.TEX-I::before{padding:.68em .749em 0 0;content:"F"}mjx-c.mjx-c1D45F.TEX-I::before{padding:.442em .451em .011em 0;content:"r"}mjx-c.mjx-c1D452.TEX-I::before{padding:.442em .466em .011em 0;content:"e"}mjx-c.mjx-c1D45E.TEX-I::before{padding:.442em .46em .194em 0;content:"q"}mjx-c.mjx-c1D462.TEX-I::before{padding:.442em .572em .011em 0;content:"u"}mjx-c.mjx-c1D45B.TEX-I::before{padding:.442em .6em .011em 0;content:"n"}mjx-c.mjx-c1D450.TEX-I::before{padding:.442em .433em .011em 0;content:"c"}mjx-c.mjx-c1D466.TEX-I::before{padding:.442em .49em .205em 0;content:"y"}mjx-c.mjx-c28::before{padding:.75em .389em .25em 0;content:"("}mjx-c.mjx-c1D436.TEX-I::before{padding:.705em .76em .022em 0;content:"C"}mjx-c.mjx-c31::before{padding:.666em .5em 0 0;content:"1"}mjx-c.mjx-c29::before{padding:.75em .389em .25em 0;content:")"}mjx-c.mjx-c3D::before{padding:.583em .778em .082em 0;content:"="}</style><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="摘要"><p dir="auto">摘要</p></h1><div><blockquote dir="auto">
<p dir="auto">A globally diverse reference alignment and&nbsp;panel for&nbsp;imputation of&nbsp;mitochondrial DNA variants</p>
</blockquote></div><div class="heading-wrapper"><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="背景" dir="auto" class="heading" id="背景"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>背景</h2><div class="heading-children"><div><p dir="auto">通过<strong>基因分型微阵列</strong>或<strong>仅对基因组高变区</strong>进行测序来鉴定线粒体 DNA (mtDNA) 的变异可能<strong>不足以可靠地将线粒体基因组分配给系统发育谱系或单倍群</strong>。这种分辨率的缺乏可能会限制对大量现有 mtDNA 数据的功能和临床解释。为了解决这一限制，我们开发并评估了完整 mtDNA 序列的大型、精心策划的参考比对，作为估算缺失 mtDNA 单核苷酸变异 (mtSNV) 的管道的一部分。我们将参考对齐和管道称为 MitoImpute。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="结果" dir="auto" class="heading" id="结果"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>结果</h2><div class="heading-children"><div><p dir="auto">我们对从 GenBank 下载的 36,960 个完整人类线粒体基因组的序列进行了比对，并进行了过滤和质量控制。这些序列被重新格式化以用于插补软件 IMPUTE2。我们通过测量 1000 个基因组计划和阿尔茨海默病神经影像计划 (ADNI) 数据中的单倍群和基因型一致性来评估 MitoImpute 的插补准确性。 1000 个基因组样本中单倍群分配的平均改善率为 42.7%（马修相关系数 = 0.64）。在 ADNI 队列中，我们估算了缺失的单核苷酸变异。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="结论" dir="auto" class="heading" id="结论"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>结论</h2><div class="heading-children"><div><p dir="auto">这些结果表明，我们的参考比对和面板可用于估算使用微阵列获得的现有数据中缺失的 mtSNV，从而扩大 mtDNA 的功能和临床研究范围。这种改进可能在随着时间的推移招募参与者并使用不同方法获得 mtDNA 数据的研究中特别有用，从而能够更好地将使用不太准确的方法收集的早期数据与较新的序列数据整合起来。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="介绍" dir="auto" class="heading" id="介绍">介绍</h1><div class="heading-children"><div><p dir="auto">“Partial mtSNV data obtained using such arrays may be insufficient for reliable haplotype assignment of mitochondrial genomes.” (McInerney 等, 2021, p. 2) 使用此类阵列获得的部分 mtSNV 数据可能不足以对线粒体基因组进行可靠的单倍型分配。</p></div><div><p dir="auto">“In addition, not all microarrays are designed to assay variation at the same sites in the human mitochondrial genome. Inconsistencies in the design of microarrays used in different studies can result in mtSNV datasets that are partially incompatible, making it difficult to combine them for joint analysis.” (McInerney 等, 2021, p. 2) 此外，并非所有微阵列都旨在检测人类线粒体基因组中相同位点的变异。<strong>不同研究中使用的微阵列设计的不一致可能会导致 mtSNV 数据集部分不兼容，从而很难将它们组合起来进行联合分析</strong>。</p></div><div><p dir="auto">“Additionally, studies as recent as 2021 have used imputation as a tool to fill in missing mtSNVs for genotypephenotype association studies , highlighting the continued importance imputation plays in biomedical research.” (McInerney 等, 2021, p. 2) 此外，<strong>最近在 2021 年的研究已使用插补作为工具来填补基因型表型关联研究中缺失的 mtSNV <sup data-footnote-id="fnref-1-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-1-8d2dbafc70cf3b8c"><a data-footref="1" href="#fn-1-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[1]</a></sup>，这突显了插补在生物医学研究中持续发挥的重要作用</strong>。</p></div><div><p dir="auto">“Accurate estimation of these probabilities has two fundamental requirements: (1) An accurate multiple sequence alignment (MSA) of genome sequences; and (2) A reference panel of genome sequences that is representative of the population being investigated.” (McInerney 等, 2021, p. 2) 准确估计这些概率有两个基本要求：</p></div><div><ol dir="auto">
<li data-line="0" dir="auto">基因组序列的<strong>准确多序列比对（MSA）</strong></li>
<li data-line="1" dir="auto">有<strong>代表的研究人群的参考基因组序列</strong></li>
</ol></div><div><p dir="auto">“Data that is unrepresentative because it was obtained from an inappropriate population, can cause imputation to be biased and inaccurate” (McInerney 等, 2021, p. 2) 由于数据是从不适当的人群中获得的，因此不具有代表性，可能会导致估算出现偏差和不准确<sup data-footnote-id="fnref-2-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-2-8d2dbafc70cf3b8c"><a data-footref="2" href="#fn-2-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[2]</a></sup> <sup data-footnote-id="fnref-3-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-3-8d2dbafc70cf3b8c"><a data-footref="3" href="#fn-3-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[3]</a></sup> <sup data-footnote-id="fnref-4-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-4-8d2dbafc70cf3b8c"><a data-footref="4" href="#fn-4-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[4]</a></sup> <sup data-footnote-id="fnref-5-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-5-8d2dbafc70cf3b8c"><a data-footref="5" href="#fn-5-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[5]</a></sup> <sup data-footnote-id="fnref-6-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-6-8d2dbafc70cf3b8c"><a data-footref="6" href="#fn-6-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[6]</a></sup> <sup data-footnote-id="fnref-7-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-7-8d2dbafc70cf3b8c"><a data-footref="7" href="#fn-7-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[7]</a></sup>。</p></div><div><p dir="auto">“Additional bias and inaccuracy may arise during construction of MSAs, which entails inserting alignment gaps (‘–’) between some of the nucleotides in some of the sequences being aligned—doing so accurately is a nontrivial challenge” (McInerney 等, 2021, p. 2) 在构建 MSA 过程中可能会出现额外的偏差和不准确性，这需要在一些正在比对的序列中的一些核苷酸之间插入比对间隙（“-”）——准确地做到这一点是一个不小的挑战。<br>
“reference panels containing high-quality complete data from as many globally diverse populations is required.” (McInerney 等, 2021, p. 3) <strong>需要包含来自尽可能多的全球不同人群的高质量完整数据的参考面板</strong>。</p></div><div><p dir="auto">“preliminary data reformatting, identification and curation of suitable reference data panels, and standardisation of high-quality multisequence alignments.” (McInerney 等, 2021, p. 3) 必须实现的基础是：</p></div><div><ol dir="auto">
<li data-line="0" dir="auto">初步数据重新<strong>格式化</strong></li>
<li data-line="1" dir="auto">合适<strong>参考数据</strong>组的识别和管理</li>
<li data-line="2" dir="auto">高质量<strong>多序列比对的标准化</strong></li>
</ol></div></div></div><div class="heading-wrapper"><h1 data-heading="方法" dir="auto" class="heading" id="方法">方法</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="填补" dir="auto" class="heading" id="填补"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>填补</h2><div class="heading-children"><div><p dir="auto">“IMPUTE2 information score of ≤ 0.3.” (McInerney 等, 2021, p. 5)：(McInerney 等, 2021, p. 5) 信息评分≤0.3的位点在遗传学数据分析中，<strong>信息评分（Info Score 或 IMPUTE2信息评分）是一种用来衡量基因型插补质量的指标</strong>。IMPUTE2软件在进行基因型插补后会输出这一评分，该评分范围通常<strong>在0到1之间。评分越接近1，表明插补的置信度越高，即该位点的插补结果越可靠</strong>。相反，评分较低（如≤0.3）的位点表明插补的不确定性较高，因此在某些研究中可能会选择移除这些质量较低的插补结果。具体计算方法涉及对每个插补位点的估计等位基因频率的置信度进行评估。在 IMPUTE2中，这通常通过<strong>对比插补得到的等位基因频率的预测分布与实际观察值</strong>来进行。</p></div><div><p dir="auto">“HaploGrep2 has the advantage of covering the full scope of the PhyloTree haplogroup nomenclature” (McInerney 等, 2021, p. 5) <strong>HaploGrep2 的优势在于涵盖了 PhyloTree 单倍群命名法的全部范围</strong> <sup data-footnote-id="fnref-8-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-8-8d2dbafc70cf3b8c"><a data-footref="8" href="#fn-8-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[8]</a></sup>。<br>
“Hi-MC was developed for epidemiological research that uses high-throughput data by reducing PhyloTree nomenclature to 46 common haplogroups using a limited array of mtSNVs from which to assign haplogroups.” (McInerney 等, 2021, p. 5) Hi-MC 是为流行病学研究而开发的，该研究使用高通量数据，通过使用<strong>有限的 mtSNV 阵列将 PhyloTree 命名法减少到 46 个常见单倍群，从中分配单倍群</strong> <sup data-footnote-id="fnref-9-8d2dbafc70cf3b8c" class="footnote-ref" id="fnref-9-8d2dbafc70cf3b8c"><a data-footref="9" href="#fn-9-8d2dbafc70cf3b8c" class="footnote-link" target="_self" rel="noopener">[9]</a></sup>。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="讨论" dir="auto" class="heading" id="讨论">讨论</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="局限性" dir="auto" class="heading" id="局限性"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>局限性</h2><div class="heading-children"><div><p dir="auto">“Performance testing of the MitoImpute pipeline using microarray datasets revealed a seemingly counterintuitive result; the decrease in imputation accuracy as the khap parameter increases.” (McInerney 等, 2021, p. 11) 使用微阵列数据集对 MitoImpute 管道进行性能测试揭示了看似违反直觉的结果；随着 khap 参数的增加，插补精度会降低。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div>
</div><div><section class="footnotes"><hr dir="auto"><ol dir="auto">
<li data-line="-8" data-footnote-id="fn-1-8d2dbafc70cf3b8c" id="fn-1-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Yonova‑Doing E, Calabrese C, Gomez‑Duran A, Schon K, Wei W, Karthikeyan S, et al. An atlas of mitochondrial DNA genotype–phenotype associations in the UK Biobank. Nat Genet. 2021.<a href="#fnref-1-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-7" data-footnote-id="fn-2-8d2dbafc70cf3b8c" id="fn-2-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Huang J, Howie B, McCarthy S, Memari Y, Walter K, Min JL, et al. Improved imputation of low‑frequency and rare variants using the UK10K haplotype reference panel. Nat Commun. 2015; 6:8111.<a href="#fnref-2-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-6" data-footnote-id="fn-3-8d2dbafc70cf3b8c" id="fn-3-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Yoo S‑K, Kim C‑U, Kim HL, Kim S, Shin J‑Y, Kim N, et al. NARD: whole‑genome reference panel of 1779 Northeast Asians improves imputation accuracy of rare and low‑frequency variants. Genome Med. 2019;11(1):64<a href="#fnref-3-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-5" data-footnote-id="fn-4-8d2dbafc70cf3b8c" id="fn-4-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Sariya S, Lee JH, Mayeux R, Vardarajan BN, Reyes‑Dumeyer D, Manly JJ, et al. Rare variants imputation in admixed populations: comparison across reference panels and bioinformatics tools. Front Genet. 2019; 10:239.<a href="#fnref-4-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-4" data-footnote-id="fn-5-8d2dbafc70cf3b8c" id="fn-5-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Das S, Forer L, Schonherr S, Sidore C, Locke AE, Kwong A, et al. Next‑generation genotype imputation service and methods. Nat Genet. 2016;48(10):1284–7.<a href="#fnref-5-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-3" data-footnote-id="fn-6-8d2dbafc70cf3b8c" id="fn-6-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Zheng H‑F, Ladouceur M, Greenwood CMT, Richards JB. Effect of genome‑wide genotyping and reference panels on rare variants imputation. J Genet Genom. 2012;39(10):545–50.<a href="#fnref-6-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-2" data-footnote-id="fn-7-8d2dbafc70cf3b8c" id="fn-7-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype‑phase inference for large data sets of trios and unrelated individuals. Am J Hum Genet. 2009;84(2):210–23.<a href="#fnref-7-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="-1" data-footnote-id="fn-8-8d2dbafc70cf3b8c" id="fn-8-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Weissensteiner H, Pacher D, Kloss‑Brandstätter A, Forer L, Specht G, Bandelt H‑J, et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high‑throughput sequencing.<a href="#fnref-8-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
<li data-line="0" data-footnote-id="fn-9-8d2dbafc70cf3b8c" id="fn-9-8d2dbafc70cf3b8c" dir="auto"><p dir="auto">Smieszek S, Mitchell SL, Farber‑Eger EH, Veatch OJ, Wheeler NR, Goodloe RJ, et al. Hi‑MC: a novel method for highthroughput mitochondrial haplogroup classification. PeerJ. 2018;6:e5149.<a href="#fnref-9-8d2dbafc70cf3b8c" class="footnote-backref footnote-link" target="_self" rel="noopener">↩︎</a></p></li>
</ol></section></div><div class="mod-footer"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#摘要"><div class="tree-item-contents heading-link" heading-name="摘要"><span class="tree-item-title">摘要</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#背景"><div class="tree-item-contents heading-link" heading-name="背景"><span class="tree-item-title">背景</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#结果"><div class="tree-item-contents heading-link" heading-name="结果"><span class="tree-item-title">结果</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#结论"><div class="tree-item-contents heading-link" heading-name="结论"><span class="tree-item-title">结论</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#介绍"><div class="tree-item-contents heading-link" heading-name="介绍"><span class="tree-item-title">介绍</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#方法"><div class="tree-item-contents heading-link" heading-name="方法"><span class="tree-item-title">方法</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#填补"><div class="tree-item-contents heading-link" heading-name="填补"><span class="tree-item-title">填补</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#讨论"><div class="tree-item-contents heading-link" heading-name="讨论"><span class="tree-item-title">讨论</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#局限性"><div class="tree-item-contents heading-link" heading-name="局限性"><span class="tree-item-title">局限性</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\bmc-bioinformatics-用于估算线粒体-dna-变异的全球多样化参考比对和面板.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>