<html><head>
		<title>人类端粒到端粒参考基因组中线粒体假基因 (NUMT) 的全面鉴定</title>
		<base href="..\../">
		<meta id="root-path" root-path="..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 人类端粒到端粒参考基因组中线粒体假基因 (NUMT) 的全面鉴定">
		<meta property="og:title" content="人类端粒到端粒参考基因组中线粒体假基因 (NUMT) 的全面鉴定">
		<meta property="og:description" content="思维领域 - 人类端粒到端粒参考基因组中线粒体假基因 (NUMT) 的全面鉴定">
		<meta property="og:type" content="website">
		<meta property="og:url" content="文献及报道/文献/人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html">
		<meta property="og:image" content="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602093029.png">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon trim-cols h1-l h2-l h3-l h4-l h5-l h6-l image-blend-light active-line-on checkbox-square hide-help styled-scrollbars fast-animations"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-c.mjx-cB1::before{padding:.666em .778em 0 0;content:"±"}mjx-c.mjx-c221A.TEX-S3::before{padding:1.45em 1.02em .95em 0;content:"√"}mjx-c.mjx-c1D444.TEX-I::before{padding:.704em .791em .194em 0;content:"Q"}mjx-c.mjx-c1D454.TEX-I::before{padding:.442em .477em .205em 0;content:"g"}mjx-c.mjx-cD7::before{padding:.491em .778em 0 0;content:"×"}mjx-c.mjx-c1D461.TEX-I::before{padding:.626em .361em .011em 0;content:"t"}mjx-c.mjx-c1D703.TEX-I::before{padding:.705em .469em .01em 0;content:"θ"}mjx-c.mjx-c393::before{padding:.68em .625em 0 0;content:"Γ"}mjx-c.mjx-c74::before{padding:.615em .389em .01em 0;content:"t"}mjx-c.mjx-c6F::before{padding:.448em .5em .01em 0;content:"o"}mjx-c.mjx-c61::before{padding:.448em .5em .011em 0;content:"a"}mjx-c.mjx-c6C::before{padding:.694em .278em 0 0;content:"l"}mjx-c.mjx-c1D441.TEX-I::before{padding:.683em .888em 0 0;content:"N"}mjx-c.mjx-c1D457.TEX-I::before{padding:.661em .412em .204em 0;content:"j"}mjx-c.mjx-c1D451.TEX-I::before{padding:.694em .52em .01em 0;content:"d"}mjx-c.mjx-c1D458.TEX-I::before{padding:.694em .521em .011em 0;content:"k"}mjx-c.mjx-c1D459.TEX-I::before{padding:.694em .298em .011em 0;content:"l"}mjx-c.mjx-c22C5::before{padding:.31em .278em 0 0;content:"⋅"}mjx-c.mjx-c2013::before{padding:.285em .5em 0 0;content:"–"}mjx-c.mjx-c2026::before{padding:.12em 1.172em 0 0;content:"…"}mjx-c.mjx-c1D43E.TEX-I::before{padding:.683em .889em 0 0;content:"K"}mjx-c.mjx-c1D6FC.TEX-I::before{padding:.442em .64em .011em 0;content:"α"}mjx-c.mjx-c1D736.TEX-BI::before{padding:.452em .761em .008em 0;content:"α"}mjx-c.mjx-c1D446.TEX-I::before{padding:.705em .645em .022em 0;content:"S"}mjx-c.mjx-c1D44C.TEX-I::before{padding:.683em .763em 0 0;content:"Y"}mjx-mover{display:inline-block;text-align:left}mjx-mover:not([limits=false]){padding-top:.1em}mjx-mover:not([limits=false])>*{display:block;text-align:left}mjx-munder{display:inline-block;text-align:left}mjx-over{text-align:left}mjx-munder:not([limits=false]){display:inline-table}mjx-munder>mjx-row{text-align:left}mjx-under{padding-bottom:.1em}mjx-msqrt{display:inline-block;text-align:left}mjx-root{display:inline-block;white-space:nowrap}mjx-surd{display:inline-block;vertical-align:top}mjx-sqrt{display:inline-block;padding-top:.07em}mjx-sqrt>mjx-box{border-top:.07em solid}mjx-sqrt.mjx-tall>mjx-box{padding-left:.3em;margin-left:-.3em}.mjx-stretched mjx-c.mjx-c2013::before{content:"–"}mjx-stretchy-h.mjx-c2013 mjx-ext mjx-c::before{content:"–";padding:.285em 0 0}mjx-c.mjx-c1D438.TEX-I::before{padding:.68em .764em 0 0;content:"E"}mjx-c.mjx-c1D460.TEX-I::before{padding:.442em .469em .01em 0;content:"s"}mjx-c.mjx-c221A::before{padding:.8em .853em .2em 0;content:"√"}mjx-c.mjx-c2260::before{padding:.716em .778em .215em 0;content:"≠"}mjx-c.mjx-c1D434.TEX-I::before{padding:.716em .75em 0 0;content:"A"}mjx-msub{display:inline-block;text-align:left}mjx-munderover{display:inline-block;text-align:left}mjx-munderover:not([limits=false]){padding-top:.1em}mjx-munderover:not([limits=false])>*{display:block}mjx-msubsup{display:inline-block;text-align:left}mjx-script{display:inline-block;padding-right:.05em;padding-left:.033em}mjx-script>mjx-spacer{display:block}mjx-mtable{display:inline-block;text-align:center;vertical-align:.25em;position:relative;box-sizing:border-box;border-spacing:0px;border-collapse:collapse}mjx-mstyle[size="s"] mjx-mtable{vertical-align:.354em}mjx-labels{position:absolute;left:0;top:0}mjx-table{display:inline-block;vertical-align:-.5ex;box-sizing:border-box}mjx-table>mjx-itable{vertical-align:middle;text-align:left;box-sizing:border-box}mjx-labels>mjx-itable{position:absolute;top:0}mjx-mtable[justify=left]{text-align:left}mjx-mtable[justify=right]{text-align:right}mjx-mtable[justify=left][side=left]{padding-right:0!important}mjx-mtable[justify=left][side=right]{padding-left:0!important}mjx-mtable[justify=right][side=left]{padding-right:0!important}mjx-mtable[justify=right][side=right]{padding-left:0!important}mjx-mtable[align]{vertical-align:baseline}mjx-mtable[align=top]>mjx-table{vertical-align:top}mjx-mtable[align=bottom]>mjx-table{vertical-align:bottom}mjx-mtable[side=right] mjx-labels{min-width:100%}mjx-mtr{display:table-row;text-align:left}mjx-mtr[rowalign=top]>mjx-mtd{vertical-align:top}mjx-mtr[rowalign=center]>mjx-mtd{vertical-align:middle}mjx-mtr[rowalign=bottom]>mjx-mtd{vertical-align:bottom}mjx-mtr[rowalign=baseline]>mjx-mtd{vertical-align:baseline}mjx-mtr[rowalign=axis]>mjx-mtd{vertical-align:.25em}mjx-mtd{display:table-cell;text-align:center;padding:.215em .4em}mjx-mtd:first-child{padding-left:0}mjx-mtd:last-child{padding-right:0}mjx-mtable>*>mjx-itable>:first-child>mjx-mtd{padding-top:0}mjx-mtable>*>mjx-itable>:last-child>mjx-mtd{padding-bottom:0}mjx-tstrut{display:inline-block;height:1em;vertical-align:-.25em}mjx-labels[align=left]>mjx-mtr>mjx-mtd{text-align:left}mjx-labels[align=right]>mjx-mtr>mjx-mtd{text-align:right}mjx-mtd[extra]{padding:0}mjx-mtd[rowalign=top]{vertical-align:top}mjx-mtd[rowalign=center]{vertical-align:middle}mjx-mtd[rowalign=bottom]{vertical-align:bottom}mjx-mtd[rowalign=baseline]{vertical-align:baseline}mjx-mtd[rowalign=axis]{vertical-align:.25em}mjx-mtext{display:inline-block;text-align:left}mjx-mspace{display:inline-block;text-align:left}mjx-c.mjx-c1D44A.TEX-I::before{padding:.683em 1.048em .022em 0;content:"W"}mjx-c.mjx-c2211.TEX-S2::before{padding:.95em 1.444em .45em 0;content:"∑"}mjx-c.mjx-c3A::before{padding:.43em .278em 0 0;content:":"}mjx-c.mjx-c2219::before{padding:.444em .5em 0 0;content:"∙"}mjx-c.mjx-c58::before{padding:.683em .75em 0 0;content:"X"}mjx-c.mjx-c57::before{padding:.683em 1.028em .022em 0;content:"W"}mjx-c.mjx-cB7::before{padding:.31em .278em 0 0;content:"⋅"}mjx-c.mjx-c2F::before{padding:.75em .5em .25em 0;content:"/"}mjx-c.mjx-c22C6::before{padding:.486em .5em 0 0;content:"⋆"}mjx-c.mjx-c38::before{padding:.666em .5em .022em 0;content:"8"}mjx-c.mjx-c48::before{padding:.683em .75em 0 0;content:"H"}mjx-c.mjx-c77::before{padding:.431em .722em .011em 0;content:"w"}mjx-c.mjx-c78::before{padding:.431em .528em 0 0;content:"x"}mjx-c.mjx-c1D464.TEX-I::before{padding:.443em .716em .011em 0;content:"w"}mjx-c.mjx-c50::before{padding:.683em .681em 0 0;content:"P"}mjx-c.mjx-c59::before{padding:.683em .75em 0 0;content:"Y"}mjx-c.mjx-c7C::before{padding:.75em .278em .249em 0;content:"|"}mjx-c.mjx-c68::before{padding:.694em .556em 0 0;content:"h"}mjx-c.mjx-c65::before{padding:.448em .444em .011em 0;content:"e"}mjx-c.mjx-c54::before{padding:.677em .722em 0 0;content:"T"}mjx-c.mjx-c1D449.TEX-I::before{padding:.683em .769em .022em 0;content:"V"}mjx-c.mjx-c1D43C.TEX-I::before{padding:.683em .504em 0 0;content:"I"}mjx-c.mjx-c36::before{padding:.666em .5em .022em 0;content:"6"}mjx-c.mjx-c33::before{padding:.665em .5em .022em 0;content:"3"}mjx-c.mjx-c37::before{padding:.676em .5em .022em 0;content:"7"}mjx-c.mjx-c3E::before{padding:.54em .778em .04em 0;content:">"}mjx-c.mjx-c1D44D.TEX-I::before{padding:.683em .723em 0 0;content:"Z"}mjx-c.mjx-c35::before{padding:.666em .5em .022em 0;content:"5"}mjx-texatom{display:inline-block;text-align:left}mjx-c.mjx-c5B::before{padding:.75em .278em .25em 0;content:"["}mjx-c.mjx-c2C::before{padding:.121em .278em .194em 0;content:","}mjx-c.mjx-c5D::before{padding:.75em .278em .25em 0;content:"]"}mjx-c.mjx-c1D453.TEX-I::before{padding:.705em .55em .205em 0;content:"f"}mjx-c.mjx-c2035.TEX-A::before{padding:.56em .275em 0 0;content:"‵"}mjx-c.mjx-c1D45A.TEX-I::before{padding:.442em .878em .011em 0;content:"m"}mjx-c.mjx-c2E::before{padding:.12em .278em 0 0;content:"."}mjx-c.mjx-c1D465.TEX-I::before{padding:.442em .572em .011em 0;content:"x"}mjx-c.mjx-c1D445.TEX-I::before{padding:.683em .759em .021em 0;content:"R"}mjx-c.mjx-c1D440.TEX-I::before{padding:.683em 1.051em 0 0;content:"M"}mjx-c.mjx-c39::before{padding:.666em .5em .022em 0;content:"9"}mjx-c.mjx-c2B::before{padding:.583em .778em .082em 0;content:"+"}mjx-c.mjx-c2192::before{padding:.511em 1em .011em 0;content:"→"}mjx-c.mjx-c1D443.TEX-I::before{padding:.683em .751em 0 0;content:"P"}mjx-msup{display:inline-block;text-align:left}mjx-c.mjx-c1D43B.TEX-I::before{padding:.683em .888em 0 0;content:"H"}mjx-c.mjx-c1D437.TEX-I::before{padding:.683em .828em 0 0;content:"D"}mjx-c.mjx-c2212::before{padding:.583em .778em .082em 0;content:"−"}mjx-c.mjx-c2211.TEX-S1::before{padding:.75em 1.056em .25em 0;content:"∑"}mjx-c.mjx-c32::before{padding:.666em .5em 0 0;content:"2"}mjx-c.mjx-c1D442.TEX-I::before{padding:.704em .763em .022em 0;content:"O"}mjx-c.mjx-c1D435.TEX-I::before{padding:.683em .759em 0 0;content:"B"}mjx-c.mjx-c1D44E.TEX-I::before{padding:.441em .529em .01em 0;content:"a"}mjx-c.mjx-c1D456.TEX-I::before{padding:.661em .345em .011em 0;content:"i"}mjx-c.mjx-c5F::before{padding:0 .5em .062em 0;content:"_"}mjx-c.mjx-c1D43A.TEX-I::before{padding:.705em .786em .022em 0;content:"G"}mjx-c.mjx-c1D467.TEX-I::before{padding:.442em .465em .011em 0;content:"z"}mjx-c.mjx-c210E.TEX-I::before{padding:.694em .576em .011em 0;content:"h"}mjx-c.mjx-c1D45C.TEX-I::before{padding:.441em .485em .011em 0;content:"o"}mjx-mfrac{display:inline-block;text-align:left}mjx-frac{display:inline-block;vertical-align:.17em;padding:0 .22em}mjx-frac[type="d"]{vertical-align:.04em}mjx-frac[delims]{padding:0 .1em}mjx-frac[atop]{padding:0 .12em}mjx-frac[atop][delims]{padding:0}mjx-dtable{display:inline-table;width:100%}mjx-dtable>*{font-size:2000%}mjx-dbox{display:block;font-size:5%}mjx-num{display:block;text-align:center}mjx-den{display:block;text-align:center}mjx-mfrac[bevelled]>mjx-num{display:inline-block}mjx-mfrac[bevelled]>mjx-den{display:inline-block}mjx-den[align=right],mjx-num[align=right]{text-align:right}mjx-den[align=left],mjx-num[align=left]{text-align:left}mjx-nstrut{display:inline-block;height:.054em;width:0;vertical-align:-.054em}mjx-nstrut[type="d"]{height:.217em;vertical-align:-.217em}mjx-dstrut{display:inline-block;height:.505em;width:0}mjx-dstrut[type="d"]{height:.726em}mjx-line{display:block;box-sizing:border-box;min-height:1px;height:.06em;border-top:.06em solid;margin:.06em -.1em;overflow:hidden}mjx-line[type="d"]{margin:.18em -.1em}mjx-mrow{display:inline-block;text-align:left}mjx-c.mjx-c1D439.TEX-I::before{padding:.68em .749em 0 0;content:"F"}mjx-c.mjx-c1D45F.TEX-I::before{padding:.442em .451em .011em 0;content:"r"}mjx-c.mjx-c1D452.TEX-I::before{padding:.442em .466em .011em 0;content:"e"}mjx-c.mjx-c1D45E.TEX-I::before{padding:.442em .46em .194em 0;content:"q"}mjx-c.mjx-c1D462.TEX-I::before{padding:.442em .572em .011em 0;content:"u"}mjx-c.mjx-c1D45B.TEX-I::before{padding:.442em .6em .011em 0;content:"n"}mjx-c.mjx-c1D450.TEX-I::before{padding:.442em .433em .011em 0;content:"c"}mjx-c.mjx-c1D466.TEX-I::before{padding:.442em .49em .205em 0;content:"y"}mjx-c.mjx-c28::before{padding:.75em .389em .25em 0;content:"("}mjx-c.mjx-c31::before{padding:.666em .5em 0 0;content:"1"}mjx-c.mjx-c29::before{padding:.75em .389em .25em 0;content:")"}mjx-c.mjx-c3D::before{padding:.583em .778em .082em 0;content:"="}mjx-container[jax=CHTML]{line-height:0}mjx-container [space="1"]{margin-left:.111em}mjx-container [space="2"]{margin-left:.167em}mjx-container [space="3"]{margin-left:.222em}mjx-container [space="4"]{margin-left:.278em}mjx-container [space="5"]{margin-left:.333em}mjx-container [rspace="1"]{margin-right:.111em}mjx-container [rspace="2"]{margin-right:.167em}mjx-container [rspace="3"]{margin-right:.222em}mjx-container [rspace="4"]{margin-right:.278em}mjx-container [rspace="5"]{margin-right:.333em}mjx-container [size="s"]{font-size:70.7%}mjx-container [size=ss]{font-size:50%}mjx-container [size=Tn]{font-size:60%}mjx-container [size=sm]{font-size:85%}mjx-container [size=lg]{font-size:120%}mjx-container [size=Lg]{font-size:144%}mjx-container [size=LG]{font-size:173%}mjx-container [size=hg]{font-size:207%}mjx-container [size=HG]{font-size:249%}mjx-container [width=full]{width:100%}mjx-box{display:inline-block}mjx-block{display:block}mjx-itable{display:inline-table}mjx-row{display:table-row}mjx-row>*{display:table-cell}mjx-mtext{display:inline-block}mjx-mstyle{display:inline-block}mjx-merror{display:inline-block;color:red;background-color:#ff0}mjx-mphantom{visibility:hidden}mjx-assistive-mml{top:0;left:0;clip:rect(1px,1px,1px,1px);user-select:none;position:absolute!important;padding:1px 0 0!important;border:0!important;display:block!important;width:auto!important;overflow:hidden!important}mjx-assistive-mml[display=block]{width:100%!important}mjx-math{display:inline-block;text-align:left;line-height:0;text-indent:0;font-style:normal;font-weight:400;font-size:100%;letter-spacing:normal;border-collapse:collapse;overflow-wrap:normal;word-spacing:normal;white-space:nowrap;direction:ltr;padding:1px 0}mjx-container[jax=CHTML][display=true]{display:block;text-align:center;margin:1em 0}mjx-container[jax=CHTML][display=true][width=full]{display:flex}mjx-container[jax=CHTML][display=true] mjx-math{padding:0}mjx-container[jax=CHTML][justify=left]{text-align:left}mjx-container[jax=CHTML][justify=right]{text-align:right}mjx-mo{display:inline-block;text-align:left}mjx-stretchy-h{display:inline-table;width:100%}mjx-stretchy-h>*{display:table-cell;width:0}mjx-stretchy-h>*>mjx-c{display:inline-block;transform:scaleX(1)}mjx-stretchy-h>*>mjx-c::before{display:inline-block;width:initial}mjx-stretchy-h>mjx-ext{overflow:clip visible;width:100%}mjx-stretchy-h>mjx-ext>mjx-c::before{transform:scaleX(500)}mjx-stretchy-h>mjx-ext>mjx-c{width:0}mjx-stretchy-h>mjx-beg>mjx-c{margin-right:-.1em}mjx-stretchy-h>mjx-end>mjx-c{margin-left:-.1em}mjx-stretchy-v{display:inline-block}mjx-stretchy-v>*{display:block}mjx-stretchy-v>mjx-beg{height:0}mjx-stretchy-v>mjx-end>mjx-c{display:block}mjx-stretchy-v>*>mjx-c{transform:scaleY(1);transform-origin:left center;overflow:hidden}mjx-stretchy-v>mjx-ext{display:block;height:100%;box-sizing:border-box;border:0 solid transparent;overflow:visible clip}mjx-stretchy-v>mjx-ext>mjx-c::before{width:initial;box-sizing:border-box}mjx-stretchy-v>mjx-ext>mjx-c{transform:scaleY(500) translateY(.075em);overflow:visible}mjx-mark{display:inline-block;height:0}mjx-c{display:inline-block}mjx-utext{display:inline-block;padding:.75em 0 .2em}mjx-mn{display:inline-block;text-align:left}mjx-mi{display:inline-block;text-align:left}mjx-c::before{display:block;width:0}.MJX-TEX{font-family:MJXZERO,MJXTEX}.TEX-B{font-family:MJXZERO,MJXTEX-B}.TEX-I{font-family:MJXZERO,MJXTEX-I}.TEX-MI{font-family:MJXZERO,MJXTEX-MI}.TEX-BI{font-family:MJXZERO,MJXTEX-BI}.TEX-S1{font-family:MJXZERO,MJXTEX-S1}.TEX-S2{font-family:MJXZERO,MJXTEX-S2}.TEX-S3{font-family:MJXZERO,MJXTEX-S3}.TEX-S4{font-family:MJXZERO,MJXTEX-S4}.TEX-A{font-family:MJXZERO,MJXTEX-A}.TEX-C{font-family:MJXZERO,MJXTEX-C}.TEX-CB{font-family:MJXZERO,MJXTEX-CB}.TEX-FR{font-family:MJXZERO,MJXTEX-FR}.TEX-FRB{font-family:MJXZERO,MJXTEX-FRB}.TEX-SS{font-family:MJXZERO,MJXTEX-SS}.TEX-SSB{font-family:MJXZERO,MJXTEX-SSB}.TEX-SSI{font-family:MJXZERO,MJXTEX-SSI}.TEX-SC{font-family:MJXZERO,MJXTEX-SC}.TEX-T{font-family:MJXZERO,MJXTEX-T}.TEX-V{font-family:MJXZERO,MJXTEX-V}.TEX-VB{font-family:MJXZERO,MJXTEX-VB}mjx-stretchy-h mjx-c,mjx-stretchy-v mjx-c{font-family:MJXZERO,MJXTEX-S1,MJXTEX-S4,MJXTEX,MJXTEX-A!important}@font-face{font-family:MJXZERO;src:url("lib/fonts/mathjax_zero.woff") format("woff")}@font-face{font-family:MJXTEX;src:url("lib/fonts/mathjax_main-regular.woff") format("woff")}@font-face{font-family:MJXTEX-B;src:url("lib/fonts/mathjax_main-bold.woff") format("woff")}@font-face{font-family:MJXTEX-I;src:url("lib/fonts/mathjax_math-italic.woff") format("woff")}@font-face{font-family:MJXTEX-MI;src:url("lib/fonts/mathjax_main-italic.woff") format("woff")}@font-face{font-family:MJXTEX-BI;src:url("lib/fonts/mathjax_math-bolditalic.woff") format("woff")}@font-face{font-family:MJXTEX-S1;src:url("lib/fonts/mathjax_size1-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S2;src:url("lib/fonts/mathjax_size2-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S3;src:url("lib/fonts/mathjax_size3-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S4;src:url("lib/fonts/mathjax_size4-regular.woff") format("woff")}@font-face{font-family:MJXTEX-A;src:url("lib/fonts/mathjax_ams-regular.woff") format("woff")}@font-face{font-family:MJXTEX-C;src:url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff")}@font-face{font-family:MJXTEX-CB;src:url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff")}@font-face{font-family:MJXTEX-FR;src:url("lib/fonts/mathjax_fraktur-regular.woff") format("woff")}@font-face{font-family:MJXTEX-FRB;src:url("lib/fonts/mathjax_fraktur-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SS;src:url("lib/fonts/mathjax_sansserif-regular.woff") format("woff")}@font-face{font-family:MJXTEX-SSB;src:url("lib/fonts/mathjax_sansserif-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SSI;src:url("lib/fonts/mathjax_sansserif-italic.woff") format("woff")}@font-face{font-family:MJXTEX-SC;src:url("lib/fonts/mathjax_script-regular.woff") format("woff")}@font-face{font-family:MJXTEX-T;src:url("lib/fonts/mathjax_typewriter-regular.woff") format("woff")}@font-face{font-family:MJXTEX-V;src:url("lib/fonts/mathjax_vector-regular.woff") format("woff")}@font-face{font-family:MJXTEX-VB;src:url("lib/fonts/mathjax_vector-bold.woff") format("woff")}mjx-c.mjx-c3C::before{padding:.54em .778em .04em 0;content:"<"}mjx-c.mjx-c34::before{padding:.677em .5em 0 0;content:"4"}mjx-c.mjx-c30::before{padding:.666em .5em .022em 0;content:"0"}mjx-c.mjx-c1D44F.TEX-I::before{padding:.694em .429em .011em 0;content:"b"}mjx-c.mjx-c1D45D.TEX-I::before{padding:.442em .503em .194em 0;content:"p"}mjx-c.mjx-c1D436.TEX-I::before{padding:.705em .76em .022em 0;content:"C"}mjx-c.mjx-c1D447.TEX-I::before{padding:.677em .704em 0 0;content:"T"}</style><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="摘要"><p dir="auto">摘要</p></h1><div><blockquote dir="auto">
<p dir="auto">Comprehensive Identification of Mitochondrial Pseudogenes (NUMTs) in the Human Telomere-to-Telomere Reference Genome<br>
<img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602093029.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
<img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602103343.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"></p>
</blockquote></div><div class="heading-wrapper"><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="背景" dir="auto" class="heading" id="背景"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>背景</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">线粒体拟基因（NUMTs）是存在于核基因组中的线粒体DNA片段。这些NUMTs在研究中常因干扰线粒体DNA的分析而被视为障碍。</li>
<li data-line="1" dir="auto">人类参考基因组（如hg38）已部分集合了NUMTs，但在目前唯一完成的参考基因组T2T-CHR13中，NUMTs的详尽信息仍未明确。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="研究的意义" dir="auto" class="heading" id="研究的意义"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究的意义</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">使用人类线粒体泛基因组（Human Pan-Mitogenome, HPMT）从GeneBank库中，研究团队全面识别出参考基因组中的固定NUMTs。</li>
<li data-line="1" dir="auto"><strong>HPMT的使用旨在建立一个真实的线粒体DNA（mtDNA）突变谱，用以区分NUMTs中的多态性变异</strong>。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="研究结果" dir="auto" class="heading" id="研究结果"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>研究结果</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">在更严格的阈值下，研究者在三个人类参考基因组中识别出了<strong>约10%的额外NUMTs</strong>。</li>
<li data-line="1" dir="auto">相较于<strong>hg38，T2T-CHR13中NUMTs的数量增加了约6%</strong>，其中包括之前未组装的<strong>13、14和15号染色体短臂</strong>上的NUMTs。</li>
<li data-line="2" dir="auto">通过基于mtDNA的20-mer对齐，<strong>研究表明核基因组中存在更多类mtDNA的短片段，这些片段在进行短扩增子或无细胞mtDNA检测时应被避免</strong>。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="方法和技术" dir="auto" class="heading" id="方法和技术"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>方法和技术</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">研究中还使用了高通量测序的<a data-href="转座酶可访问染色质测定（Assay for Transposase-Accessible Chromatin with high-throughput sequencing，ATAC-seq）" href="疾病与基因/转座酶可访问染色质测定（assay-for-transposase-accessible-chromatin-with-high-throughput-sequencing，atac-seq）.html" class="internal-link" target="_self" rel="noopener">转座酶可访问染色质测定（Assay for Transposase-Accessible Chromatin with high-throughput sequencing，ATAC-seq）</a>技术，以分析在消除mtDNA前后的细胞系。</li>
<li data-line="1" dir="auto">结果显示，尽管有16%的测序数据源自mtDNA，但NUMTs对总体ATAC-seq的影响甚微。</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="介绍" dir="auto" class="heading" id="介绍">介绍</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="背景知识" dir="auto" class="heading" id="背景知识"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>背景知识</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">核线粒体DNA片段（NUMTs）是指嵌入核DNA中的线粒体DNA片段，也被称为线粒体拟基因。这些片段可能是固定的（大部分已标注在人类基因组中的等位基因）或在人群中呈多态性（在部分但非所有个体中存在）<sup data-footnote-id="fnref-1-19d4dedccb857110" class="footnote-ref" id="fnref-1-19d4dedccb857110"><a data-footref="1" href="#fn-1-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[1]</a></sup>。</li>
<li data-line="1" dir="auto">人类参考基因组的构建版本不同，固定的NUMTs数量也有所不同。例如，广泛使用的人类参考基因组hg19包含612至1049个NUMTs<sup data-footnote-id="fnref-2-19d4dedccb857110" class="footnote-ref" id="fnref-2-19d4dedccb857110"><a data-footref="2" href="#fn-2-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[2]</a></sup> <sup data-footnote-id="fnref-3-19d4dedccb857110" class="footnote-ref" id="fnref-3-19d4dedccb857110"><a data-footref="3" href="#fn-3-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[3]</a></sup> <sup data-footnote-id="fnref-4-19d4dedccb857110" class="footnote-ref" id="fnref-4-19d4dedccb857110"><a data-footref="4" href="#fn-4-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[4]</a></sup> <sup data-footnote-id="fnref-5-19d4dedccb857110" class="footnote-ref" id="fnref-5-19d4dedccb857110"><a data-footref="5" href="#fn-5-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[5]</a></sup>。</li>
<li data-line="2" dir="auto"><a data-href="核-线片段（NUMTs）" href="疾病与基因/核-线片段（numts）.html" class="internal-link" target="_self" rel="noopener">核-线片段（NUMTs）</a>的存在为准确识别 mtDNA 和 nDNA 带来了干扰，但是同时也为系统发育带来了新的见解。例如，一些研究认为由于 mtDNA 一旦插入 nDNA 中，就遵循细胞核的遗传规律和进化速率，所以可能作为一种构建<strong>系统发育</strong>的手段，有助于<strong>从独特的角度探讨物种进化的历史</strong><sup data-footnote-id="fnref-6-19d4dedccb857110" class="footnote-ref" id="fnref-6-19d4dedccb857110"><a data-footref="6" href="#fn-6-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[6]</a></sup> <sup data-footnote-id="fnref-7-19d4dedccb857110" class="footnote-ref" id="fnref-7-19d4dedccb857110"><a data-footref="7" href="#fn-7-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[7]</a></sup> <sup data-footnote-id="fnref-8-19d4dedccb857110" class="footnote-ref" id="fnref-8-19d4dedccb857110"><a data-footref="8" href="#fn-8-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[8]</a></sup>。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="之前研究的不足" dir="auto" class="heading" id="之前研究的不足"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>之前研究的不足</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">目前的人类参考基因组（如hg38和hg19）是从20多个个体组装而成的基因组，这可能导致某些NUMTs在新组装上丢失。</li>
<li data-line="1" dir="auto">现有研究<strong>未能充分代表T2T-CHR13中的NUMTs</strong>，而这是目前唯一由单一个体（Y染色体除外）衍生而成的完全组装的人类参考基因组。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="此次研究的创新点和意义" dir="auto" class="heading" id="此次研究的创新点和意义"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>此次研究的创新点和意义</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">本研究使用全基因组测序（WGS）方法，通过对比mtDNA与参考基因组的相似性<sup data-footnote-id="fnref-9-19d4dedccb857110" class="footnote-ref" id="fnref-9-19d4dedccb857110"><a data-footref="9" href="#fn-9-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[9]</a></sup>，新发现了许多NUMTs，特别是在人类基因组新完整组装的T2T-CHRM13中。<sup data-footnote-id="fnref-10-19d4dedccb857110" class="footnote-ref" id="fnref-10-19d4dedccb857110"><a data-footref="10" href="#fn-10-19d4dedccb857110" class="footnote-link" target="_self" rel="noopener">[10]</a></sup></li>
<li data-line="1" dir="auto">创新地建立了一个有效的人类线粒体泛基因组（human pan-mitogenome），该基因组包含所有主要的线粒体单倍群，更好地代表了人类mtDNA的突变谱。</li>
<li data-line="2" dir="auto">在ATAC-seq实验中，识别并排除了可能影响实验结果的mtDNA类序列（NUMTs、短片段及mito-blacklist区域），从而提高了实验的准确性和可靠性。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="综合评价" dir="auto" class="heading" id="综合评价"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>综合评价</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto">这项研究不仅提高了对NUMTs在人类基因组中作用的理解，而且通过更准确地识别这些序列，为线粒体相关的实验和分析提供了基础设施。</li>
<li data-line="1" dir="auto">此外，通过这些研究成果，能够更好地应对NUMTs在染色质靶向技术、mtDNA异质性检测和PCR检测中可能引入的偏差，从而推动相关领域的科学发展。</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="方法" dir="auto" class="heading" id="方法">方法</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="1. 准备人类线粒体泛基因组（HPMT）" dir="auto" class="heading" id="1._准备人类线粒体泛基因组（HPMT）"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>1. 准备人类线粒体泛基因组（HPMT）</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>数据获取</strong>：从GeneBank下载了62,052个完整的mtDNA序列和80,841个控制区域序列。</li>
<li data-line="1" dir="auto"><strong>数据处理</strong>：使用自身BLAST（self-blast）方法来消除重复的序列，保留长度至少为16,530 bp的独特序列，最终留下了47,525个独特的完整mtDNA序列。</li>
<li data-line="2" dir="auto"><strong>数据整合</strong>：结合完整的mtDNA序列、rCRS和控制区序列，共计128,367个序列，形成了人类线粒体泛基因组池（HPMT）以供后续分析。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="2. BLAST和过滤" dir="auto" class="heading" id="2._BLAST和过滤"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>2. BLAST和过滤</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>BLAST设置</strong>：在Linux环境下运行BLAST-2.14.0+，为hg19、hg38和T2T-CHR13参考基因组各自建立blastn数据库。</li>
<li data-line="1" dir="auto"><strong>查询与过滤</strong>：使用HPMT查询参考基因组数据库，设置-stringent的参数（如e-value 10^-4）来确保结果的精确性。使用bash脚本过滤出身份大于63%且e-value小于10^-4的序列，合并重叠的NUMT区域。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="3. 短片段的比对" dir="auto" class="heading" id="3._短片段的比对"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>3. 短片段的比对</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>滑动窗口法</strong>：将HPMT逐步分割成20-mer片段，并生成单端测序的fastq文件，模拟35的测序质量。</li>
<li data-line="1" dir="auto"><strong>比对</strong>：使用Bowtie-1.3.1将短读比对到人类参考基因组，生成的sam文件转换为bam文件，并使用samtools和sambamba移除含有"N"或IUPAC编码的读取。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="4. 在HEK293细胞系中消除线粒体拷贝数" dir="auto" class="heading" id="4._在HEK293细胞系中消除线粒体拷贝数"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>4. 在HEK293细胞系中消除线粒体拷贝数</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>实验方法</strong>：采用Domenico等人的方法，使用pCW57-MCS1-P2A-MCS2（Neo）载体在HEK293野生型细胞中消除线粒体拷贝数。所用的HSV-1病毒的UL12.5片段特异性定位于线粒体并通过线粒体核酸酶ExoG和EndoG实现mtDNA的削减。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="5. ATAC-Seq数据分析" dir="auto" class="heading" id="5._ATAC-Seq数据分析"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>5. ATAC-Seq数据分析</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>质量控制</strong>：使用fastqc进行原始fastq文件的质量控制，使用trimmomatic剪切适配器和低质量读取。</li>
<li data-line="1" dir="auto"><strong>比对与峰值调用</strong>：使用bwa进行比对，sambamba移除重复，samtools提取映射质量高于30的读取，使用MACS2进行峰值调用。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="6. 统计与可视化" dir="auto" class="heading" id="6._统计与可视化"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>6. 统计与可视化</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>数据可视化</strong>：使用ChromoMap和Circos生成染色体地图和Circos图，所有数据统计和图表准备均在R环境中完成。</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="结果" dir="auto" class="heading" id="结果">结果</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="1. 人类线粒体泛基因组（HPMT）" dir="auto" class="heading" id="1._人类线粒体泛基因组（HPMT）"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>1. 人类线粒体泛基因组（HPMT）</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>数据组装</strong>：从GeneBank下载并处理后，<strong>得到47,525个有效的mtDNA序列和80,841个控制区序列，这些数据一起构成了覆盖主要单倍群的HPMT，总计128,367个序列</strong>。</li>
<li data-line="1" dir="auto"><strong>代表性</strong>：这些序列能够代表人类mtDNA的多样性，涵盖了从400 bp到1598 bp不等长度的控制区序列。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="2. 使用HPMT识别更多的NUMTs" dir="auto" class="heading" id="2._使用HPMT识别更多的NUMTs"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>2. 使用HPMT识别更多的NUMTs</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>NUMTs识别</strong>：在三个参考基因组hg19、hg38和T2T-CHR13中，分别识别出814、815和863个NUMTs。</li>
<li data-line="1" dir="auto"><strong>长度和分布</strong>：这些NUMTs的总长度分别为584,137 bp到620,968 bp，长度范围从35 bp到14,855 bp。</li>
<li data-line="2" dir="auto"><strong>合并NUMTs</strong>：将来自完整mtDNA和控制区序列识别的NUMTs区域合并后，hg19、hg38和T2T-CHR13中的NUMTs数量分别为907、908和958个，总长度分别为594,169 bp、596,516 bp和631,156 bp。</li>
<li data-line="3" dir="auto"><strong>基因组组装的影响</strong>：T2T-CHR13显示出比其他两个参考基因组有更多和更长的NUMTs，这突显了高质量基因组组装在NUMTs识别中的重要性。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="3. 与已知NUMTs集合的比较" dir="auto" class="heading" id="3._与已知NUMTs集合的比较"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>3. 与已知NUMTs集合的比较</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>与以往研究比较</strong>：与Li等人和Calabrese等人的研究相比，这项研究识别出的NUMTs数量更多，长度也更长。</li>
<li data-line="1" dir="auto"><strong>e-value设定</strong>：通过设定更严格的e-value（10^-4），确保了结果的精确性，尽管可能降低了灵敏度。</li>
<li data-line="2" dir="auto"><strong>Mega-NUMTs</strong>：一些研究提出人类基因组中可能存在大型NUMTs（Mega-NUMTs），本研究中由Li等人识别的NUMTs可能被过于保守地分割成了更小的片段。</li>
<li data-line="3" dir="auto"><strong>mito-blacklist与NUMTs的重叠</strong>：Ogata等人定义的mito-blacklist部分区域与本研究中识别的NUMTs重叠，但不完全相同，这表明可能还有更多的mtDNA插入到核基因组中。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="4. NUMTs在人类参考基因组中的分布" dir="auto" class="heading" id="4._NUMTs在人类参考基因组中的分布"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>4. NUMTs在人类参考基因组中的分布</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>特定染色体上的NUMTs</strong>：在T2T-CHR13基因组的chr13、chr14和chr15的短臂上观察到NUMTs的存在，这些区域在hg19和hg38中未完全组装，因此之前没有报告过这些区域的NUMTs。</li>
<li data-line="1" dir="auto"><strong>NUMTs的染色体分布</strong>：<strong>NUMTs在各染色体上的分布与染色体的长度成比例，但chr2显示出相对较高的NUMTs密度</strong>；而chr1、chr3、chr6和chr12的NUMTs则相对稀疏。</li>
<li data-line="2" dir="auto"><strong>无NUMTs区域</strong>：某些染色体上存在无NUMTs的区域，最长的无NUMTs区域位于<strong>Y染色体的长臂</strong>上，长度达35Mb。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="5. 核基因组中的mtDNA类短片段" dir="auto" class="heading" id="5._核基因组中的mtDNA类短片段"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>5. 核基因组中的mtDNA类短片段</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>短片段的分布和鉴定</strong>：通过滑动窗口方法，将128,367个HPMT序列分割为20-mer短片段，总计生成了854,733,854个短序列，其中423,658种是独特的20-mer类型。</li>
<li data-line="1" dir="auto"><strong>核基因组中的20-mer映射</strong>：使用Bowtie对HPMT-20进行到人类参考基因组的比对，涵盖了约200,000个核位置。</li>
<li data-line="2" dir="auto"><strong>短片段对实验的潜在影响</strong>：在进行短扩增子或无细胞mtDNA检测时，建议避免这些高频重复的mtDNA同源序列。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="6. NUMTs对ATAC-seq的影响似乎有限" dir="auto" class="heading" id="6._NUMTs对ATAC-seq的影响似乎有限"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>6. NUMTs对ATAC-seq的影响似乎有限</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>实验设计</strong>：在HEK293细胞系中进行mtDNA数量的减少，然后在mtDNA减少前后进行ATAC-seq实验。</li>
<li data-line="1" dir="auto"><strong>读数比例和mito-blacklist区域的影响</strong>：mtDNA减少前后的读数比例表明，NUMTs和mtDNA类区域对ATAC-seq峰值调用的影响有限。</li>
<li data-line="2" dir="auto"><strong>NUMTs-Blacklist的设置和评估</strong>：将NUMTs和mito-blacklist合并为排除集，评估其对ATAC-seq峰值调用的影响。</li>
</ul></div><div><p dir="auto">尽管NUMTs和mtDNA类区域对ATAC-seq结果的直接影响有限，但合理识别并排除这些区域对于确保实验结果的准确性和可靠性仍然至关重要。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="总结" dir="auto" class="heading" id="总结">总结</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="讨论" dir="auto" class="heading" id="讨论"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>讨论</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>NUMTs的分布</strong>：与当前的参考基因组版本hg38相比，<strong>CHR13包含约6%更多的NUMTs。考虑到CHR13的组装完整度比hg38高出约8%</strong>，这一发现是合理的。</li>
<li data-line="1" dir="auto"><strong>样本来源的差异</strong>：CHR13来源于完全不同的样本，而hg38约70%来自于一个白种人个体，因此不能排除<strong>个体样本差异对结果的影响</strong>。</li>
<li data-line="2" dir="auto"><strong>mtDNA断点的起源</strong>：mtDNA的断点主要起源于tRNA和控制区，这些区域的突变率相对较高。目前还不能为这些断点的产生提供明确的解释。</li>
<li data-line="3" dir="auto"><strong>核基因组中的mtDNA类短片段</strong>：观察到核基因组中存在许多mtDNA类的短片段，它们太短，无法被识别为NUMTs。这些短片段可能是mtDNA插入的结果，也可能是巧合产生的类似序列。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="结论" dir="auto" class="heading" id="结论"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>结论</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>系统识别mtDNA类区域</strong>：使用全线粒体基因组（HPMT），系统地识别了三个人类参考基因组中的mtDNA类区域，包括NUMTs、mtDNA类短片段和可能导致错误读数堆积的mito-blacklist区域。</li>
<li data-line="1" dir="auto"><strong>NUMTs的新发现</strong>：发现NUMTs的内容可能比之前已知的多约10%，并首次报道了在唯一完全组装的人类参考基因组T2T-CHR13中NUMTs的分布。</li>
<li data-line="2" dir="auto"><strong>ATAC-seq实验</strong>：通过在减少mtDNA拷贝数前后进行的ATAC-seq实验，得出结论认为mtDNA类区域不太可能显著影响ATAC-seq结果，即使在测序数据中有16%源自mtDNA。</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="局限性与未来展望" dir="auto" class="heading" id="局限性与未来展望"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>局限性与未来展望</h2><div class="heading-children"><div><ul dir="auto">
<li data-line="0" dir="auto"><strong>局限性</strong>：研究的局限性包括<strong>样本来源的差异可能影响NUMTs的识别</strong>，以及现有技术难以准确区分非常短的mtDNA类序列。另外，<strong>公共数据库中的数据虽然覆盖广泛，但可能并不完全反映所有人群的遗传多样性</strong>。特别是对于那些在现有数据库中采样较少的群体，使用这些数据可能无法全面揭示其遗传特征和突变谱。<strong>公共数据库中的FASTA格式的序列提供了基因组的连续片段，但不包含原始测序数据中的读段信息</strong>。读段信息对于分析如插入、删除以及复杂的结构变异非常重要。没有读段信息，很难确定某些序列变异的确切性质和它们在基因组中的具体位置。大多数基因组研究使用的是单一的参考序列，这可能不足以涵盖所有人群的遗传多样性。尤其是在研究NUMTs时，个体间的差异可能对识别和分类NUMTs片段有重要影响。</li>
<li data-line="1" dir="auto"><strong>未来展望</strong>：建议在设计短扩增子或进行无细胞mtDNA检测时考虑这些短片段。未来的研究需要进一步探讨这些短片段的起源及其在基因组学中的作用。</li>
</ul></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="图表" dir="auto" class="heading" id="图表">图表</h1><div class="heading-children"><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602103923.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 1. 三个人类参考基因组中的NUMTs（hg19、hg38、T2T-CHR13）。（A）全线粒体基因组数据库中的线粒体单倍群及其比例。不同颜色代表主要单倍群，同色中的小方块代表主要单倍群内的亚型单倍体。（B）三个参考基因组中NUMTs的数量和长度。（C、D）NUMTs的长度（对数尺度log2）分布。不同颜色代表不同的参考基因组。图（D）中箱形图上方的点代表离群值。（E）NUMTs长度分布的直方图（25 bp~14,855 bp）。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602103946.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 2. 与已知的NUMT（hg19）比较。（A）本研究中识别的NUMTs的数量和长度（对数尺度log2）与已知NUMTs的比较。（B）不同NUMT中NUMTs长度的分布（对数尺度log2）。（C）本研究中识别的NUMTs与之前三项研究中识别的NUMTs的重叠情况。两个NUMT的并集构成了100%。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104002.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 3. 不同染色体上的 NUMT。 (A,B) 不同染色体上 NUMT 的分布（(A) hg38；(B) T2T-CHR13）。带的颜色表示 NUMT 的长度。红色框突出显示 Y 染色体上的间隙区域以及染色体 chr13、chr14 和 chr15 的短臂。 NUMT 的总长度 (C) 和数量 (D) 与染色体长度呈线性关系。不同的颜色表示参考基因组。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104010.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 4. NUMTs的断点。Circos图展示了在两个参考基因组hg38（A）和T2T-CHR13（B）上NUMTs的分布。Circos图的左半圆表示线粒体基因组的坐标，而右半圆表示核染色体1-22和XY的坐标。连接线表示NUMTs的断点，线的颜色代表不同染色体的起源。（C）映射到mtDNA的NUMTs的覆盖率。x轴表示对应于mtDNA的位置（1~16,569），y轴表示覆盖率。覆盖率低的区域表明存在断点。灰色和绿色阴影分别突出显示了mtDNA控制区和tRNA区的断点。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104023.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 5. 映射到核基因组的20-mer片段的数量。x轴显示映射位置的长度（由20-mer决定）。左侧的y轴对应20-mer片段的数量，右侧的y轴对应NUMTs的数量。虚线曲线显示随着映射位置长度的增加，20-mer的数量呈现下降趋势（左侧y轴）。相反，柱状图显示随着映射位置长度的增加，NUMTs的数量呈上升趋势。此外，从图1E可见，较短的NUMTs（&lt;70 bp）的数量较少，与较长的NUMTs（&gt;70 bp）相比。这表明可能存在未被注意到的短NUMTs。此外，点曲线的转折点出现在25~30 bp的范围内，表明我们应该将最短NUMTs的截止点设置在这个范围内（黑色虚线，28 bp）。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104121.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
图 6. mtDNA 还原前后 ATAC-seq 的比较。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602103935.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
表 1. 本研究和之前研究中的 NUMT 汇编。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104033.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
表 2. 核基因组内的短 mtDNA 片段（20 聚体）。</p></div><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/20240602104127.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"><br>
表 3. ATAC-seq（NUMTs-Blacklist）上具有潜在影响的 mtDNA 样区域。</p></div><div dir="ltr" style="overflow-x: auto;"><table>
<thead>
<tr>
<th dir="ltr"><strong>术语</strong></th>
<th dir="ltr"><strong>描述</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td dir="ltr">NUMTs</td>
<td dir="ltr">核线粒体DNA片段</td>
</tr>
<tr>
<td dir="ltr">mtDNA</td>
<td dir="ltr">线粒体DNA</td>
</tr>
<tr>
<td dir="ltr">HPMT</td>
<td dir="ltr">人类线粒体泛基因组</td>
</tr>
<tr>
<td dir="ltr">ATAC-seq</td>
<td dir="ltr">高通量测序的转座酶可访问染色质测定</td>
</tr>
<tr>
<td dir="ltr">cf-mtDNA</td>
<td dir="ltr">无细胞线粒体DNA</td>
</tr>
</tbody>
</table></div></div></div><div class="heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div>
</div><div><section class="footnotes"><hr dir="auto"><ol dir="auto">
<li data-line="-18" data-footnote-id="fn-1-19d4dedccb857110" id="fn-1-19d4dedccb857110" dir="auto"><p dir="auto">Woerner, A.E.; Cihlar, J.C.; Smart, U.; Budowle, B. Numt identification and removal with RtN! Bioinformatics 2020, 36, 5115–5116. [CrossRef]<a href="#fnref-1-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-16" data-footnote-id="fn-2-19d4dedccb857110" id="fn-2-19d4dedccb857110" dir="auto"><p dir="auto">Woischnik, M.; Moraes, C.T. Pattern of organization of human mitochondrial pseudogenes in the nuclear genome. Genome Res. 2002, 12, 885–893. [CrossRef] [PubMed]<a href="#fnref-2-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-14" data-footnote-id="fn-3-19d4dedccb857110" id="fn-3-19d4dedccb857110" dir="auto"><p dir="auto">Calabrese, F.M.; Balacco, D.L.; Preste, R.; Diroma, M.A.; Forino, R.; Ventura, M.; Attimonelli, M. NumtS colonization in mammalian genomes. Sci. Rep. 2017, 7, 16357. [CrossRef] [PubMed]<a href="#fnref-3-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-12" data-footnote-id="fn-4-19d4dedccb857110" id="fn-4-19d4dedccb857110" dir="auto"><p dir="auto">Li, M.; Schroeder, R.; Ko, A.; Stoneking, M. Fidelity of capture-enrichment for mtDNA genome sequencing: Influence of NUMTs. Nucleic Acids Res. 2012, 40, e137. [CrossRef] [PubMed]<a href="#fnref-4-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-10" data-footnote-id="fn-5-19d4dedccb857110" id="fn-5-19d4dedccb857110" dir="auto"><p dir="auto">Calabrese, F.M.; Simone, D.; Attimonelli, M. Primates and mouse NumtS in the UCSC Genome Browser. BMC Bioinform. 2012, 13 (Suppl. S4), S15. [CrossRef]<a href="#fnref-5-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-8" data-footnote-id="fn-6-19d4dedccb857110" id="fn-6-19d4dedccb857110" dir="auto"><p dir="auto">Calabrese, F.M.; Balacco, D.L.; Preste, R.; Diroma, M.A.; Forino, R.; Ventura, M.; Attimonelli, M. NumtS colonization in mammalian genomes. Sci. Rep. 2017, 7, 16357. [CrossRef] [PubMed]<a href="#fnref-6-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-6" data-footnote-id="fn-7-19d4dedccb857110" id="fn-7-19d4dedccb857110" dir="auto"><p dir="auto">Popadin, K.; Gunbin, K.; Peshkin, L.; Annis, S.; Fleischmann, Z.; Franco, M.; Kraytsberg, Y.; Markuzon, N.; Ackermann, R.; Khrapko, K. Mitochondrial Pseudogenes Suggest Repeated Inter-Species Hybridization among Direct Human Ancestors. Genes. 2022, 13, 810. [CrossRef] [PubMed]<a href="#fnref-7-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-4" data-footnote-id="fn-8-19d4dedccb857110" id="fn-8-19d4dedccb857110" dir="auto"><p dir="auto">Bucking, R.; Cox, M.P.; Hudjashov, G.; Saag, L.; Sudoyo, H.; Stoneking, M. Archaic mitochondrial DNA inserts in modern day nuclear genomes. BMC Genom. 2019, 20, 1017. [CrossRef] Hazkani-Covo, E.; Graur, D. A comparative analysis of numt evolution in human and chimpanzee. Mol. Biol. Evol. 2007, 24, 13–18. [CrossRef] [PubMed] Dayama, G.; Zhou, W.; Prado-Martinez, J.; Marques-Bonet, T.; Mills, R.E. Characterization of nuclear mitochondrial insertions in the whole genomes of primates. NAR Genom. Bioinform. 2020, 2, lqaa089. [CrossRef] [PubMed] Gherman, A.; Chen, P.E.; Teslovich, T.M.; Stankiewicz, P.; Withers, M.; Kashuk, C.S.; Chakravarti, A.; Lupski, J.R.; Cutler, D.J.; Katsanis, N. Population Bottlenecks as a Potential Major Shaping Force of Human Genome Architecture. PLoS Genet. 2007, 3, e119. [CrossRef]  Gunbin, K.; Peshkin, L.; Popadin, K.; Annis, S.; Ackermann, R.; Khrapko, K. Integration of mtDNA pseudogenes into the nuclear genome coincides with speciation of the human genus. A hypothesis. Mitochondrion 2017, 34, 20–23. [CrossRef]<a href="#fnref-8-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="-2" data-footnote-id="fn-9-19d4dedccb857110" id="fn-9-19d4dedccb857110" dir="auto"><p dir="auto">Dayama, G.; Emery, S.B.; Kidd, J.M.; Mills, R.E. The genomic landscape of polymorphic human nuclear mitochondrial insertions. Nucleic Acids Res. 2014, 42, 12640–12649. [<a data-tooltip-position="top" aria-label="https://doi.org/10.1093/nar/gku1038" rel="noopener" class="external-link" href="https://doi.org/10.1093/nar/gku1038" target="_blank">CrossRef</a>]<a href="#fnref-9-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
<li data-line="0" data-footnote-id="fn-10-19d4dedccb857110" id="fn-10-19d4dedccb857110" dir="auto"><p dir="auto">Gao, Y.; Yang, X.; Chen, H.; Tan, X.; Yang, Z.; Deng, L.; Wang, B.; Kong, S.; Li, S.; Cui, Y.; et al. A pangenome reference of 36 Chinese populations. Nature 2023, 619, 112–121. [<a data-tooltip-position="top" aria-label="https://doi.org/10.1038/s41586-023-06173-7" rel="noopener" class="external-link" href="https://doi.org/10.1038/s41586-023-06173-7" target="_blank">CrossRef</a>] [<a data-tooltip-position="top" aria-label="https://www.ncbi.nlm.nih.gov/pubmed/37316654" rel="noopener" class="external-link" href="https://www.ncbi.nlm.nih.gov/pubmed/37316654" target="_blank">PubMed</a>] Liao, W.W.; Asri, M.; Ebler, J.; Doerr, D.; Haukness, M.; Hickey, G.; Lu, S.; Lucas, J.K.; Monlong, J.; Abel, H.J.; et al. A draft human pangenome reference. Nature 2023, 617, 312–324. [<a data-tooltip-position="top" aria-label="https://doi.org/10.1038/s41586-023-05896-x" rel="noopener" class="external-link" href="https://doi.org/10.1038/s41586-023-05896-x" target="_blank">CrossRef</a>]<a href="#fnref-10-19d4dedccb857110" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
</ol></section></div><div class="mod-footer"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#摘要"><div class="tree-item-contents heading-link" heading-name="摘要"><span class="tree-item-title">摘要</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#背景"><div class="tree-item-contents heading-link" heading-name="背景"><span class="tree-item-title">背景</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#研究的意义"><div class="tree-item-contents heading-link" heading-name="研究的意义"><span class="tree-item-title">研究的意义</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#研究结果"><div class="tree-item-contents heading-link" heading-name="研究结果"><span class="tree-item-title">研究结果</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#方法和技术"><div class="tree-item-contents heading-link" heading-name="方法和技术"><span class="tree-item-title">方法和技术</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#介绍"><div class="tree-item-contents heading-link" heading-name="介绍"><span class="tree-item-title">介绍</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#背景知识"><div class="tree-item-contents heading-link" heading-name="背景知识"><span class="tree-item-title">背景知识</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#之前研究的不足"><div class="tree-item-contents heading-link" heading-name="之前研究的不足"><span class="tree-item-title">之前研究的不足</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#此次研究的创新点和意义"><div class="tree-item-contents heading-link" heading-name="此次研究的创新点和意义"><span class="tree-item-title">此次研究的创新点和意义</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#综合评价"><div class="tree-item-contents heading-link" heading-name="综合评价"><span class="tree-item-title">综合评价</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#方法"><div class="tree-item-contents heading-link" heading-name="方法"><span class="tree-item-title">方法</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#1._准备人类线粒体泛基因组（HPMT）"><div class="tree-item-contents heading-link" heading-name="1. 准备人类线粒体泛基因组（HPMT）"><span class="tree-item-title">1. 
准备人类线粒体泛基因组（HPMT）
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#2._BLAST和过滤"><div class="tree-item-contents heading-link" heading-name="2. BLAST和过滤"><span class="tree-item-title">2. 
BLAST和过滤
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#3._短片段的比对"><div class="tree-item-contents heading-link" heading-name="3. 短片段的比对"><span class="tree-item-title">3. 
短片段的比对
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#4._在HEK293细胞系中消除线粒体拷贝数"><div class="tree-item-contents heading-link" heading-name="4. 在HEK293细胞系中消除线粒体拷贝数"><span class="tree-item-title">4. 
在HEK293细胞系中消除线粒体拷贝数
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#5._ATAC-Seq数据分析"><div class="tree-item-contents heading-link" heading-name="5. ATAC-Seq数据分析"><span class="tree-item-title">5. 
ATAC-Seq数据分析
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#6._统计与可视化"><div class="tree-item-contents heading-link" heading-name="6. 统计与可视化"><span class="tree-item-title">6. 
统计与可视化
</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#结果"><div class="tree-item-contents heading-link" heading-name="结果"><span class="tree-item-title">结果</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#1._人类线粒体泛基因组（HPMT）"><div class="tree-item-contents heading-link" heading-name="1. 人类线粒体泛基因组（HPMT）"><span class="tree-item-title">1. 
人类线粒体泛基因组（HPMT）
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#2._使用HPMT识别更多的NUMTs"><div class="tree-item-contents heading-link" heading-name="2. 使用HPMT识别更多的NUMTs"><span class="tree-item-title">2. 
使用HPMT识别更多的NUMTs
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#3._与已知NUMTs集合的比较"><div class="tree-item-contents heading-link" heading-name="3. 与已知NUMTs集合的比较"><span class="tree-item-title">3. 
与已知NUMTs集合的比较
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#4._NUMTs在人类参考基因组中的分布"><div class="tree-item-contents heading-link" heading-name="4. NUMTs在人类参考基因组中的分布"><span class="tree-item-title">4. 
NUMTs在人类参考基因组中的分布
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#5._核基因组中的mtDNA类短片段"><div class="tree-item-contents heading-link" heading-name="5. 核基因组中的mtDNA类短片段"><span class="tree-item-title">5. 
核基因组中的mtDNA类短片段
</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#6._NUMTs对ATAC-seq的影响似乎有限"><div class="tree-item-contents heading-link" heading-name="6. NUMTs对ATAC-seq的影响似乎有限"><span class="tree-item-title">6. 
NUMTs对ATAC-seq的影响似乎有限
</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#总结"><div class="tree-item-contents heading-link" heading-name="总结"><span class="tree-item-title">总结</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#讨论"><div class="tree-item-contents heading-link" heading-name="讨论"><span class="tree-item-title">讨论</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#结论"><div class="tree-item-contents heading-link" heading-name="结论"><span class="tree-item-title">结论</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#局限性与未来展望"><div class="tree-item-contents heading-link" heading-name="局限性与未来展望"><span class="tree-item-title">局限性与未来展望</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#图表"><div class="tree-item-contents heading-link" heading-name="图表"><span class="tree-item-title">图表</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="文献及报道\文献\人类端粒到端粒参考基因组中线粒体假基因-(numt)-的全面鉴定.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>