<html><head>
		<title>PAMLX：最大似然计算</title>
		<base href="..\..\../">
		<meta id="root-path" root-path="..\..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - PAMLX：最大似然计算">
		<meta property="og:title" content="PAMLX：最大似然计算">
		<meta property="og:description" content="思维领域 - PAMLX：最大似然计算">
		<meta property="og:type" content="website">
		<meta property="og:url" content="软件/其它生信软件/k-s/pamlx：最大似然计算.html">
		<meta property="og:image" content="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/1f5e3.svg">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon fancy-code h2-underline"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-container[jax="CHTML"] { line-height: 0; }
mjx-container [space="1"] { margin-left: 0.111em; }
mjx-container [space="2"] { margin-left: 0.167em; }
mjx-container [space="3"] { margin-left: 0.222em; }
mjx-container [space="4"] { margin-left: 0.278em; }
mjx-container [space="5"] { margin-left: 0.333em; }
mjx-container [rspace="1"] { margin-right: 0.111em; }
mjx-container [rspace="2"] { margin-right: 0.167em; }
mjx-container [rspace="3"] { margin-right: 0.222em; }
mjx-container [rspace="4"] { margin-right: 0.278em; }
mjx-container [rspace="5"] { margin-right: 0.333em; }
mjx-container [size="s"] { font-size: 70.7%; }
mjx-container [size="ss"] { font-size: 50%; }
mjx-container [size="Tn"] { font-size: 60%; }
mjx-container [size="sm"] { font-size: 85%; }
mjx-container [size="lg"] { font-size: 120%; }
mjx-container [size="Lg"] { font-size: 144%; }
mjx-container [size="LG"] { font-size: 173%; }
mjx-container [size="hg"] { font-size: 207%; }
mjx-container [size="HG"] { font-size: 249%; }
mjx-container [width="full"] { width: 100%; }
mjx-box { display: inline-block; }
mjx-block { display: block; }
mjx-itable { display: inline-table; }
mjx-row { display: table-row; }
mjx-row > * { display: table-cell; }
mjx-mtext { display: inline-block; }
mjx-mstyle { display: inline-block; }
mjx-merror { display: inline-block; color: red; background-color: yellow; }
mjx-mphantom { visibility: hidden; }
mjx-assistive-mml { top: 0px; left: 0px; clip: rect(1px, 1px, 1px, 1px); user-select: none; position: absolute !important; padding: 1px 0px 0px !important; border: 0px !important; display: block !important; width: auto !important; overflow: hidden !important; }
mjx-assistive-mml[display="block"] { width: 100% !important; }
mjx-math { display: inline-block; text-align: left; line-height: 0; text-indent: 0px; font-style: normal; font-weight: normal; font-size: 100%; letter-spacing: normal; border-collapse: collapse; overflow-wrap: normal; word-spacing: normal; white-space: nowrap; direction: ltr; padding: 1px 0px; }
mjx-container[jax="CHTML"][display="true"] { display: block; text-align: center; margin: 1em 0px; }
mjx-container[jax="CHTML"][display="true"][width="full"] { display: flex; }
mjx-container[jax="CHTML"][display="true"] mjx-math { padding: 0px; }
mjx-container[jax="CHTML"][justify="left"] { text-align: left; }
mjx-container[jax="CHTML"][justify="right"] { text-align: right; }
mjx-mi { display: inline-block; text-align: left; }
mjx-c { display: inline-block; }
mjx-utext { display: inline-block; padding: 0.75em 0px 0.2em; }
mjx-mo { display: inline-block; text-align: left; }
mjx-stretchy-h { display: inline-table; width: 100%; }
mjx-stretchy-h > * { display: table-cell; width: 0px; }
mjx-stretchy-h > * > mjx-c { display: inline-block; transform: scaleX(1); }
mjx-stretchy-h > * > mjx-c::before { display: inline-block; width: initial; }
mjx-stretchy-h > mjx-ext { overflow: clip visible; width: 100%; }
mjx-stretchy-h > mjx-ext > mjx-c::before { transform: scaleX(500); }
mjx-stretchy-h > mjx-ext > mjx-c { width: 0px; }
mjx-stretchy-h > mjx-beg > mjx-c { margin-right: -0.1em; }
mjx-stretchy-h > mjx-end > mjx-c { margin-left: -0.1em; }
mjx-stretchy-v { display: inline-block; }
mjx-stretchy-v > * { display: block; }
mjx-stretchy-v > mjx-beg { height: 0px; }
mjx-stretchy-v > mjx-end > mjx-c { display: block; }
mjx-stretchy-v > * > mjx-c { transform: scaleY(1); transform-origin: left center; overflow: hidden; }
mjx-stretchy-v > mjx-ext { display: block; height: 100%; box-sizing: border-box; border: 0px solid transparent; overflow: visible clip; }
mjx-stretchy-v > mjx-ext > mjx-c::before { width: initial; box-sizing: border-box; }
mjx-stretchy-v > mjx-ext > mjx-c { transform: scaleY(500) translateY(0.075em); overflow: visible; }
mjx-mark { display: inline-block; height: 0px; }
mjx-texatom { display: inline-block; text-align: left; }
mjx-c::before { display: block; width: 0px; }
.MJX-TEX { font-family: MJXZERO, MJXTEX; }
.TEX-B { font-family: MJXZERO, MJXTEX-B; }
.TEX-I { font-family: MJXZERO, MJXTEX-I; }
.TEX-MI { font-family: MJXZERO, MJXTEX-MI; }
.TEX-BI { font-family: MJXZERO, MJXTEX-BI; }
.TEX-S1 { font-family: MJXZERO, MJXTEX-S1; }
.TEX-S2 { font-family: MJXZERO, MJXTEX-S2; }
.TEX-S3 { font-family: MJXZERO, MJXTEX-S3; }
.TEX-S4 { font-family: MJXZERO, MJXTEX-S4; }
.TEX-A { font-family: MJXZERO, MJXTEX-A; }
.TEX-C { font-family: MJXZERO, MJXTEX-C; }
.TEX-CB { font-family: MJXZERO, MJXTEX-CB; }
.TEX-FR { font-family: MJXZERO, MJXTEX-FR; }
.TEX-FRB { font-family: MJXZERO, MJXTEX-FRB; }
.TEX-SS { font-family: MJXZERO, MJXTEX-SS; }
.TEX-SSB { font-family: MJXZERO, MJXTEX-SSB; }
.TEX-SSI { font-family: MJXZERO, MJXTEX-SSI; }
.TEX-SC { font-family: MJXZERO, MJXTEX-SC; }
.TEX-T { font-family: MJXZERO, MJXTEX-T; }
.TEX-V { font-family: MJXZERO, MJXTEX-V; }
.TEX-VB { font-family: MJXZERO, MJXTEX-VB; }
mjx-stretchy-v mjx-c, mjx-stretchy-h mjx-c { font-family: MJXZERO, MJXTEX-S1, MJXTEX-S4, MJXTEX, MJXTEX-A !important; }
@font-face { font-family: MJXZERO; src: url("lib/fonts/mathjax_zero.woff") format("woff"); }
@font-face { font-family: MJXTEX; src: url("lib/fonts/mathjax_main-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-B; src: url("lib/fonts/mathjax_main-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-I; src: url("lib/fonts/mathjax_math-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-MI; src: url("lib/fonts/mathjax_main-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-BI; src: url("lib/fonts/mathjax_math-bolditalic.woff") format("woff"); }
@font-face { font-family: MJXTEX-S1; src: url("lib/fonts/mathjax_size1-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S2; src: url("lib/fonts/mathjax_size2-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S3; src: url("lib/fonts/mathjax_size3-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-S4; src: url("lib/fonts/mathjax_size4-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-A; src: url("lib/fonts/mathjax_ams-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-C; src: url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-CB; src: url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-FR; src: url("lib/fonts/mathjax_fraktur-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-FRB; src: url("lib/fonts/mathjax_fraktur-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SS; src: url("lib/fonts/mathjax_sansserif-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSB; src: url("lib/fonts/mathjax_sansserif-bold.woff") format("woff"); }
@font-face { font-family: MJXTEX-SSI; src: url("lib/fonts/mathjax_sansserif-italic.woff") format("woff"); }
@font-face { font-family: MJXTEX-SC; src: url("lib/fonts/mathjax_script-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-T; src: url("lib/fonts/mathjax_typewriter-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-V; src: url("lib/fonts/mathjax_vector-regular.woff") format("woff"); }
@font-face { font-family: MJXTEX-VB; src: url("lib/fonts/mathjax_vector-bold.woff") format("woff"); }
mjx-c.mjx-c1D440.TEX-I::before { padding: 0.683em 1.051em 0px 0px; content: "M"; }
mjx-c.mjx-c1D445.TEX-I::before { padding: 0.683em 0.759em 0.021em 0px; content: "R"; }
mjx-c.mjx-c1D436.TEX-I::before { padding: 0.705em 0.76em 0.022em 0px; content: "C"; }
mjx-c.mjx-c1D434.TEX-I::before { padding: 0.716em 0.75em 0px 0px; content: "A"; }
mjx-c.mjx-c3D::before { padding: 0.583em 0.778em 0.082em 0px; content: "="; }
mjx-c.mjx-cD7::before { padding: 0.491em 0.778em 0px 0px; content: "×"; }
mjx-c.mjx-c2F::before { padding: 0.75em 0.5em 0.25em 0px; content: "/"; }
</style><pre class="frontmatter language-yaml" tabindex="0" style="display: none;"><code class="is-loaded language-yaml"><span class="token key atrule">icon</span><span class="token punctuation">:</span> LiQuestionMarkGlyph</code><button class="copy-code-button">复制</button></pre><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="软件介绍"><div id="webpage-icon"><p dir="auto"><span class="cm-iconize-icon" aria-label="LiQuestionMarkGlyph" data-icon="LiQuestionMarkGlyph" aria-hidden="true" style="display: inline-flex; transform: translateY(13%);"><svg xmlns="http://www.w3.org/2000/svg" width="16px" height="16px" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="question-mark-glyph"><path d="M12 22C17.5228 22 22 17.5228 22 12C22 6.47715 17.5228 2 12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22Z"></path><path d="M9.09009 9.00003C9.32519 8.33169 9.78924 7.76813 10.4 7.40916C11.0108 7.05019 12.079 6.94542 12.7773 7.06519C13.9093 7.25935 14.9767 8.25497 14.9748 9.49073C14.9748 11.9908 12 11.2974 12 14"></path><path d="M12 17H12.01"></path></svg></span></p></div><p dir="auto">软件介绍</p></h1><div class="heading-wrapper"><div class="heading-children"><div><p dir="auto">PAML（Phylogenetic Analysis by Maximum Likelihood）是一款广泛使用的生物信息学软件，主要用于基于最大似然方法的分子序列数据的进化分析。这个工具主要用来估计核苷酸或氨基酸序列之间的进化关系，如<strong>分支长度的估计</strong>。PAML 特别适用于复杂的统计模型，如可变替换率和不同的进化模型。</p></div><div><div data-callout-metadata="" data-callout-fold="" data-callout="bug" class="callout"><div class="callout-title" dir="auto"><div class="callout-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon lucide-bug"><path d="m8 2 1.88 1.88"></path><path d="M14.12 3.88 16 2"></path><path d="M9 7.13v-1a3.003 3.003 0 1 1 6 0v1"></path><path d="M12 20c-3.3 0-6-2.7-6-6v-3a4 4 0 0 1 4-4h4a4 4 0 0 1 4 4v3c0 3.3-2.7 6-6 6"></path><path d="M12 20v-9"></path><path d="M6.53 9C4.6 8.8 3 7.1 3 5"></path><path d="M6 13H2"></path><path d="M3 21c0-2.1 1.7-3.9 3.8-4"></path><path d="M20.97 5c0 2.1-1.6 3.8-3.5 4"></path><path d="M22 13h-4"></path><path d="M17.2 17c2.1.1 3.8 1.9 3.8 4"></path></svg></div><div class="callout-title-inner">请注意，这个软件不适用于大规模数据的最大似然树建立!!!</div></div><div class="callout-content">
<p dir="auto">相反的，你应该需要使用其它建树软件建立最大似然树，并连同 <code>树文件</code> 和 <code>序列文件</code> 一同输入到 PAMLX 软件中进行计算。<br>
建树软件 <a data-href="iqtree：线粒体最大似然树" href="软件/iqtree/iqtree：线粒体最大似然树.html" class="internal-link" target="_self" rel="noopener"><span class="iconize-icon-in-link" title="🗣" aria-label="🗣" data-icon="🗣" aria-hidden="true" style="transform: translateY(0px);"><img class="emoji" draggable="false" alt="🗣" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/1f5e3.svg" height="16px" style="max-width: 100%;"></span>iqtree：线粒体最大似然树</a> 等</p>
</div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="软件使用" dir="auto" class="heading" id="软件使用">软件使用</h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="BaseML 方法" dir="auto" class="heading" id="BaseML_方法"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>BaseML 方法</h2><div class="heading-children"><div><p dir="auto"><img alt="image.png" src="https://picturerealm.oss-cn-chengdu.aliyuncs.com/obsidian/202409031655470.png" referrerpolicy="no-referrer" style="width: 500px; max-width: 100%;"></p></div><div class="heading-wrapper"><h3 data-heading="输入部分" dir="auto" class="heading" id="输入部分"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>输入部分</h3><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><strong>seqfile</strong>：指定序列数据文件的路径。</li>
<li data-line="1" dir="auto"><strong>treefile</strong>：指定进化树文件的路径。</li>
</ul></div></div></div><div class="heading-wrapper"><h3 data-heading="输出部分" dir="auto" class="heading" id="输出部分"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>输出部分</h3><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><strong>outfile name</strong>：输出文件的名称。</li>
<li data-line="1" dir="auto"><strong>location</strong>：输出文件的保存位置。</li>
</ul></div></div></div><div class="heading-wrapper"><h3 data-heading="参数配置" dir="auto" class="heading" id="参数配置"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>参数配置</h3><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>fix k</strong>：固定转换/颠换比率（κ）的值。如果选中，用户必须提供κ的值。请阅读<a data-href="图解核苷酸替换模型（Nucleotide substitution models）" href="术语/图解核苷酸替换模型（nucleotide-substitution-models）.html" class="internal-link" target="_self" rel="noopener">图解核苷酸替换模型（Nucleotide substitution models）</a>
<ul>
<li data-line="1" dir="auto"><strong>fix_kappa</strong> 用于指定是否固定 K80、F84 或 HKY85 模型中的转换与颠换比率参数 κ。</li>
<li data-line="2" dir="auto"><strong>如果 fix_kappa = 1</strong>，则 kappa 的值被固定；</li>
<li data-line="3" dir="auto">如果 fix_kappa = 0，则 kappa 的值用作迭代的初始估计值。</li>
<li data-line="4" dir="auto">对于 JC69 或 F81（没有 κ 参数）以及 TN93 和 REV（分别有两个和五个速率参数，且都从数据中估计）模型，fix_kappa 和 kappa 无效。</li>
<li data-line="5" dir="auto"><strong>fix_kappa = 2</strong> 与 nhomo = 5 一起用于 baseml 中指定非平稳模型，其中整个速率矩阵 Q（包括交换性参数和碱基频率参数）在不同分支上变化。</li>
</ul>
</li>
<li data-line="6" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>fix α</strong>：固定 gamma 分布形状参数（α）。如果选中，α的值为0，表示不考虑位点间的变异。
<ul>
<li data-line="7" dir="auto"><strong>fix_alpha</strong> 和 <strong>alpha</strong> 选项控制跨位点的可变替换速率的 gamma 分布的形状参数 α。当选中了 fix_alpha 且alpha = 0（0 表示无穷大）时，<strong>所有位点的替换率相同</strong>。正值的 alpha 表示使用离散 gamma 模型，ncatG 是离散 gamma 模型的类别数。</li>
</ul>
</li>
<li data-line="8" dir="auto"><strong>different alpha's for genes</strong>：为不同基因设置不同的 alpha 值。</li>
<li data-line="9" dir="auto"><strong>ncatG</strong>：离散 gamma 模型的类别数。</li>
<li data-line="10" dir="auto"><strong>fix ρ</strong>：固定相关性参数ρ。选中时，ρ的值为0，表示位点间的替换率独立。</li>
<li data-line="11" dir="auto"><strong>TipDate</strong>：如果进行时间相关的序列分析，此处可以输入样本的采集时间。</li>
<li data-line="12" dir="auto"><strong>time unit</strong>：时间单位，用于时间树分析。</li>
<li data-line="13" dir="auto"><strong>nparK</strong>：选择非参数速率模型的类型。</li>
<li data-line="14" dir="auto"><strong>Small_Diff</strong>：算法中使用的数值微分阈值。</li>
<li data-line="15" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>ndata</strong>：数据集的数量，用于多数据集分析。
<ul>
<li data-line="16" dir="auto">ndata 设置ndata 变量用于指定序列文件中数据集的数量，有助于处理模拟数据集或多基因对齐数据。具体设置如下：</li>
<li data-line="17" dir="auto"><strong>ndata = 100</strong>：所有数据集/对齐使用相同的树结构（一个树块）。</li>
<li data-line="18" dir="auto"><strong>ndata = 100 separate_trees</strong>：每个对齐有其自己的树块。</li>
<li data-line="19" dir="auto"><strong>ndata = 100 maintree</strong>：使用主树文件生成每个基因对齐的子树，并进行最大似然（ML）分析。</li>
<li data-line="20" dir="auto"><strong>ndata = 100 maintree 1</strong>：同上，进行ML分析。</li>
<li data-line="21" dir="auto"><strong>ndata = 100 maintree 0</strong>：生成基因树文件，但不运行ML分析。</li>
<li data-line="22" dir="auto">我们一般选择默认。</li>
</ul>
</li>
</ul></div></div></div><div class="heading-wrapper"><h3 data-heading="运行模式和模型设置" dir="auto" class="heading" id="运行模式和模型设置"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>运行模式和模型设置</h3><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>runmode</strong>：选择树的使用方式，0 表示使用用户提供的树。
<ul>
<li data-line="1" dir="auto"><strong>runmode = 0</strong>：评估树结构文件中指定的树拓扑。</li>
<li data-line="2" dir="auto"><strong>runmode = 1 和 2</strong>：通过启发式搜索算法构建进化树，其中1从多分枝树开始，2从星形树开始。</li>
<li data-line="3" dir="auto"><strong>runmode = 3</strong>：逐步添加法。</li>
<li data-line="4" dir="auto"><strong>runmode = 4 和 5</strong>：从给定或计算的树开始进行NNI（最近邻互换）扰动。</li>
<li data-line="5" dir="auto"><strong>不建议对大规模数据集执行搜树，因为该软件的速度不能满足需求</strong>！</li>
</ul>
</li>
<li data-line="6" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>model</strong>：选择核苷酸替换模型，7 表示使用 REV（GTR）模型。
<ul>
<li data-line="7" dir="auto"><strong>模型编号</strong>：0 至 8 对应不同的替换模型，如 JC69、K80、F81、HKY85 等。</li>
<li data-line="8" dir="auto"><strong>model = 9 和 10</strong>：分别对应特殊情况的 REV/GTR 和 UNREST 模型。</li>
<li data-line="9" dir="auto"><strong>自定义速率模型</strong>：如 UNRESTu 和 REVu，允许用户指定具有相同速率的核苷酸对。</li>
<li data-line="10" dir="auto">具体参见<a data-href="图解核苷酸替换模型（Nucleotide substitution models）" href="术语/图解核苷酸替换模型（nucleotide-substitution-models）.html" class="internal-link" target="_self" rel="noopener">图解核苷酸替换模型（Nucleotide substitution models）</a></li>
</ul>
</li>
<li data-line="11" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>clock</strong>：选择分子钟模型，0 表示不使用分子钟。
<ul>
<li data-line="12" dir="auto">clock 变量定义了分子钟假设的不同模型：</li>
<li data-line="13" dir="auto"><strong>clock = 0</strong>：无分子钟模型，分支速率完全自由变化，使用无根树。</li>
<li data-line="14" dir="auto"><strong>clock = 1</strong>：全局分子钟模型，所有分支速率相同，适用于有根树。</li>
<li data-line="15" dir="auto"><strong>clock = 2</strong>：局部分子钟模型，大多数分支符合钟模型，默认速率为1，但某些特定分支可有不同速率。</li>
<li data-line="16" dir="auto"><strong>clock = 3</strong>：用于多基因或多区段数据的分析，允许不同数据区段的分支速率以不同方式变化。</li>
<li data-line="17" dir="auto">对于人类的 Y 染色体和 mtDNA 来说，我们一般选择<strong>严格分子钟</strong>，在这里，也就是说我们选择1.另外，我们需要注意，如果我们输入的 DNA 中明显分区，那么我们就应该选择3.</li>
</ul>
</li>
<li data-line="18" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>Mgene</strong>：选择是否对基因或数据分区进行不同处理，0 表示速率相同。
<ul>
<li data-line="19" dir="auto">多基因或多位点分析（Mgene）<strong>Mgene = 0</strong>：未使用 G 选项，假设完全同质。</li>
<li data-line="20" dir="auto"><strong>Mgene = 1</strong>：分区模型，对每个基因或位点使用相同的模型。</li>
<li data-line="21" dir="auto"><strong>Mgene = 2、3、4</strong>：不同的设置允许基因之间具有不同的频率和速率参数。</li>
<li data-line="22" dir="auto">这点比较适合一段 DNA 序列包含多个分区的情况！</li>
</ul>
</li>
<li data-line="23" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>nhomo</strong>：选择是否使用非同质性模型，0 表示同质模型。
<ul>
<li data-line="24" dir="auto"><strong>nhomo = 0</strong>：基本替换模型，碱基频率参数通过观察频率平均值估计。</li>
<li data-line="25" dir="auto"><strong>nhomo = 1</strong>：同质模型，通过最大似然迭代估计频率参数 πT, πC 和 πA。</li>
<li data-line="26" dir="auto"><strong>nhomo = 2</strong>：每个树枝使用一个转换/颠换比率（κ），适用于 K80、F84 和 HKY85 模型。</li>
<li data-line="27" dir="auto"><strong>nhomo = 3, 4, 5</strong>：非同质模型，碱基组成在树上发生漂移，不同树枝使用不同的频率参数。</li>
<li data-line="28" dir="auto"><strong>nhomo = 3</strong>：每个末端分支、所有内部分支和根使用各自的一套频率参数。</li>
<li data-line="29" dir="auto"><strong>nhomo = 4</strong>：为根和每个分支设定不同的初始和末端频率参数。</li>
<li data-line="30" dir="auto"><strong>nhomo = 5</strong>：用户指定多套频率参数，并定义每个分支使用哪套参数。</li>
</ul>
</li>
<li data-line="31" dir="auto"><strong>fix blength</strong>：选择是否固定分支长度。</li>
<li data-line="32" dir="auto"><strong>optimization method</strong>：选择参数优化方法。</li>
<li data-line="33" dir="auto"><strong>icode</strong>：选择遗传密码表，用于编码序列。</li>
</ul></div></div></div><div class="heading-wrapper"><h3 data-heading="其他选项" dir="auto" class="heading" id="其他选项"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>其他选项</h3><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>getSE</strong>：计算参数的标准误差。
<ul>
<li data-line="1" dir="auto">建议勾选</li>
</ul>
</li>
<li data-line="2" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>RateAncestor</strong>：计算祖先节点的速率。
<ul>
<li data-line="3" dir="auto">可以勾选。即使勾选，也可以在结果中忽视。</li>
</ul>
</li>
<li data-line="4" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>clean data</strong>：在分析前清洗数据。
<ul>
<li data-line="5" dir="auto">可以勾选。</li>
</ul>
</li>
</ul></div></div></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="其他文献中的使用" dir="auto" class="heading" id="其他文献中的使用">其他文献中的使用</h1><div class="heading-children"><div><p dir="auto">让我们来看看别人如何使用。PAML 软件经常被用来计算<a data-href="最近共同祖先时间(Time to most recent common ancestor，MRCA)" href="术语/最近共同祖先时间(time-to-most-recent-common-ancestor，mrca).html" class="internal-link" target="_self" rel="noopener">最近共同祖先时间(Time to most recent common ancestor，MRCA)</a> 。怎么计算呢？<br>
首先，利用建树软件获得最大似然树（例如 RaxML-NG、<a data-href="iqtree：基础操作" href="软件/iqtree/iqtree：基础操作.html" class="internal-link" target="_self" rel="noopener">iqtree：基础操作</a>等），然后将序列与树文件丢给 PAML 计算最大似然值。获得的最大似然值可以<strong>被当作是遗传距离的一种指标</strong>。</p></div><div><div data-callout-metadata="" data-callout-fold="" data-callout="tip" class="callout"><div class="callout-title" dir="auto"><div class="callout-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon lucide-flame"><path d="M8.5 14.5A2.5 2.5 0 0 0 11 12c0-1.38-.5-2-1-3-1.072-2.143-.224-4.054 2-6 .5 2.5 2 4.9 4 6.5 2 1.6 3 3.5 3 5.5a7 7 0 1 1-14 0c0-1.153.433-2.294 1-3a2.5 2.5 0 0 0 2.5 2.5z"></path></svg></div><div class="callout-title-inner">当然，你也可以直接把序列之间的平均核苷酸差异数目当成是遗传距离的指标。并不一定说要把最大似然值作为遗传距离的指标。<a data-href="ρ（rho）方法" href="术语/ρ（rho）方法.html" class="internal-link" target="_self" rel="noopener">ρ（rho）方法</a> <a data-href="σ方法" href="术语/σ方法.html" class="internal-link" target="_self" rel="noopener">σ方法</a>。</div></div></div></div><div><p dir="auto">知道了遗传距离之后，就可以通过以下公式求得<a data-href="最近共同祖先时间(Time to most recent common ancestor，MRCA)" href="术语/最近共同祖先时间(time-to-most-recent-common-ancestor，mrca).html" class="internal-link" target="_self" rel="noopener">最近共同祖先时间(Time to most recent common ancestor，MRCA)</a>：</p></div><div><div class="math math-block is-loaded"><mjx-container class="MathJax" jax="CHTML" display="true"><mjx-math display="true" class="MJX-TEX" style="margin-left: 0px; margin-right: 0px;"><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D440 TEX-I"></mjx-c></mjx-mi><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D445 TEX-I"></mjx-c></mjx-mi><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D436 TEX-I"></mjx-c></mjx-mi><mjx-mi class="mjx-i"><mjx-c class="mjx-c1D434 TEX-I"></mjx-c></mjx-mi><mjx-mo class="mjx-n" space="4"><mjx-c class="mjx-c3D"></mjx-c></mjx-mo><mjx-mi class="mjx-n" space="4"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">遗</mjx-utext></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">传</mjx-utext></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">距</mjx-utext></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">离</mjx-utext></mjx-mi><mjx-mi class="mjx-i"><mjx-c class="mjx-cD7"></mjx-c></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">突</mjx-utext></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">变</mjx-utext></mjx-mi><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">率</mjx-utext></mjx-mi></mjx-math></mjx-container></div></div><div><p dir="auto">突变率往往在之前的研究中提到，单位一般是 <span class="math math-inline is-loaded"><mjx-container class="MathJax" jax="CHTML"><mjx-math class="MJX-TEX"><mjx-texatom texclass="ORD"><mjx-mo class="mjx-n"><mjx-c class="mjx-c2F"></mjx-c></mjx-mo></mjx-texatom><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">年</mjx-utext></mjx-mi></mjx-math></mjx-container></span> 或者 <span class="math math-inline is-loaded"><mjx-container class="MathJax" jax="CHTML"><mjx-math class="MJX-TEX"><mjx-texatom texclass="ORD"><mjx-mo class="mjx-n"><mjx-c class="mjx-c2F"></mjx-c></mjx-mo></mjx-texatom><mjx-mi class="mjx-n"><mjx-utext variant="normal" style="font-family: MJXZERO, serif;">代</mjx-utext></mjx-mi></mjx-math></mjx-container></span>。具体查看 <a data-href="2009 AJHG 净化选择校正：改进的人类线粒体分子钟" href="文献及报道/文献/2009-ajhg-净化选择校正：改进的人类线粒体分子钟.html" class="internal-link" target="_self" rel="noopener"><span class="iconize-icon-in-link" title="🥵" aria-label="🥵" data-icon="🥵" aria-hidden="true" style="transform: translateY(0px);"><img class="emoji" draggable="false" alt="🥵" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/1f975.svg" height="16px" style="max-width: 100%;"></span>2009 AJHG 净化选择校正：改进的人类线粒体分子钟</a>。</p></div><div class="heading-wrapper"><h2 data-heading="示例1" dir="auto" class="heading" id="示例1"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>示例1</h2><div class="heading-children"><div><blockquote dir="auto">
<p>In details, we used PAMLX v1.3.1 (Yang 1997) to calculate maximum-likelihood (ML)  estimates, assuming the HKY85 mutation model with gamma-distributed (32 categories) rates (plus  invariant sites) and two partitions: coding region (positions 00577–16023) and control region  (positions 16024–00576), as described elsewhere (Achilli et al. 2013). We converted ML mutational  distances into years by employing both the corrected molecular clock embedded in the calculator of  (Soares et al. 2009) and the linear mutation rate recently published of 2.7±0.2 x 10-8 base substitution  per nucleotide per year (Posth et al. 2016)<sup data-footnote-id="fnref-1-5ac8eaf4de2da317" class="footnote-ref" id="fnref-1-5ac8eaf4de2da317"><a data-footref="1" href="#fn-1-5ac8eaf4de2da317" class="footnote-link" target="_self" rel="noopener">[1]</a></sup>.</p>
</blockquote></div><div><p dir="auto">由上我们可知，该研究者使用该软件。选定了 <code>HKY85</code> 模型，并设定了32种突变速率。</p></div><div><div data-callout-metadata="" data-callout-fold="" data-callout="question" class="callout"><div class="callout-title" dir="auto"><div class="callout-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon lucide-help-circle"><circle cx="12" cy="12" r="10"></circle><path d="M9.09 9a3 3 0 0 1 5.83 1c0 2-3 3-3 3"></path><path d="M12 17h.01"></path></svg></div><div class="callout-title-inner">不明白模型的区别？看看这个：<a data-href="图解核苷酸替换模型（Nucleotide substitution models）" href="术语/图解核苷酸替换模型（nucleotide-substitution-models）.html" class="internal-link" target="_self" rel="noopener">图解核苷酸替换模型（Nucleotide substitution models）</a>。</div></div></div></div><div><p dir="auto">然后，作者将 mtDNA 分隔成2个区域，分别是<strong>控制区和编码区</strong>。计算得到了最大似然值的距离，然后根据分子钟转化为时间。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="示例2" dir="auto" class="heading" id="示例2"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>示例2</h2><div class="heading-children"><div><p dir="auto">加入我想选择最熟悉的 <code>GTR+G</code> 模型，应该如何设置呢？</p></div><div><ul>
<li data-line="0" dir="auto">首先，GTR 模型通常设置为：<code>model = 7</code>；</li>
<li data-line="1" dir="auto">设置分子钟模型为严格分子钟：<code>clock = 1</code>；</li>
<li data-line="2" dir="auto">启用伽马分布 (G) 和不变位点 (I)：如下，我们需要注意伽马分布的类别数需要自行设定，一个比较保险的方式是设定为 <code>5</code>. 当然，你也可以设定的更多。具体的优缺点请自行 <strong>chatgpt</strong>。</li>
</ul></div><div><pre class="language-txt" tabindex="0"><code class="is-loaded language-txt">fix_alpha = 0
alpha = 0.5
ncatG = 5
</code><button class="copy-code-button">复制</button></pre></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="参考文献" dir="auto" class="heading" id="参考文献">参考文献</h1><div class="heading-children"><div></div><div>
</div><div><section class="footnotes"><hr><ol>
<li data-line="0" data-footnote-id="fn-1-5ac8eaf4de2da317" id="fn-1-5ac8eaf4de2da317" dir="auto"><p>Mitogenome Diversity in Sardinians: A Genetic Window onto an Island's Past 附件<a href="#fnref-1-5ac8eaf4de2da317" class="footnote-backref footnote-link" target="_self" rel="noopener"><span><img class="emoji" draggable="false" alt="↩" src="https://cdn.jsdelivr.net/gh/jdecked/twemoji@15.1.0/assets/svg/21a9.svg" height="16px" style="max-width: 100%;"></span>︎</a></p></li>
</ol></section></div><div class="mod-footer"></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#软件介绍"><div class="tree-item-contents heading-link" heading-name="软件介绍"><span class="tree-item-title">软件介绍</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#软件使用"><div class="tree-item-contents heading-link" heading-name="软件使用"><span class="tree-item-title">软件使用</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item mod-collapsible" data-depth="2"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#BaseML_方法"><div class="tree-item-contents heading-link" heading-name="BaseML 方法"><div class="collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><span class="tree-item-title">BaseML 方法</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#输入部分"><div class="tree-item-contents heading-link" heading-name="输入部分"><span class="tree-item-title">输入部分</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#输出部分"><div class="tree-item-contents heading-link" heading-name="输出部分"><span class="tree-item-title">输出部分</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#参数配置"><div class="tree-item-contents heading-link" heading-name="参数配置"><span class="tree-item-title">参数配置</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#运行模式和模型设置"><div class="tree-item-contents heading-link" heading-name="运行模式和模型设置"><span class="tree-item-title">运行模式和模型设置</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#其他选项"><div class="tree-item-contents heading-link" heading-name="其他选项"><span class="tree-item-title">其他选项</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#其他文献中的使用"><div class="tree-item-contents heading-link" heading-name="其他文献中的使用"><span class="tree-item-title">其他文献中的使用</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#示例1"><div class="tree-item-contents heading-link" heading-name="示例1"><span class="tree-item-title">示例1</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#示例2"><div class="tree-item-contents heading-link" heading-name="示例2"><span class="tree-item-title">示例2</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\k-s\pamlx：最大似然计算.html#参考文献"><div class="tree-item-contents heading-link" heading-name="参考文献"><span class="tree-item-title">参考文献</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>