<!DOCTYPE html> <html><head>
		<title>序列比对（Align）软件评测</title>
		<base href="..\..\../">
		<meta id="root-path" root-path="..\..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 序列比对（Align）软件评测">
		<meta property="og:title" content="序列比对（Align）软件评测">
		<meta property="og:description" content="思维领域 - 序列比对（Align）软件评测">
		<meta property="og:type" content="website">
		<meta property="og:url" content="软件/其它生信软件/格式比对/序列比对（align）软件评测.html">
		<meta property="og:image" content="undefined">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon h2-underline"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles"></style><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="ClustalW"><p dir="auto">ClustalW</p></h1><div class="heading-wrapper"><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="版本" dir="auto" class="heading" id="版本"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>版本</h2><div class="heading-children"><div><p dir="auto"><a data-tooltip-position="top" aria-label="http://www.clustal.org/" rel="noopener" class="external-link" href="http://www.clustal.org/" target="_blank">这个软件</a>有多个版本，包括<br>
GUI (ClustalX), command line (ClustalW), web server。</p></div></div></div><div class="heading-wrapper"><h2 data-heading="说明书" dir="auto" class="heading" id="说明书"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>说明书</h2><div class="heading-children"><div><p dir="auto">这里是说明书：</p></div><div><pre class="language-shell" tabindex="0"><code class="language-shell is-loaded"><span class="token operator">&gt;&gt;</span> HELP <span class="token number">9</span> <span class="token operator">&lt;&lt;</span>             Help <span class="token keyword">for</span> <span class="token builtin class-name">command</span> line parameters

                DATA <span class="token punctuation">(</span>sequences<span class="token punctuation">)</span>

-INFILE<span class="token operator">=</span>file.ext                             :input sequences.
-PROFILE1<span class="token operator">=</span>file.ext  and  -PROFILE2<span class="token operator">=</span>file.ext  :profiles <span class="token punctuation">(</span>old alignment<span class="token punctuation">)</span>.


                VERBS <span class="token punctuation">(</span>do things<span class="token punctuation">)</span>

-OPTIONS            :list the <span class="token builtin class-name">command</span> line parameters
-HELP  or -CHECK    :outline the <span class="token builtin class-name">command</span> line params.
-FULLHELP           :output full <span class="token builtin class-name">help</span> content.
-ALIGN              :do full multiple alignment. <span class="token comment"># 这个可能有用</span>
-TREE               :calculate NJ tree.
-PIM                :output percent identity matrix <span class="token punctuation">(</span>while calculating the tree<span class="token punctuation">)</span>
-BOOTSTRAP<span class="token punctuation">(</span><span class="token operator">=</span>n<span class="token punctuation">)</span>      :bootstrap a NJ tree <span class="token punctuation">(</span>n<span class="token operator">=</span> number of bootstraps<span class="token punctuation">;</span> def. <span class="token operator">=</span> <span class="token number">1000</span><span class="token punctuation">)</span>.
-CONVERT            :output the input sequences <span class="token keyword">in</span> a different <span class="token function">file</span> format.


                PARAMETERS <span class="token punctuation">(</span>set things<span class="token punctuation">)</span>

***General settings:****
-INTERACTIVE :read <span class="token builtin class-name">command</span> line, <span class="token keyword">then</span> enter normal interactive menus
-QUICKTREE   :use FAST algorithm <span class="token keyword">for</span> the alignment guide tree
-TYPE<span class="token operator">=</span>       :PROTEIN or DNA sequences
-NEGATIVE    :protein alignment with negative values <span class="token keyword">in</span> matrix
-OUTFILE<span class="token operator">=</span>    :sequence alignment <span class="token function">file</span> name
-OUTPUT<span class="token operator">=</span>     :CLUSTAL<span class="token punctuation">(</span>default<span class="token punctuation">)</span>, GCG, GDE, PHYLIP, PIR, NEXUS and FASTA
-OUTORDER<span class="token operator">=</span>   :INPUT or ALIGNED
-CASE        :LOWER or UPPER <span class="token punctuation">(</span>for GDE output only<span class="token punctuation">)</span>
-SEQNOS<span class="token operator">=</span>     :OFF or ON <span class="token punctuation">(</span>for Clustal output only<span class="token punctuation">)</span>
-SEQNO_RANGE<span class="token operator">=</span>:OFF or ON <span class="token punctuation">(</span>NEW: <span class="token keyword">for</span> all output formats<span class="token punctuation">)</span>
-RANGE<span class="token operator">=</span>m,n   :sequence range to <span class="token function">write</span> starting m to m+n
-MAXSEQLEN<span class="token operator">=</span>n :maximum allowed input sequence length
-QUIET       :Reduce console output to minimum
-STATS<span class="token operator">=</span>      :Log some alignents statistics to <span class="token function">file</span>

***Fast Pairwise Alignments:***
-KTUPLE<span class="token operator">=</span>n    :word size
-TOPDIAGS<span class="token operator">=</span>n  :number of best diags.
-WINDOW<span class="token operator">=</span>n    :window around best diags.
-PAIRGAP<span class="token operator">=</span>n   :gap penalty
-SCORE       :PERCENT or ABSOLUTE


***Slow Pairwise Alignments:***
-PWMATRIX<span class="token operator">=</span>    :Protein weight <span class="token assign-left variable">matrix</span><span class="token operator">=</span>BLOSUM, PAM, GONNET, ID or filename
-PWDNAMATRIX<span class="token operator">=</span> :DNA weight <span class="token assign-left variable">matrix</span><span class="token operator">=</span>IUB, CLUSTALW or filename
-PWGAPOPEN<span class="token operator">=</span>f  :gap opening penalty        
-PWGAPEXT<span class="token operator">=</span>f   :gap opening penalty


***Multiple Alignments:***
-NEWTREE<span class="token operator">=</span>      :file <span class="token keyword">for</span> new guide tree
-USETREE<span class="token operator">=</span>      :file <span class="token keyword">for</span> old guide tree
-MATRIX<span class="token operator">=</span>       :Protein weight <span class="token assign-left variable">matrix</span><span class="token operator">=</span>BLOSUM, PAM, GONNET, ID or filename
-DNAMATRIX<span class="token operator">=</span>    :DNA weight <span class="token assign-left variable">matrix</span><span class="token operator">=</span>IUB, CLUSTALW or filename
-GAPOPEN<span class="token operator">=</span>f     :gap opening penalty        
-GAPEXT<span class="token operator">=</span>f      :gap extension penalty
-ENDGAPS       :no end gap separation pen. 
-GAPDIST<span class="token operator">=</span>n     :gap separation pen. range
-NOPGAP        :residue-specific gaps off  
-NOHGAP        :hydrophilic gaps off
-HGAPRESIDUES<span class="token operator">=</span> :list hydrophilic res.    
-MAXDIV<span class="token operator">=</span>n      :% ident. <span class="token keyword">for</span> delay
-TYPE<span class="token operator">=</span>         :PROTEIN or DNA
-TRANSWEIGHT<span class="token operator">=</span>f :transitions weighting
-ITERATION<span class="token operator">=</span>    :NONE or TREE or ALIGNMENT
-NUMITER<span class="token operator">=</span>n     :maximum number of iterations to perform
-NOWEIGHTS     :disable sequence weighting


***Profile Alignments:***
-PROFILE      :Merge two alignments by profile alignment
-NEWTREE1<span class="token operator">=</span>    :file <span class="token keyword">for</span> new guide tree <span class="token keyword">for</span> profile1
-NEWTREE2<span class="token operator">=</span>    :file <span class="token keyword">for</span> new guide tree <span class="token keyword">for</span> profile2
-USETREE1<span class="token operator">=</span>    :file <span class="token keyword">for</span> old guide tree <span class="token keyword">for</span> profile1
-USETREE2<span class="token operator">=</span>    :file <span class="token keyword">for</span> old guide tree <span class="token keyword">for</span> profile2


***Sequence to Profile Alignments:***
-SEQUENCES   :Sequentially <span class="token function">add</span> profile2 sequences to profile1 alignment
-NEWTREE<span class="token operator">=</span>    :file <span class="token keyword">for</span> new guide tree
-USETREE<span class="token operator">=</span>    :file <span class="token keyword">for</span> old guide tree


***Structure Alignments:***
-NOSECSTR1     :do not use secondary structure-gap penalty mask <span class="token keyword">for</span> profile <span class="token number">1</span> 
-NOSECSTR2     :do not use secondary structure-gap penalty mask <span class="token keyword">for</span> profile <span class="token number">2</span>
-SECSTROUT<span class="token operator">=</span>STRUCTURE or MASK or BOTH or NONE   :output <span class="token keyword">in</span> alignment <span class="token function">file</span>
-HELIXGAP<span class="token operator">=</span>n    :gap penalty <span class="token keyword">for</span> helix core residues 
-STRANDGAP<span class="token operator">=</span>n   :gap penalty <span class="token keyword">for</span> strand core residues
-LOOPGAP<span class="token operator">=</span>n     :gap penalty <span class="token keyword">for</span> loop regions
-TERMINALGAP<span class="token operator">=</span>n :gap penalty <span class="token keyword">for</span> structure termini
-HELIXENDIN<span class="token operator">=</span>n  :number of residues inside helix to be treated as terminal
-HELIXENDOUT<span class="token operator">=</span>n :number of residues outside helix to be treated as terminal
-STRANDENDIN<span class="token operator">=</span>n :number of residues inside strand to be treated as terminal
-STRANDENDOUT<span class="token operator">=</span>n:number of residues outside strand to be treated as terminal 


***Trees:***
-OUTPUTTREE<span class="token operator">=</span>nj OR phylip OR dist OR nexus
-SEED<span class="token operator">=</span>n        :seed number <span class="token keyword">for</span> bootstraps.
-KIMURA        :use Kimura's correction.   
-TOSSGAPS      :ignore positions with gaps.
-BOOTLABELS<span class="token operator">=</span>node OR branch :position of bootstrap values <span class="token keyword">in</span> tree display
-CLUSTERING<span class="token operator">=</span>   :NJ or UPGMA
</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">在Linux系统下，这个软件的操作命令如下：</p></div><div><pre class="language-shell" tabindex="0"><code class="language-shell is-loaded"><span class="token comment"># 进入目录,进行Paire比对</span>
clustalw2 -INFILE<span class="token operator">=</span>/home/luolintao/mt_DNA_whole/DQ372883.fasta -OUTFILE<span class="token operator">=</span>/home/luolintao/序列比对/output.aln

<span class="token comment"># 进行Multi比对</span>
clustalw2 -INFILE<span class="token operator">=</span>/home/luolintao/mt_DNA_whole/DQ372883.fasta -OUTFILE<span class="token operator">=</span>/home/luolintao/序列比对/output.aln -OUTPUT<span class="token operator">=</span>CLUSTAL -ALIGN
</code><button class="copy-code-button">复制</button></pre></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="Clustal Omega" dir="auto" class="heading" id="Clustal_Omega">Clustal Omega</h1><div class="heading-children"><div><p dir="auto">上个软件的进化版<a data-tooltip-position="top" aria-label="http://www.clustal.org/omega/" rel="noopener" class="external-link" href="http://www.clustal.org/omega/" target="_blank">Clustal Omega - fast, accurate, scalable multiple sequence alignment for proteins</a>。新的 HMM 比对引擎而具有更高的准确性。数小时内比对数十万个序列。</p></div><div class="heading-wrapper"><h2 data-heading="版本" dir="auto" class="heading" id="版本"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>版本</h2><div class="heading-children"><div><p dir="auto">只有CLI。可以使用Win、Mac、Linux。</p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="MAFFT" dir="auto" class="heading" id="MAFFT">MAFFT</h1><div class="heading-children"><div><p dir="auto">MAFFT 是一个用于类 UNIX 操作系统的多序列比对程序。它提供了一系列多重比对方法，L-INS-i（精确；用于&lt;∼200个序列的比对）、FFT-NS-2（快速；用于&lt;∼30,000个序列的比对）等。<br>
<a data-tooltip-position="top" aria-label="https://mafft.cbrc.jp/alignment/software/" rel="noopener" class="external-link" href="https://mafft.cbrc.jp/alignment/software/" target="_blank">MAFFT - a multiple sequence alignment program (cbrc.jp)</a></p></div><div class="heading-wrapper"><h2 data-heading="版本" dir="auto" class="heading" id="版本"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>版本</h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><strong><a data-tooltip-position="top" aria-label="https://mafft.cbrc.jp/alignment/software/macosx.html" rel="noopener" class="external-link" href="https://mafft.cbrc.jp/alignment/software/macosx.html" target="_blank">Mac OS X</a></strong>&nbsp;&nbsp;Mac OS X</li>
<li data-line="1" dir="auto"><strong><a data-tooltip-position="top" aria-label="https://mafft.cbrc.jp/alignment/software/linux.html" rel="noopener" class="external-link" href="https://mafft.cbrc.jp/alignment/software/linux.html" target="_blank">Linux</a></strong>&nbsp;&nbsp;Linux</li>
<li data-line="2" dir="auto"><strong><a data-tooltip-position="top" aria-label="https://mafft.cbrc.jp/alignment/software/windows.html" rel="noopener" class="external-link" href="https://mafft.cbrc.jp/alignment/software/windows.html" target="_blank">Windows</a></strong>&nbsp;&nbsp;视窗</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="用法" dir="auto" class="heading" id="用法"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div>用法</h2><div class="heading-children"><div><pre class="language-shell" tabindex="0"><code class="language-shell is-loaded">% mafft <span class="token punctuation">[</span>arguments<span class="token punctuation">]</span> input <span class="token operator">&gt;</span> output
% mafft-linsi input <span class="token operator">&gt;</span> output
% mafft input <span class="token operator">&gt;</span> output
% mafft --auto input <span class="token operator">&gt;</span> output <span class="token comment"># 自动选择参数</span>
</code><button class="copy-code-button">复制</button></pre></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="~~比对软件~~" dir="auto" class="heading" id="~~比对软件~~"><del>比对软件</del></h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="~~MUMmer~~" dir="auto" class="heading" id="~~MUMmer~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>MUMmer</del></h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><div class="list-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del><strong>MUMmer</strong>是一种快速、准确的DNA序列比对工具。它可以用于比对多个长度有细微差异的fasta序列。MUMmer使用一种称为“最小唯一匹配”(MUM)的算法来进行比对。MUM算法可以快速找到两个序列之间的最大公共子序列(MCS)。</del>
<ul>
<li data-line="1" dir="auto"><del><strong>MUMmer 3.0 can find all 20-basepair or longer exact matches between a pair of 5-megabase genomes in 13.7 seconds</strong>, using 78 MB of memory, on a 2.4 GHz Linux desktop computer.<strong>10条序列需要1分29秒</strong>。</del></li>
<li data-line="2" dir="auto"><del>MUMmer 3.0 is described in our&nbsp;<a data-tooltip-position="top" aria-label="https://mummer.sourceforge.net/MUMmer3.pdf" rel="noopener" class="external-link" href="https://mummer.sourceforge.net/MUMmer3.pdf" target="_blank">2004&nbsp;<em>Genome Biology</em>&nbsp;paper</a>. We also developed a GPU-accelerated version of MUMmer called&nbsp;<a data-tooltip-position="top" aria-label="http://mummergpu.sf.net/" rel="noopener" class="external-link" href="http://mummergpu.sf.net/" target="_blank">MUMmerGPU</a>.</del></li>
</ul>
</li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="~~ClustalW~~" dir="auto" class="heading" id="~~ClustalW~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>ClustalW</del></h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><del><strong>ClustalW</strong>是一种常用的多序列比对工具。它可以用于比对多个长度有细微差异的fasta序列。ClustalW使用一种称为“渐进式比对”的算法来进行比对。渐进式比对算法将多个序列逐个进行比对，最终得到一个多序列比对结果。- "Classic Clustal"&nbsp;“经典星团”- <strong>GUI (ClustalX), command line (ClustalW), web server versions available</strong>      <strong>GUI (ClustalX)、命令行 (ClustalW)、可用的 Web 服务器版本（web服务器限制Too large! datasize&gt;5,000,000 bytes）</strong>。<strong>10条序列需要1分49秒</strong>。</del></li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="~~Clustal Omega~~" dir="auto" class="heading" id="~~Clustal_Omega~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>Clustal Omega</del></h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><del><strong>Clustal Omega（ClustalΩ）</strong>: Clustal Omega是一个流行的序列比对工具，它支持多序列比对，适用于蛋白质和核酸序列。Clustal Omega特别适合大规模数据集的比对，能够快速准确地处理大量序列。# <a data-tooltip-position="top" aria-label="http://www.clustal.org/omega/" rel="noopener" class="external-link" href="http://www.clustal.org/omega/" target="_blank">Clustal Omega</a>&nbsp;水晶欧米茄- Latest version of Clustal - fast and scalable (can align hundreds of thousands of sequences in hours), greater accuracy due to new HMM alignment engine      最新版本的 Clustal - 快速且可扩展（<strong>可在数小时内比对数十万个序列</strong>），由于新的 <strong>HMM 比对引擎</strong>而具有更高的准确性- Command line/web server only (GUI public beta available soon)    仅<strong>命令行/Web 服务器</strong>（GUI 公共测试版即将推出）.<strong>速度：41s/10条</strong>。默认参数。</del></li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="~~EMBOSS:cons~~" dir="auto" class="heading" id="~~EMBOSS:cons~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>EMBOSS:cons</del></h2><div class="heading-children"><div><p dir="auto"><del>Web，CLI，Web限制2MB</del></p></div></div></div><div class="heading-wrapper"><h2 data-heading="~~MAFFT~~" dir="auto" class="heading" id="~~MAFFT~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>MAFFT</del></h2><div class="heading-children"><div><ul>
<li data-line="0" dir="auto"><del><strong>MAFFT</strong>: MAFFT是一种快速且高效的多序列比对工具，适用于大规模序列数据。它提供了多种比对策略，包括FFT-NS-2、FFT-NS-i等，用户可以根据需要选择最适合自己数据的比对策略。</del></li>
<li data-line="1" dir="auto"><del>可以在Mac、Linux、Windows运行。支持CLI以及GUI，但是GUI比较慢且可选参数较少。L-INS-i（精确；用于&lt;∼200个序列的比对，<strong>4s10个序列</strong>）、FFT-NS-2（快速；用于&lt;∼30,000个序列的比对。<strong>1秒10个序列</strong>）等。</del></li>
</ul></div></div></div><div class="heading-wrapper"><h2 data-heading="~~MUSCLE 5.0~~" dir="auto" class="heading" id="~~MUSCLE_5.0~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>MUSCLE 5.0</del></h2><div class="heading-children"><div><p dir="auto"> <del><strong>MUSCLE</strong>: MUSCLE（Multiple Sequence Comparison by Log-Expectation）是一种用于蛋白质和核酸序列比对的软件，它以其<strong>高效和高准确性</strong>著称。MUSCLE适合用于大量序列的比对，尤其是当准确性非常重要时。它可以在低成本商用计算机（例如，具有 32 Gb RAM 的 8 核 Intel CPU）上<strong>高精度</strong>地对齐数万个序列。在大型数据集上，Muscle v5 的<strong>准确度</strong>比 MAFFT 和 Clustal-Omega 高 20-30%。Muscle <strong>没有图形用户界面 (GUI)</strong>。支持<strong>win,linux</strong>。速度：<strong>1分52秒完成10条。super5模式下</strong>。<strong>19分21秒完成10条，PPP模式下</strong>。</del></p></div></div></div></div></div><div class="heading-wrapper"><h1 data-heading="~~T-Coffee~~" dir="auto" class="heading" id="~~T-Coffee~~"><del>T-Coffee</del></h1><div class="heading-children"><div><p dir="auto"><del><strong>T-Coffee</strong>: T-Coffee是一种多序列比对工具，它可以用于不同类型的序列比对任务，包括蛋白质序列、核酸序列和结构比对。T-Coffee试图在保持高准确度的同时最大化比对的速度和效率。它还能够将序列信息与蛋白质结构信息 (3D-Coffee/Expresso)、概况信息 (PSI-Coffee) 或 RNA 二级结构 (R-Coffee) 相结合。支持网页版服务，但是You can enter <strong>at most 150 sequences</strong>。<strong>只能在Linux和Mac中运行，CLI</strong>。渐进式比对算法将多个序列逐个进行比对，最终得到一个多序列比对结果。<strong>10条序列1分40秒</strong>。<strong>上传限制：2500 char length</strong></del></p></div><div class="heading-wrapper"><h2 data-heading="~~MEGA~~" dir="auto" class="heading" id="~~MEGA~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>MEGA</del></h2><div class="heading-children"><div><p dir="auto"><del><strong>MEGA</strong>: MEGA（Molecular Evolutionary Genetics Analysis）是一种集成工具，提供了系统发育分析、序列比对、分子钟分析等多种功能。它的用户界面友好，非常适合生物信息学和系统发育学的初学者使用。<strong>有GUI，有CLI，在Win、Linux和Mac均能运行</strong>。但是align方法用的是ClustalW和MUSCLE，所以不再进行比较。</del></p></div></div></div><div class="heading-wrapper"><h2 data-heading="~~HAlign~~" dir="auto" class="heading" id="~~HAlign~~"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><del>HAlign</del></h2><div class="heading-children"><div><p dir="auto"><del>HAlign是一个多平台Java软件工具包，旨在大规模多个相似的DNA/RNA/Protein序列比对。&nbsp;center star multiple sequence alignment strategy。<strong>支持网页端上传</strong>但是我登不上去。支持下载，可以在<strong>Win、Linux和Mac系统</strong>运行，需要Java环境。</del></p></div><div class="mod-footer"></div></div></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#ClustalW"><div class="tree-item-contents heading-link" heading-name="ClustalW"><span class="tree-item-title">ClustalW</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#版本"><div class="tree-item-contents heading-link" heading-name="版本"><span class="tree-item-title">版本</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#说明书"><div class="tree-item-contents heading-link" heading-name="说明书"><span class="tree-item-title">说明书</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#Clustal_Omega"><div class="tree-item-contents heading-link" heading-name="Clustal Omega"><span class="tree-item-title">Clustal Omega</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#版本"><div class="tree-item-contents heading-link" heading-name="版本"><span class="tree-item-title">版本</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#MAFFT"><div class="tree-item-contents heading-link" heading-name="MAFFT"><span class="tree-item-title">MAFFT</span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#版本"><div class="tree-item-contents heading-link" heading-name="版本"><span class="tree-item-title">版本</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#用法"><div class="tree-item-contents heading-link" heading-name="用法"><span class="tree-item-title">用法</span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~比对软件~~"><div class="tree-item-contents heading-link" heading-name="~~比对软件~~"><span class="tree-item-title"><del>比对软件</del></span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~MUMmer~~"><div class="tree-item-contents heading-link" heading-name="~~MUMmer~~"><span class="tree-item-title"><del>MUMmer</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~ClustalW~~"><div class="tree-item-contents heading-link" heading-name="~~ClustalW~~"><span class="tree-item-title"><del>ClustalW</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~Clustal_Omega~~"><div class="tree-item-contents heading-link" heading-name="~~Clustal Omega~~"><span class="tree-item-title"><del>Clustal Omega</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~EMBOSS:cons~~"><div class="tree-item-contents heading-link" heading-name="~~EMBOSS:cons~~"><span class="tree-item-title"><del>EMBOSS:cons</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~MAFFT~~"><div class="tree-item-contents heading-link" heading-name="~~MAFFT~~"><span class="tree-item-title"><del>MAFFT</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~MUSCLE_5.0~~"><div class="tree-item-contents heading-link" heading-name="~~MUSCLE 5.0~~"><span class="tree-item-title"><del>MUSCLE 5.0</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~T-Coffee~~"><div class="tree-item-contents heading-link" heading-name="~~T-Coffee~~"><span class="tree-item-title"><del>T-Coffee</del></span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~MEGA~~"><div class="tree-item-contents heading-link" heading-name="~~MEGA~~"><span class="tree-item-title"><del>MEGA</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\格式比对\序列比对（align）软件评测.html#~~HAlign~~"><div class="tree-item-contents heading-link" heading-name="~~HAlign~~"><span class="tree-item-title"><del>HAlign</del></span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>