<!DOCTYPE html> <html><head>
		<title>线粒体基因组分析流程</title>
		<base href="..\..\../">
		<meta id="root-path" root-path="..\..\../">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes, minimum-scale=1.0, maximum-scale=5.0">
		<meta charset="UTF-8">
		<meta name="description" content="思维领域 - 线粒体基因组分析流程">
		<meta property="og:title" content="线粒体基因组分析流程">
		<meta property="og:description" content="思维领域 - 线粒体基因组分析流程">
		<meta property="og:type" content="website">
		<meta property="og:url" content="软件/其它生信软件/使用心得/线粒体基因组分析流程.html">
		<meta property="og:image" content="https://cdn.nlark.com/yuque/0/2024/png/40551061/1709024234747-a4d99195-63f7-48e6-b473-6de9a5d44ed7.png#averageHue=%23fdfdfc&amp;clientId=uf2cae3a5-51f1-4&amp;from=paste&amp;id=u416810a9&amp;originHeight=444&amp;originWidth=853&amp;originalType=url&amp;ratio=1.5&amp;rotation=0&amp;showTitle=false&amp;status=done&amp;style=none&amp;taskId=u1df18701-586a-4e09-81f0-45c3df948a4&amp;title=">
		<meta property="og:site_name" content="思维领域">
		<link rel="alternate" type="application/rss+xml" title="RSS Feed" href="lib/rss.xml"><script async="" id="webpage-script" src="lib/scripts/webpage.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script type="module" async="" id="graph-view-script" src="lib/scripts/graph-view.js"></script><script async="" id="graph-wasm-script" src="lib/scripts/graph-wasm.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="graph-render-worker-script" src="lib/scripts/graph-render-worker.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="tinycolor-script" src="lib/scripts/tinycolor.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="pixi-script" src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/7.4.0/pixi.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><script async="" id="minisearch-script" src="https://cdn.jsdelivr.net/npm/minisearch@6.3.0/dist/umd/index.min.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><link rel="icon" href="lib/media/favicon.png"><script async="" id="graph-data-script" src="lib/scripts/graph-data.js" onload="this.onload=null;this.setAttribute(&quot;loaded&quot;, &quot;true&quot;)"></script><style>body{--line-width:40em;--line-width-adaptive:40em;--file-line-width:40em;--sidebar-width:min(20em, 80vw);--collapse-arrow-size:11px;--tree-horizontal-spacing:0.6em;--tree-vertical-spacing:0.6em;--sidebar-margin:12px}.sidebar{height:100%;min-width:calc(var(--sidebar-width) + var(--divider-width-hover));max-width:calc(var(--sidebar-width) + var(--divider-width-hover));font-size:14px;z-index:10;position:relative;overflow:hidden;transition:min-width ease-in-out,max-width ease-in-out;transition-duration:.2s;contain:size}.sidebar-left{left:0}.sidebar-right{right:0}.sidebar.is-collapsed{min-width:0;max-width:0}body.floating-sidebars .sidebar{position:absolute}.sidebar-content{height:100%;min-width:calc(var(--sidebar-width) - var(--divider-width-hover));top:0;padding:var(--sidebar-margin);padding-top:4em;line-height:var(--line-height-tight);background-color:var(--background-secondary);transition:background-color,border-right,border-left,box-shadow;transition-duration:var(--color-fade-speed);transition-timing-function:ease-in-out;position:absolute;display:flex;flex-direction:column}.sidebar:not(.is-collapsed) .sidebar-content{min-width:calc(max(100%,var(--sidebar-width)) - 3px);max-width:calc(max(100%,var(--sidebar-width)) - 3px)}.sidebar-left .sidebar-content{left:0;border-top-right-radius:var(--radius-l);border-bottom-right-radius:var(--radius-l)}.sidebar-right .sidebar-content{right:0;border-top-left-radius:var(--radius-l);border-bottom-left-radius:var(--radius-l)}.sidebar:has(.sidebar-content:empty):has(.topbar-content:empty){display:none}.sidebar-topbar{height:2em;width:var(--sidebar-width);top:var(--sidebar-margin);padding-inline:var(--sidebar-margin);z-index:1;position:fixed;display:flex;align-items:center;transition:width ease-in-out;transition-duration:inherit}.sidebar.is-collapsed .sidebar-topbar{width:calc(2.3em + var(--sidebar-margin) * 2)}.sidebar .sidebar-topbar.is-collapsed{width:0}.sidebar-left .sidebar-topbar{left:0}.sidebar-right .sidebar-topbar{right:0}.topbar-content{overflow:hidden;overflow:clip;width:100%;height:100%;display:flex;align-items:center;transition:inherit}.sidebar.is-collapsed .topbar-content{width:0;transition:inherit}.clickable-icon.sidebar-collapse-icon{background-color:transparent;color:var(--icon-color-focused);padding:0!important;margin:0!important;height:100%!important;width:2.3em!important;margin-inline:0.14em!important;position:absolute}.sidebar-left .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);right:var(--sidebar-margin)}.sidebar-right .clickable-icon.sidebar-collapse-icon{transform:rotateY(180deg);left:var(--sidebar-margin)}.clickable-icon.sidebar-collapse-icon svg.svg-icon{width:100%;height:100%}.sidebar-section-header{margin:0 0 1em 0;text-transform:uppercase;letter-spacing:.06em;font-weight:600}body{transition:background-color var(--color-fade-speed) ease-in-out}.webpage-container{display:flex;flex-direction:row;height:100%;width:100%;align-items:stretch;justify-content:center}.document-container{opacity:1;flex-basis:100%;max-width:100%;width:100%;height:100%;display:flex;flex-direction:column;align-items:center;transition:opacity .2s ease-in-out;contain:inline-size}.hide{opacity:0;transition:opacity .2s ease-in-out}.document-container>.markdown-preview-view{margin:var(--sidebar-margin);margin-bottom:0;width:100%;width:-webkit-fill-available;width:-moz-available;width:fill-available;background-color:var(--background-primary);transition:background-color var(--color-fade-speed) ease-in-out;border-top-right-radius:var(--window-radius,var(--radius-m));border-top-left-radius:var(--window-radius,var(--radius-m));overflow-x:hidden!important;overflow-y:auto!important;display:flex!important;flex-direction:column!important;align-items:center!important;contain:inline-size}.document-container>.markdown-preview-view>.markdown-preview-sizer{padding-bottom:80vh!important;width:100%!important;max-width:var(--line-width)!important;flex-basis:var(--line-width)!important;transition:background-color var(--color-fade-speed) ease-in-out;contain:inline-size}.markdown-rendered img:not([width]),.view-content img:not([width]){max-width:100%;outline:0}.document-container>.view-content.embed{display:flex;padding:1em;height:100%;width:100%;align-items:center;justify-content:center}.document-container>.view-content.embed>*{max-width:100%;max-height:100%;object-fit:contain}:has(> :is(.math,table)){overflow-x:auto!important}.document-container>.view-content{overflow-x:auto;contain:content;padding:0;margin:0;height:100%}.scroll-highlight{position:absolute;width:100%;height:100%;pointer-events:none;z-index:1000;background-color:hsla(var(--color-accent-hsl),.25);opacity:0;padding:1em;inset:50%;translate:-50% -50%;border-radius:var(--radius-s)}</style><script defer="">async function loadIncludes(){if("file:"!=location.protocol){let e=document.querySelectorAll("include");for(let t=0;t<e.length;t++){let o=e[t],l=o.getAttribute("src");try{const e=await fetch(l);if(!e.ok){console.log("Could not include file: "+l),o?.remove();continue}let t=await e.text(),n=document.createRange().createContextualFragment(t),i=Array.from(n.children);for(let e of i)e.classList.add("hide"),e.style.transition="opacity 0.5s ease-in-out",setTimeout((()=>{e.classList.remove("hide")}),10);o.before(n),o.remove(),console.log("Included file: "+l)}catch(e){o?.remove(),console.log("Could not include file: "+l,e);continue}}}else{if(document.querySelectorAll("include").length>0){var e=document.createElement("div");e.id="error",e.textContent="Web server exports must be hosted on an http / web server to be viewed correctly.",e.style.position="fixed",e.style.top="50%",e.style.left="50%",e.style.transform="translate(-50%, -50%)",e.style.fontSize="1.5em",e.style.fontWeight="bold",e.style.textAlign="center",document.body.appendChild(e),document.querySelector(".document-container")?.classList.remove("hide")}}}document.addEventListener("DOMContentLoaded",(()=>{loadIncludes()}));let isFileProtocol="file:"==location.protocol;function waitLoadScripts(e,t){let o=e.map((e=>document.getElementById(e+"-script"))),l=0;!function e(){let n=o[l];l++,n&&"true"!=n.getAttribute("loaded")||l<o.length&&e(),l<o.length?n.addEventListener("load",e):t()}()}</script><link rel="stylesheet" href="lib/styles/obsidian.css"><link rel="preload" href="lib/styles/other-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/other-plugins.css"></noscript><link rel="stylesheet" href="lib/styles/theme.css"><link rel="preload" href="lib/styles/global-variable-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/global-variable-styles.css"></noscript><link rel="preload" href="lib/styles/supported-plugins.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/supported-plugins.css"></noscript><link rel="preload" href="lib/styles/main-styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"><noscript><link rel="stylesheet" href="lib/styles/main-styles.css"></noscript></head><body class="publish css-settings-manager theme-dark show-inline-title show-ribbon h2-underline"><script defer="">let theme=localStorage.getItem("theme")||(window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light");"dark"==theme?(document.body.classList.add("theme-dark"),document.body.classList.remove("theme-light")):(document.body.classList.add("theme-light"),document.body.classList.remove("theme-dark")),window.innerWidth<480?document.body.classList.add("is-phone"):window.innerWidth<768?document.body.classList.add("is-tablet"):window.innerWidth<1024?document.body.classList.add("is-small-screen"):document.body.classList.add("is-large-screen")</script><div class="webpage-container workspace"><div class="sidebar-left sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"><label class="theme-toggle-container" for="theme_toggle"><input class="theme-toggle-input" type="checkbox" id="theme_toggle"><div class="toggle-background"></div></label></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="search-input-container"><input enterkeyhint="search" type="search" spellcheck="false" placeholder="Search..."><div class="search-input-clear-button" aria-label="Clear search"></div></div><include src="lib/html/file-tree.html"></include></div><script defer="">let ls = document.querySelector(".sidebar-left"); ls.classList.add("is-collapsed"); if (window.innerWidth > 768) ls.classList.remove("is-collapsed"); ls.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-left-width"));</script></div><div class="document-container markdown-reading-view hide"><div class="markdown-preview-view markdown-rendered allow-fold-headings allow-fold-lists is-readable-line-width"><style id="MJX-CHTML-styles">mjx-c.mjx-c2192::before{padding:.511em 1em .011em 0;content:"→"}mjx-c.mjx-c1D443.TEX-I::before{padding:.683em .751em 0 0;content:"P"}mjx-munderover{display:inline-block;text-align:left}mjx-munderover:not([limits=false]){padding-top:.1em}mjx-munderover:not([limits=false])>*{display:block}mjx-msubsup{display:inline-block;text-align:left}mjx-script{display:inline-block;padding-right:.05em;padding-left:.033em}mjx-script>mjx-spacer{display:block}mjx-texatom{display:inline-block;text-align:left}mjx-msub{display:inline-block;text-align:left}mjx-msqrt{display:inline-block;text-align:left}mjx-root{display:inline-block;white-space:nowrap}mjx-surd{display:inline-block;vertical-align:top}mjx-sqrt{display:inline-block;padding-top:.07em}mjx-sqrt>mjx-box{border-top:.07em solid}mjx-sqrt.mjx-tall>mjx-box{padding-left:.3em;margin-left:-.3em}mjx-mover{display:inline-block;text-align:left}mjx-mover:not([limits=false]){padding-top:.1em}mjx-mover:not([limits=false])>*{display:block;text-align:left}mjx-c.mjx-c1D70B.TEX-I::before{padding:.431em .57em .011em 0;content:"π"}mjx-c.mjx-c1D458.TEX-I::before{padding:.694em .521em .011em 0;content:"k"}mjx-c.mjx-c2F::before{padding:.75em .5em .25em 0;content:"/"}mjx-c.mjx-c1D703.TEX-I::before{padding:.705em .469em .01em 0;content:"θ"}mjx-c.mjx-c1D446.TEX-I::before{padding:.705em .645em .022em 0;content:"S"}mjx-c.mjx-c1D460.TEX-I::before{padding:.442em .469em .01em 0;content:"s"}mjx-c.mjx-c2E::before{padding:.12em .278em 0 0;content:"."}mjx-c.mjx-c1D451.TEX-I::before{padding:.694em .52em .01em 0;content:"d"}mjx-c.mjx-c221A.TEX-S4::before{padding:1.75em 1.02em 1.25em 0;content:"√"}mjx-c.mjx-c1D465.TEX-I::before{padding:.442em .572em .011em 0;content:"x"}mjx-c.mjx-cAF::before{padding:.59em .5em 0 0;content:"¯"}mjx-c.mjx-c1D441.TEX-I::before{padding:.683em .888em 0 0;content:"N"}mjx-c.mjx-c1D442.TEX-I::before{padding:.704em .763em .022em 0;content:"O"}mjx-c.mjx-c1D435.TEX-I::before{padding:.683em .759em 0 0;content:"B"}mjx-c.mjx-c1D44E.TEX-I::before{padding:.441em .529em .01em 0;content:"a"}mjx-c.mjx-c5F::before{padding:0 .5em .062em 0;content:"_"}mjx-c.mjx-c1D43A.TEX-I::before{padding:.705em .786em .022em 0;content:"G"}mjx-c.mjx-c1D467.TEX-I::before{padding:.442em .465em .011em 0;content:"z"}mjx-c.mjx-c210E.TEX-I::before{padding:.694em .576em .011em 0;content:"h"}mjx-c.mjx-c1D45C.TEX-I::before{padding:.441em .485em .011em 0;content:"o"}mjx-msup{display:inline-block;text-align:left}mjx-c.mjx-c1D43B.TEX-I::before{padding:.683em .888em 0 0;content:"H"}mjx-c.mjx-c1D437.TEX-I::before{padding:.683em .828em 0 0;content:"D"}mjx-c.mjx-c2212::before{padding:.583em .778em .082em 0;content:"−"}mjx-c.mjx-c2211.TEX-S1::before{padding:.75em 1.056em .25em 0;content:"∑"}mjx-c.mjx-c1D45D.TEX-I::before{padding:.442em .503em .194em 0;content:"p"}mjx-c.mjx-c1D456.TEX-I::before{padding:.661em .345em .011em 0;content:"i"}mjx-c.mjx-c32::before{padding:.666em .5em 0 0;content:"2"}mjx-c.mjx-c1D434.TEX-I::before{padding:.716em .75em 0 0;content:"A"}mjx-c.mjx-c1D440.TEX-I::before{padding:.683em 1.051em 0 0;content:"M"}mjx-c.mjx-c39::before{padding:.666em .5em .022em 0;content:"9"}mjx-c.mjx-c2B::before{padding:.583em .778em .082em 0;content:"+"}mjx-container[jax=CHTML]{line-height:0}mjx-container [space="1"]{margin-left:.111em}mjx-container [space="2"]{margin-left:.167em}mjx-container [space="3"]{margin-left:.222em}mjx-container [space="4"]{margin-left:.278em}mjx-container [space="5"]{margin-left:.333em}mjx-container [rspace="1"]{margin-right:.111em}mjx-container [rspace="2"]{margin-right:.167em}mjx-container [rspace="3"]{margin-right:.222em}mjx-container [rspace="4"]{margin-right:.278em}mjx-container [rspace="5"]{margin-right:.333em}mjx-container [size="s"]{font-size:70.7%}mjx-container [size=ss]{font-size:50%}mjx-container [size=Tn]{font-size:60%}mjx-container [size=sm]{font-size:85%}mjx-container [size=lg]{font-size:120%}mjx-container [size=Lg]{font-size:144%}mjx-container [size=LG]{font-size:173%}mjx-container [size=hg]{font-size:207%}mjx-container [size=HG]{font-size:249%}mjx-container [width=full]{width:100%}mjx-box{display:inline-block}mjx-block{display:block}mjx-itable{display:inline-table}mjx-row{display:table-row}mjx-row>*{display:table-cell}mjx-mtext{display:inline-block}mjx-mstyle{display:inline-block}mjx-merror{display:inline-block;color:red;background-color:#ff0}mjx-mphantom{visibility:hidden}mjx-assistive-mml{top:0;left:0;clip:rect(1px,1px,1px,1px);user-select:none;position:absolute!important;padding:1px 0 0!important;border:0!important;display:block!important;width:auto!important;overflow:hidden!important}mjx-assistive-mml[display=block]{width:100%!important}mjx-math{display:inline-block;text-align:left;line-height:0;text-indent:0;font-style:normal;font-weight:400;font-size:100%;letter-spacing:normal;border-collapse:collapse;overflow-wrap:normal;word-spacing:normal;white-space:nowrap;direction:ltr;padding:1px 0}mjx-container[jax=CHTML][display=true]{display:block;text-align:center;margin:1em 0}mjx-container[jax=CHTML][display=true][width=full]{display:flex}mjx-container[jax=CHTML][display=true] mjx-math{padding:0}mjx-container[jax=CHTML][justify=left]{text-align:left}mjx-container[jax=CHTML][justify=right]{text-align:right}mjx-mi{display:inline-block;text-align:left}mjx-c{display:inline-block}mjx-utext{display:inline-block;padding:.75em 0 .2em}mjx-mo{display:inline-block;text-align:left}mjx-stretchy-h{display:inline-table;width:100%}mjx-stretchy-h>*{display:table-cell;width:0}mjx-stretchy-h>*>mjx-c{display:inline-block;transform:scaleX(1)}mjx-stretchy-h>*>mjx-c::before{display:inline-block;width:initial}mjx-stretchy-h>mjx-ext{overflow:clip visible;width:100%}mjx-stretchy-h>mjx-ext>mjx-c::before{transform:scaleX(500)}mjx-stretchy-h>mjx-ext>mjx-c{width:0}mjx-stretchy-h>mjx-beg>mjx-c{margin-right:-.1em}mjx-stretchy-h>mjx-end>mjx-c{margin-left:-.1em}mjx-stretchy-v{display:inline-block}mjx-stretchy-v>*{display:block}mjx-stretchy-v>mjx-beg{height:0}mjx-stretchy-v>mjx-end>mjx-c{display:block}mjx-stretchy-v>*>mjx-c{transform:scaleY(1);transform-origin:left center;overflow:hidden}mjx-stretchy-v>mjx-ext{display:block;height:100%;box-sizing:border-box;border:0 solid transparent;overflow:visible clip}mjx-stretchy-v>mjx-ext>mjx-c::before{width:initial;box-sizing:border-box}mjx-stretchy-v>mjx-ext>mjx-c{transform:scaleY(500) translateY(.075em);overflow:visible}mjx-mark{display:inline-block;height:0}mjx-mn{display:inline-block;text-align:left}mjx-mfrac{display:inline-block;text-align:left}mjx-frac{display:inline-block;vertical-align:.17em;padding:0 .22em}mjx-frac[type="d"]{vertical-align:.04em}mjx-frac[delims]{padding:0 .1em}mjx-frac[atop]{padding:0 .12em}mjx-frac[atop][delims]{padding:0}mjx-dtable{display:inline-table;width:100%}mjx-dtable>*{font-size:2000%}mjx-dbox{display:block;font-size:5%}mjx-num{display:block;text-align:center}mjx-den{display:block;text-align:center}mjx-mfrac[bevelled]>mjx-num{display:inline-block}mjx-mfrac[bevelled]>mjx-den{display:inline-block}mjx-den[align=right],mjx-num[align=right]{text-align:right}mjx-den[align=left],mjx-num[align=left]{text-align:left}mjx-nstrut{display:inline-block;height:.054em;width:0;vertical-align:-.054em}mjx-nstrut[type="d"]{height:.217em;vertical-align:-.217em}mjx-dstrut{display:inline-block;height:.505em;width:0}mjx-dstrut[type="d"]{height:.726em}mjx-line{display:block;box-sizing:border-box;min-height:1px;height:.06em;border-top:.06em solid;margin:.06em -.1em;overflow:hidden}mjx-line[type="d"]{margin:.18em -.1em}mjx-mrow{display:inline-block;text-align:left}mjx-c::before{display:block;width:0}.MJX-TEX{font-family:MJXZERO,MJXTEX}.TEX-B{font-family:MJXZERO,MJXTEX-B}.TEX-I{font-family:MJXZERO,MJXTEX-I}.TEX-MI{font-family:MJXZERO,MJXTEX-MI}.TEX-BI{font-family:MJXZERO,MJXTEX-BI}.TEX-S1{font-family:MJXZERO,MJXTEX-S1}.TEX-S2{font-family:MJXZERO,MJXTEX-S2}.TEX-S3{font-family:MJXZERO,MJXTEX-S3}.TEX-S4{font-family:MJXZERO,MJXTEX-S4}.TEX-A{font-family:MJXZERO,MJXTEX-A}.TEX-C{font-family:MJXZERO,MJXTEX-C}.TEX-CB{font-family:MJXZERO,MJXTEX-CB}.TEX-FR{font-family:MJXZERO,MJXTEX-FR}.TEX-FRB{font-family:MJXZERO,MJXTEX-FRB}.TEX-SS{font-family:MJXZERO,MJXTEX-SS}.TEX-SSB{font-family:MJXZERO,MJXTEX-SSB}.TEX-SSI{font-family:MJXZERO,MJXTEX-SSI}.TEX-SC{font-family:MJXZERO,MJXTEX-SC}.TEX-T{font-family:MJXZERO,MJXTEX-T}.TEX-V{font-family:MJXZERO,MJXTEX-V}.TEX-VB{font-family:MJXZERO,MJXTEX-VB}mjx-stretchy-h mjx-c,mjx-stretchy-v mjx-c{font-family:MJXZERO,MJXTEX-S1,MJXTEX-S4,MJXTEX,MJXTEX-A!important}@font-face{font-family:MJXZERO;src:url("lib/fonts/mathjax_zero.woff") format("woff")}@font-face{font-family:MJXTEX;src:url("lib/fonts/mathjax_main-regular.woff") format("woff")}@font-face{font-family:MJXTEX-B;src:url("lib/fonts/mathjax_main-bold.woff") format("woff")}@font-face{font-family:MJXTEX-I;src:url("lib/fonts/mathjax_math-italic.woff") format("woff")}@font-face{font-family:MJXTEX-MI;src:url("lib/fonts/mathjax_main-italic.woff") format("woff")}@font-face{font-family:MJXTEX-BI;src:url("lib/fonts/mathjax_math-bolditalic.woff") format("woff")}@font-face{font-family:MJXTEX-S1;src:url("lib/fonts/mathjax_size1-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S2;src:url("lib/fonts/mathjax_size2-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S3;src:url("lib/fonts/mathjax_size3-regular.woff") format("woff")}@font-face{font-family:MJXTEX-S4;src:url("lib/fonts/mathjax_size4-regular.woff") format("woff")}@font-face{font-family:MJXTEX-A;src:url("lib/fonts/mathjax_ams-regular.woff") format("woff")}@font-face{font-family:MJXTEX-C;src:url("lib/fonts/mathjax_calligraphic-regular.woff") format("woff")}@font-face{font-family:MJXTEX-CB;src:url("lib/fonts/mathjax_calligraphic-bold.woff") format("woff")}@font-face{font-family:MJXTEX-FR;src:url("lib/fonts/mathjax_fraktur-regular.woff") format("woff")}@font-face{font-family:MJXTEX-FRB;src:url("lib/fonts/mathjax_fraktur-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SS;src:url("lib/fonts/mathjax_sansserif-regular.woff") format("woff")}@font-face{font-family:MJXTEX-SSB;src:url("lib/fonts/mathjax_sansserif-bold.woff") format("woff")}@font-face{font-family:MJXTEX-SSI;src:url("lib/fonts/mathjax_sansserif-italic.woff") format("woff")}@font-face{font-family:MJXTEX-SC;src:url("lib/fonts/mathjax_script-regular.woff") format("woff")}@font-face{font-family:MJXTEX-T;src:url("lib/fonts/mathjax_typewriter-regular.woff") format("woff")}@font-face{font-family:MJXTEX-V;src:url("lib/fonts/mathjax_vector-regular.woff") format("woff")}@font-face{font-family:MJXTEX-VB;src:url("lib/fonts/mathjax_vector-bold.woff") format("woff")}mjx-c.mjx-c1D439.TEX-I::before{padding:.68em .749em 0 0;content:"F"}mjx-c.mjx-c1D45F.TEX-I::before{padding:.442em .451em .011em 0;content:"r"}mjx-c.mjx-c1D452.TEX-I::before{padding:.442em .466em .011em 0;content:"e"}mjx-c.mjx-c1D45E.TEX-I::before{padding:.442em .46em .194em 0;content:"q"}mjx-c.mjx-c1D462.TEX-I::before{padding:.442em .572em .011em 0;content:"u"}mjx-c.mjx-c1D45B.TEX-I::before{padding:.442em .6em .011em 0;content:"n"}mjx-c.mjx-c1D450.TEX-I::before{padding:.442em .433em .011em 0;content:"c"}mjx-c.mjx-c1D466.TEX-I::before{padding:.442em .49em .205em 0;content:"y"}mjx-c.mjx-c28::before{padding:.75em .389em .25em 0;content:"("}mjx-c.mjx-c1D436.TEX-I::before{padding:.705em .76em .022em 0;content:"C"}mjx-c.mjx-c31::before{padding:.666em .5em 0 0;content:"1"}mjx-c.mjx-c29::before{padding:.75em .389em .25em 0;content:")"}mjx-c.mjx-c3D::before{padding:.583em .778em .082em 0;content:"="}</style><div class="markdown-preview-sizer markdown-preview-section"><h1 class="page-title heading inline-title" id="<strong>一、线粒体基因组主要分析内容流程图</strong>"><p dir="auto"><strong>一、线粒体基因组主要分析内容流程图</strong></p></h1><div><p dir="auto">原文链接: <a data-tooltip-position="top" aria-label="https://www.yuque.com/wusheng-s0bue/cvxne8/spy5ekikgb75z80e" rel="noopener" class="external-link" href="https://www.yuque.com/wusheng-s0bue/cvxne8/spy5ekikgb75z80e" target="_blank">线粒体基因组分析流程 (yuque.com)</a></p></div><div class="heading-wrapper"><div class="heading-children"><div><p dir="auto"><img src="https://cdn.nlark.com/yuque/0/2024/png/40551061/1709024234747-a4d99195-63f7-48e6-b473-6de9a5d44ed7.png#averageHue=%23fdfdfc&amp;clientId=uf2cae3a5-51f1-4&amp;from=paste&amp;id=u416810a9&amp;originHeight=444&amp;originWidth=853&amp;originalType=url&amp;ratio=1.5&amp;rotation=0&amp;showTitle=false&amp;status=done&amp;style=none&amp;taskId=u1df18701-586a-4e09-81f0-45c3df948a4&amp;title=" referrerpolicy="no-referrer"></p></div></div></div><div class="heading-wrapper"><h1 data-heading="**二、测试数据**" dir="auto" class="heading" id="**二、测试数据**"><strong>二、测试数据</strong></h1><div class="heading-children"><div><p dir="auto">参考序列基因组: GenBank (KY085912)</p></div><div><p dir="auto">原始reads RUN ID: SRR5602602 (Laurus nobilis L.)</p></div><div><p dir="auto">更多更详细的测试：ERR1917165，ERR2206741，ERR268390，ERR964904</p></div></div></div><div class="heading-wrapper"><h1 data-heading="**三．步骤**" dir="auto" class="heading" id="**三．步骤**"><strong>三．步骤</strong></h1><div class="heading-children"><div class="heading-wrapper"><h2 data-heading="**3.1线粒体基因组组装**" dir="auto" class="heading" id="**3.1线粒体基因组组装**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.1线粒体基因组组装</strong></h2><div class="heading-children"><div class="heading-wrapper"><h3 data-heading="**3.1.1使用软件：GetOrganelle**" dir="auto" class="heading" id="**3.1.1使用软件：GetOrganelle**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.1.1使用软件：GetOrganelle</strong></h3><div class="heading-children"><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/Kinggerm/GetOrganelle" target="_blank">https://github.com/Kinggerm/GetOrganelle</a></p></div><div><p dir="auto">其他使用者说明：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/bab69c3889b5" target="_blank">https://www.jianshu.com/p/bab69c3889b5</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://blog.csdn.net/m0_62678513/article/details/122813505" target="_blank">https://blog.csdn.net/m0_62678513/article/details/122813505</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/665269253" target="_blank">https://zhuanlan.zhihu.com/p/665269253</a></p></div><div><pre><code>
Wget https://github.com/Kinggerm/GetOrganelleGallery/raw/master/Test/reads/Arabidopsis_simulated.1.fq.gz

  

wget https://github.com/Kinggerm/GetOrganelleGallery/raw/master/Test/reads/Arabidopsis_simulated.2.fq.gz

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">参数相关：</p></div><div><pre><code>
get_organelle_from_reads.py -1 Arabidopsis_simulated.1.fq.gz -2 Arabidopsis_simulated.2.fq.gz -t 1 -o Arabidopsis_simulated.plastome -F embplant_pt -R 10

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">-F：ORGANELLE_TYPE</p></div><div><p dir="auto">-t: 线程数量</p></div><div><p dir="auto">-o: 指定输出文件夹</p></div><div><p dir="auto">-1 -2 ：指定原始输入数据</p></div></div></div><div class="heading-wrapper"><h3 data-heading="**3.1.2使用软件：MitoFinder**" dir="auto" class="heading" id="**3.1.2使用软件：MitoFinder**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.1.2使用软件：MitoFinder</strong></h3><div class="heading-children"><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/RemiAllio/MitoFinder" target="_blank">https://github.com/RemiAllio/MitoFinder</a></p></div><div><p dir="auto">其他使用者说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/e623292c9cbe" target="_blank">https://www.jianshu.com/p/e623292c9cbe</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/5bcb707c095e" target="_blank">https://www.jianshu.com/p/5bcb707c095e</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/611962326" target="_blank">https://zhuanlan.zhihu.com/p/611962326</a></p></div><div><p dir="auto">参数相关说明：</p></div><div><pre><code>
#Eg.

/Bio/xpl/software/MitoFinder -j project_name -1 left_fastq.gz -2 right_fastq.gz -r reference.gb -o 5 -p 5 -m 10

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">-j 任务的id号，输出的结果文件也用id号命名</p></div><div><p dir="auto">-r 近缘物种的已经注释的线粒体基因组文件</p></div><div><p dir="auto">-o 指遗传密码类型</p></div><div><p dir="auto">-p 使用最大的线程数</p></div><div><p dir="auto">-m 最大允许的内存消耗</p></div></div></div></div></div><div class="heading-wrapper"><h2 data-heading="**3.1.3 MEANGS**" dir="auto" class="heading" id="**3.1.3_MEANGS**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.1.3 MEANGS</strong></h2><div class="heading-children"><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/YanCCscu/meangs" target="_blank">https://github.com/YanCCscu/meangs</a></p></div><div><p dir="auto">其他使用者说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/89025d63a4c8" target="_blank">https://www.jianshu.com/p/89025d63a4c8</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/12d5f74c27f6" target="_blank">https://www.jianshu.com/p/12d5f74c27f6</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/610048488" target="_blank">https://zhuanlan.zhihu.com/p/610048488</a></p></div><div><p dir="auto">example: <a rel="noopener" class="external-link" href="https://github.com/YanCCscu/MEANGS/tree/master/example" target="_blank">https://github.com/YanCCscu/MEANGS/tree/master/example</a></p></div><div><p dir="auto">参数相关：</p></div><div><pre><code>
meangs.py -1 SRR039541.3_1.clean.fq.gz -2 SRR039541.3_2.clean.fq.gz -o HumanMito -t 16 -n 2000000 -i 300 --deepin

</code><button class="copy-code-button">复制</button></pre></div></div></div><div class="heading-wrapper"><h2 data-heading="**3.2 线粒体基因组注释**" dir="auto" class="heading" id="**3.2_线粒体基因组注释**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.2 线粒体基因组注释</strong></h2><div class="heading-children"><div><p dir="auto">使用软件：MitoFinder v1.4.1</p></div><div><p dir="auto">使用说明书：<a rel="noopener" class="external-link" href="https://github.com/RemiAllio/MitoFinder" target="_blank">https://github.com/RemiAllio/MitoFinder</a></p></div><div><p dir="auto">其他使用者说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/e623292c9cbe" target="_blank">https://www.jianshu.com/p/e623292c9cbe</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/5bcb707c095e" target="_blank">https://www.jianshu.com/p/5bcb707c095e</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/611962326" target="_blank">https://zhuanlan.zhihu.com/p/611962326</a></p></div><div><p dir="auto">example: <a rel="noopener" class="external-link" href="https://github.com/RemiAllio/MitoFinder/tree/master/test_case" target="_blank">https://github.com/RemiAllio/MitoFinder/tree/master/test_case</a></p></div><div><p dir="auto">相关参数：</p></div><div><pre><code>
mitofinder -j Hospitalitermes_medioflavus_NCBI -a Hospitalitermes_medioflavus_NCBI.fasta -r Hospitalitermes_medioflavus_NCBI.gb -o 5 -m 10 -p 10

  

#解释

-a &nbsp;assembly 序列

-j &nbsp;任务的id号，输出的结果文件也用id号命名

-r 近缘物种的已经注释的线粒体基因组文件

-o 指遗传密码类型，例如5代表无脊椎动物线粒体的遗传密码

-p 允许Mitofinder在运行时使用的最大线程数

-m 允许使用的计算机最大存储量

</code><button class="copy-code-button">复制</button></pre></div></div></div><div class="heading-wrapper"><h2 data-heading="**3.3基因组数据分析**" dir="auto" class="heading" id="**3.3基因组数据分析**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3基因组数据分析</strong></h2><div class="heading-children"><div class="heading-wrapper"><h3 data-heading="**3.3.1密码子使用偏好**" dir="auto" class="heading" id="**3.3.1密码子使用偏好**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3.1密码子使用偏好</strong></h3><div class="heading-children"><div><p dir="auto">使用软件：emboss</p></div><div><p dir="auto">使用说明书：<a rel="noopener" class="external-link" href="https://github.com/google/emboss" target="_blank">https://github.com/google/emboss</a></p></div><div><p dir="auto">其他使用者说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/457001773" target="_blank">https://zhuanlan.zhihu.com/p/457001773</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://indexofire.github.io/pathongs/book/C01_Common-Utility/04_emboss/" target="_blank">https://indexofire.github.io/pathongs/book/C01_Common-Utility/04_emboss/</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://blog.csdn.net/Cassiel60/article/details/88218706" target="_blank">https://blog.csdn.net/Cassiel60/article/details/88218706</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://max.book118.com/html/2017/0522/108259246.shtm" target="_blank">https://max.book118.com/html/2017/0522/108259246.shtm</a></p></div><div><p dir="auto">example: <a rel="noopener" class="external-link" href="https://github.com/google/emboss/tree/master/testdata" target="_blank">https://github.com/google/emboss/tree/master/testdata</a></p></div><div><p dir="auto">相关参数：</p></div><div><pre><code>
usage: embossc [-h] [--color-output {always,never,if_tty,auto}] [--import-dir IMPORT_DIRS]

&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;[--generate {cc}] [--output-path OUTPUT_PATH]

&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;input_file

</code><button class="copy-code-button">复制</button></pre></div></div></div><div class="heading-wrapper"><h3 data-heading="**3.3.2重复序列分析**" dir="auto" class="heading" id="**3.3.2重复序列分析**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3.2重复序列分析</strong></h3><div class="heading-children"><div><p dir="auto">使用软件：RepeatMasker</p></div><div><p dir="auto">软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/rmhubley/RepeatMasker" target="_blank">https://github.com/rmhubley/RepeatMasker</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.repeatmasker.org/RepeatMasker/" target="_blank">https://www.repeatmasker.org/RepeatMasker/</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/ffdbedae80fa" target="_blank">https://www.jianshu.com/p/ffdbedae80fa</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/668013133" target="_blank">https://zhuanlan.zhihu.com/p/668013133</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/532849330" target="_blank">https://zhuanlan.zhihu.com/p/532849330</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://blog.csdn.net/woodcorpse/article/details/73148464" target="_blank">https://blog.csdn.net/woodcorpse/article/details/73148464</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.repeatmasker.org/webrepeatmaskerhelp.html" target="_blank">https://www.repeatmasker.org/webrepeatmaskerhelp.html</a></p></div><div><p dir="auto">example:</p></div><div><pre><code>
基因组获取自：https://www.ncbi.nlm.nih.gov/genome/68823?genome_assembly_id=380814

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">相关参数：</p></div><div><pre><code>
RepeatMasker -pa 4 -species "Fusarium tricinctum" -poly -html -gff -dir repeat1 GCA_900382705.2_FTRI.INRA104.GCA2018.2_genomic.fna

#查看帮助RepeatMasker -h

#需要要额外注意的是:

#-dir 指定的输出结果路径，须提前建立好

#-species 指定物种，否则默认比对的是人类重复序列数据库

#如果使用本地的参考库，通过 -lib 指定，替代 -species

#-s、-q、-qq 等参数可控制序列比对的灵敏度，如果你的目标物种和参考物种不是很近，可能需要提升灵敏度

</code><button class="copy-code-button">复制</button></pre></div></div></div><div class="heading-wrapper"><h3 data-heading="**3.3.3系统发育分析**" dir="auto" class="heading" id="**3.3.3系统发育分析**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3.3系统发育分析</strong></h3><div class="heading-children"><div><p dir="auto">使用软件：raxml</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/amkozlov/raxml-ng" target="_blank">https://github.com/amkozlov/raxml-ng</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/80e2c938e050" target="_blank">https://www.jianshu.com/p/80e2c938e050</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="http://www.chenlianfu.com/?p=2225" target="_blank">http://www.chenlianfu.com/?p=2225</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://bin-ye.com/post/2020/01/04/raxml-%E4%BD%BF%E7%94%A8%E7%AE%80%E4%BB%8B/" target="_blank">https://bin-ye.com/post/2020/01/04/raxml-%E4%BD%BF%E7%94%A8%E7%AE%80%E4%BB%8B/</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/170587426" target="_blank">https://zhuanlan.zhihu.com/p/170587426</a></p></div><div><p dir="auto">example: <a rel="noopener" class="external-link" href="https://github.com/amkozlov/raxml-ng/tree/master/test/src" target="_blank">https://github.com/amkozlov/raxml-ng/tree/master/test/src</a></p></div><div><p dir="auto">相关参数:</p></div><div><pre><code>
#Eg.

raxmlHPC-PTHREADS-SSE3 -f a -m GTRGAMMA -p 12345 -x 12345 -# 10 -s 20k.phy -n chr001.raxml -T 30

  

-f :表示执行快速 Bootstrap 分析并搜索最佳得分的 ML 树。

-x : 指定一个 int 数作为随机种子

-p :指定一个随机数作为 parsimony inferences 的种子。

-# :指定 bootstrap 的次数。

-m :指定核苷酸或氨基酸替代模型。

-s :指定输入文件。phy 格式的多序列比对结果。软件包中包含一个程序来将 fasta 格式转换为 phy 格式。

-n :后缀

-T :指定多线程运行的 CPUs 。

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">使用软件：IQ-TREE</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/iqtree/iqtree2" target="_blank">https://github.com/iqtree/iqtree2</a></p></div><div><p dir="auto">其他软件说明书：<a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/408382758" target="_blank">https://zhuanlan.zhihu.com/p/408382758</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/f830282866cd" target="_blank">https://www.jianshu.com/p/f830282866cd</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/df234ca0de71" target="_blank">https://www.jianshu.com/p/df234ca0de71</a></p></div><div><p dir="auto">example:</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/iqtree/iqtree2/tree/master/example" target="_blank">https://github.com/iqtree/iqtree2/tree/master/example</a></p></div><div><p dir="auto">相关参数:</p></div><div><pre><code>
iqtree -s example.phy

</code><button class="copy-code-button">复制</button></pre></div></div></div><div class="heading-wrapper"><h3 data-heading="**3.3.4RNA编辑位点预测**" dir="auto" class="heading" id="**3.3.4RNA编辑位点预测**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3.4RNA编辑位点预测</strong></h3><div class="heading-children"><div><p dir="auto">使用软件：GIREMI &nbsp;(Generalized Identification of RNA Editing by Mutual Information)</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/zhqingit/giremi" target="_blank">https://github.com/zhqingit/giremi</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.codenong.com/jsa4285564a4a9/" target="_blank">https://www.codenong.com/jsa4285564a4a9/</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/zhqingit/giremi/issues" target="_blank">https://github.com/zhqingit/giremi/issues</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/609303893" target="_blank">https://zhuanlan.zhihu.com/p/609303893</a></p></div><div><p dir="auto">相关参数:</p></div><div><pre><code>
-f, --fasta-ref &nbsp; &nbsp;FILE &nbsp; reference genome sequence file in fasta format (NOTE: the faidx index file generated by samtools should be saved in the same directory as this fasta file)

-l, --positions &nbsp; &nbsp;FILE &nbsp; the list of all filtered SNVs after removing likely sequencing errors or SNVs due to other artifacts (see our paper for details)

-o, --output &nbsp; &nbsp; &nbsp; FILE &nbsp; write output to FILE.res

-m, --min &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;INT &nbsp; &nbsp;minimal number of total reads covering candidate editing sites &nbsp;[default: 5]

-p, --paired-end &nbsp; INT &nbsp; &nbsp;1:paired-end RNA-Seq reads; 0:single-end [default: 1]

-s, --strand &nbsp; &nbsp; &nbsp; INT &nbsp; &nbsp;0:non-strand specific RNA-Seq; 1: strand-specific RNA-Seq and read 1 (first read for the paired-end reads) is sense to RNA; 2: strand-specific RNA-Seq and read 1 is anti-sense to RNA [default: 0]

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">RNA编辑类型和效率。<strong>(A)</strong>每个组织中跨线粒体基因组的 RNA 编辑位点计数。<strong>(B)</strong>每个组织中 RNA 编辑位点位置的百分比。<strong>(C)</strong>每个组织中跨质体的 RNA 编辑位点的计数。<strong>(D)</strong>每个组织中 RNA 编辑效率的百分比。</p></div></div></div><div class="heading-wrapper"><h3 data-heading="**3.3.5基因组共线性分析**" dir="auto" class="heading" id="**3.3.5基因组共线性分析**"><div class="heading-collapse-indicator collapse-indicator collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><strong>3.3.5基因组共线性分析</strong></h3><div class="heading-children"><div><p dir="auto">使用软件：</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/krishnap25/mauve" target="_blank">https://github.com/krishnap25/mauve</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/411065032" target="_blank">https://zhuanlan.zhihu.com/p/411065032</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/9df3352eb34d" target="_blank">https://www.jianshu.com/p/9df3352eb34d</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/359203720" target="_blank">https://zhuanlan.zhihu.com/p/359203720</a></p></div><div><p dir="auto">example:</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/krishnap25/mauve/tree/main/examples" target="_blank">https://github.com/krishnap25/mauve/tree/main/examples</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/krishnap25/mauve/tree/main/tests" target="_blank">https://github.com/krishnap25/mauve/tree/main/tests</a></p></div><div><p dir="auto">相关参数:</p></div><div><pre><code>
#eg.

mauveAligner [options] &lt;seq1 filename&gt; &lt;sml1 filename&gt; ... &nbsp;&lt;seqN filename&gt; &lt;smlN filename&gt;

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">使用软件：Mummer</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/mummer4/mummer" target="_blank">https://github.com/mummer4/mummer</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/2e184e5c15b7" target="_blank">https://www.jianshu.com/p/2e184e5c15b7</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/572010609" target="_blank">https://zhuanlan.zhihu.com/p/572010609</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/36347578" target="_blank">https://zhuanlan.zhihu.com/p/36347578</a></p></div><div><p dir="auto">example: <a rel="noopener" class="external-link" href="https://github.com/mummer4/mummer/tree/master/examples" target="_blank">https://github.com/mummer4/mummer/tree/master/examples</a></p></div><div><p dir="auto">相关参数:</p></div><div><pre><code>
#eg

nucmer &nbsp;--prefix IRGSP1_DHX2 ~/reference/genome/IRGSP1.0/IRGSP-1.0_genome.fasta ~/reference/genome/rice_contigs/DHX2.fa

  

# 已有delta文件

dnadiff -d IRGSP1_DHX2.delta

# 未有delta文件

dnadiff IRGSP1_DHX2 ~/reference/genome/IRGSP1.0/IRGSP-1.0_genome.fasta ~/reference/genome/rice_contigs/DHX2.fa

</code><button class="copy-code-button">复制</button></pre></div><div><p dir="auto">使用软件：circos</p></div><div><p dir="auto">软件说明书：<a rel="noopener" class="external-link" href="https://github.com/vigsterkr/circos" target="_blank">https://github.com/vigsterkr/circos</a></p></div><div><p dir="auto">其他软件说明书：</p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://www.jianshu.com/p/9c0d2b9d724e" target="_blank">https://www.jianshu.com/p/9c0d2b9d724e</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://zhuanlan.zhihu.com/p/462783428" target="_blank">https://zhuanlan.zhihu.com/p/462783428</a></p></div><div><p dir="auto"><a rel="noopener" class="external-link" href="https://github.com/node/circos-cn" target="_blank">https://github.com/node/circos-cn</a></p></div><div><p dir="auto">example:</p></div><div><p dir="auto">相关参数:</p></div><div><p dir="auto">相关配置文件</p></div><div><pre><code>
#指定染色体文件（绝对/相对路径+文件名）

karyotype = data/karyotype/karyotype.human.txt &nbsp;

#-----------------------------------------------------------------------------------

&lt;ideogram&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#这是定义染色体相关参数的标签，以&lt;/ideogram&gt;结尾，其中包括要设置的参数

&lt;spacing&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; #定义染色体间隙宽度的标签，以&lt;/spacing&gt;，其中包括要设置的参数

default = 0.005r &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#r指的是圆的周长，设置0.5%圆的周长为间隙

#&lt;pairwise hsY;hs1&gt; &nbsp; &nbsp; &nbsp; #可以用&lt;pairwise&gt;标签特别指定某些染色体的间隙（用的是ID），因为在大多数文章中，都会留一个大间隙，来放label

#spacing = 20r &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#这里20r表示是相对default = 0.005r的20倍，也就是10%的圆的周长

#&lt;/pairwise&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#标签都要以&lt;/&gt;结尾，

&lt;/spacing&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#间隙定义结束，下面是对染色体样式的调整

radius &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; = 0.90r &nbsp;#轮廓的位置，这里的r指的是半径，由圆心到圆周上范围依次是0-1r，，超出部分将不再显示。

thickness &nbsp; &nbsp; &nbsp; &nbsp;= 20p &nbsp; &nbsp;#染色体整体的宽度，这里p指的是像素大小，也可以用r表示，1r=1500p

fill &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; = yes &nbsp; &nbsp;#是否为染色体填充颜色，如果为yes，自动用第七列定义的颜色着色

stroke_color &nbsp; &nbsp; = dgrey &nbsp;#染色体边框的颜色，支持多种格式的输入，如：red或255,182,106

stroke_thickness = 2p &nbsp; &nbsp; #染色体边框的粗细

&lt;/ideogram&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; #定义染色体属性的标签结束

#-----------------------------------------------------------------------------------

#下面是每次都要复制粘贴上去的，他们属于circos自带的配置文件，用于调用颜色，距离，报错等信息

&lt;image&gt; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;#注意路径

&lt;&lt;include etc/image.conf&gt;&gt; #注意引用外部配置文件需要使用&lt;&lt;#&gt;&gt;

&lt;/image&gt;

&lt;&lt;include etc/colors_fonts_patterns.conf&gt;&gt;

#官方没有提到下面的文件，但是没有这个文件会报错，所以还是加上去

&lt;&lt;include etc/housekeeping.conf&gt;&gt;

</code><button class="copy-code-button">复制</button></pre></div><div class="mod-footer"></div></div></div></div></div></div></div></div></div></div><div class="sidebar-right sidebar"><div class="sidebar-handle"></div><div class="sidebar-topbar"><div class="topbar-content"></div><div class="clickable-icon sidebar-collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="100%" height="100%" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><path d="M21 3H3C1.89543 3 1 3.89543 1 5V19C1 20.1046 1.89543 21 3 21H21C22.1046 21 23 20.1046 23 19V5C23 3.89543 22.1046 3 21 3Z"></path><path d="M10 4V20"></path><path d="M4 7H7"></path><path d="M4 10H7"></path><path d="M4 13H7"></path></svg></div></div><div class="sidebar-content"><div class="graph-view-wrapper"><div class="sidebar-section-header">Interactive Graph</div><div class="graph-view-placeholder">
		<div class="graph-view-container">
			<div class="graph-icon graph-expand" role="button" aria-label="Expand" data-tooltip-position="top"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon"><line x1="7" y1="17" x2="17" y2="7"></line><polyline points="7 7 17 7 17 17"></polyline></svg></div>
			<canvas id="graph-canvas" class="hide" width="512px" height="512px"></canvas>
		</div>
		</div></div><div class="tree-container mod-root nav-folder tree-item outline-tree" data-depth="0"><div class="tree-header"><span class="sidebar-section-header">Table Of Contents</span><button class="clickable-icon collapse-tree-button" aria-label="Collapse All"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></svg></button></div><div class="tree-scroll-area tree-item-children nav-folder-children"><div class="tree-item mod-tree-folder nav-folder mod-collapsible is-collapsed" style="display: none;"></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#<strong>一、线粒体基因组主要分析内容流程图</strong>"><div class="tree-item-contents heading-link" heading-name="一、线粒体基因组主要分析内容流程图"><span class="tree-item-title">一、线粒体基因组主要分析内容流程图</span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**二、测试数据**"><div class="tree-item-contents heading-link" heading-name="**二、测试数据**"><span class="tree-item-title"><strong>二、测试数据</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="1"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**三．步骤**"><div class="tree-item-contents heading-link" heading-name="**三．步骤**"><span class="tree-item-title"><strong>三．步骤</strong></span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item mod-collapsible" data-depth="2"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.1线粒体基因组组装**"><div class="tree-item-contents heading-link" heading-name="**3.1线粒体基因组组装**"><div class="collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><span class="tree-item-title"><strong>3.1线粒体基因组组装</strong></span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.1.1使用软件：GetOrganelle**"><div class="tree-item-contents heading-link" heading-name="**3.1.1使用软件：GetOrganelle**"><span class="tree-item-title"><strong>3.1.1使用软件：GetOrganelle</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.1.2使用软件：MitoFinder**"><div class="tree-item-contents heading-link" heading-name="**3.1.2使用软件：MitoFinder**"><span class="tree-item-title"><strong>3.1.2使用软件：MitoFinder</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.1.3_MEANGS**"><div class="tree-item-contents heading-link" heading-name="**3.1.3 MEANGS**"><span class="tree-item-title"><strong>3.1.3 MEANGS</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="2"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.2_线粒体基因组注释**"><div class="tree-item-contents heading-link" heading-name="**3.2 线粒体基因组注释**"><span class="tree-item-title"><strong>3.2 线粒体基因组注释</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item mod-collapsible" data-depth="2"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3基因组数据分析**"><div class="tree-item-contents heading-link" heading-name="**3.3基因组数据分析**"><div class="collapse-icon"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="svg-icon right-triangle"><path d="M3 8L12 17L21 8"></path></svg></div><span class="tree-item-title"><strong>3.3基因组数据分析</strong></span></div></a><div class="tree-item-children nav-folder-children"><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3.1密码子使用偏好**"><div class="tree-item-contents heading-link" heading-name="**3.3.1密码子使用偏好**"><span class="tree-item-title"><strong>3.3.1密码子使用偏好</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3.2重复序列分析**"><div class="tree-item-contents heading-link" heading-name="**3.3.2重复序列分析**"><span class="tree-item-title"><strong>3.3.2重复序列分析</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3.3系统发育分析**"><div class="tree-item-contents heading-link" heading-name="**3.3.3系统发育分析**"><span class="tree-item-title"><strong>3.3.3系统发育分析</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3.4RNA编辑位点预测**"><div class="tree-item-contents heading-link" heading-name="**3.3.4RNA编辑位点预测**"><span class="tree-item-title"><strong>3.3.4RNA编辑位点预测</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div><div class="tree-item" data-depth="3"><a class="tree-link" href="软件\其它生信软件\使用心得\线粒体基因组分析流程.html#**3.3.5基因组共线性分析**"><div class="tree-item-contents heading-link" heading-name="**3.3.5基因组共线性分析**"><span class="tree-item-title"><strong>3.3.5基因组共线性分析</strong></span></div></a><div class="tree-item-children nav-folder-children"></div></div></div></div></div></div></div></div></div><script defer="">let rs = document.querySelector(".sidebar-right"); rs.classList.add("is-collapsed"); if (window.innerWidth > 768) rs.classList.remove("is-collapsed"); rs.style.setProperty("--sidebar-width", localStorage.getItem("sidebar-right-width"));</script></div></div></body></html>